<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.19.3 by Michael Rose
  Copyright 2013-2019 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
<html lang="en" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>Review of Papers | Jérémie</title>
<meta name="description" content="Here is the draft of a list of the papers I have read with a short personal summary of them">


  <meta name="author" content="Jérémie Kalfon">


<meta property="og:type" content="website">
<meta property="og:locale" content="en_US">
<meta property="og:site_name" content="Jérémie">
<meta property="og:title" content="Review of Papers">
<meta property="og:url" content="https://jkobject.com/review-of-papers/">


  <meta property="og:description" content="Here is the draft of a list of the papers I have read with a short personal summary of them">



  <meta property="og:image" content="https://jkobject.com/assets/images/research.jpg">



  <meta name="twitter:site" content="@jkobject">
  <meta name="twitter:title" content="Review of Papers">
  <meta name="twitter:description" content="Here is the draft of a list of the papers I have read with a short personal summary of them">
  <meta name="twitter:url" content="https://jkobject.com/review-of-papers/">

  
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image" content="https://jkobject.com/assets/images/research.jpg">
  

  







  
    <meta property="article:publisher" content="https://www.facebook.com/jkobject">
  

  


<link rel="canonical" href="https://jkobject.com/review-of-papers/">




<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": "Jeremie Kalfon",
      "url": "https://jkobject.com/",
      "sameAs": ["https://twitter.com/jkobject","https://LinkedIn.com/in/jkobject","https://ResearchGate.com/jeremie-kalfon"]
    
  }
</script>






<!-- end _includes/seo.html -->


<link href="/feed.xml" type="application/atom+xml" rel="alternate" title="Jérémie Feed">

<!-- https://t.co/dKP3o1e -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5/css/all.min.css">

<!--[if IE]>
  <style>
    /* old IE unsupported flexbox fixes */
    .greedy-nav .site-title {
      padding-right: 3em;
    }
    .greedy-nav button {
      position: absolute;
      top: 0;
      right: 0;
      height: 100%;
    }
  </style>
<![endif]-->



    <!-- start custom head snippets -->

<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    extensions: [
      "MathMenu.js",
      "MathZoom.js",
      "AssistiveMML.js",
      "a11y/accessibility-menu.js"
    ],
    jax: ["input/TeX", "output/CommonHTML"],
    TeX: {
      extensions: [
        "AMSmath.js",
        "AMSsymbols.js",
        "noErrors.js",
        "noUndefined.js",
      ]
    }
  });
</script>
<script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML">
</script>
<!-- insert favicons. use https://realfavicongenerator.net/ -->

<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/site.webmanifest">
<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#2b5797">
<meta name="theme-color" content="#ffffff">

<!-- end custom head snippets -->
  </head>

  <body class="layout--single wide">
    <nav class="skip-links">
  <h2 class="screen-reader-text">Skip links</h2>
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
          <a class="site-logo" href="/"><img src="/assets/images/id-88.png" alt=""></a>
        
        <a class="site-title" href="/">
          Jérémie K.
          
        </a>
        <ul class="visible-links"><li class="masthead__menu-item">
              <a href="/about/" title="I am a human. I wish to make the world better with science. <3">About</a>
            </li><li class="masthead__menu-item">
              <a href="/projects/" title="">Projects</a>
            </li><li class="masthead__menu-item">
              <a href="/opinions/">Opinions</a>
            </li><li class="masthead__menu-item">
              <a href="/ideas/">Ideas</a>
            </li><li class="masthead__menu-item">
              <a href="/blog/" title="A short extract about my life experiences.">Blog</a>
            </li><li class="masthead__menu-item">
              <a href="/papers/">Papers</a>
            </li><li class="masthead__menu-item">
              <a href="/courses/" title="Here is the list of the Courses I have received and the one I have given.">Courses</a>
            </li><li class="masthead__menu-item">
              <a href="/culture/" title="This is my personal list of books, Comics and films I love.">Culture</a>
            </li></ul>
        
        <button class="search__toggle" type="button">
          <span class="visually-hidden">Toggle search</span>
          <svg class="icon" width="16" height="16" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15.99 16">
            <path d="M15.5,13.12L13.19,10.8a1.69,1.69,0,0,0-1.28-.55l-0.06-.06A6.5,6.5,0,0,0,5.77,0,6.5,6.5,0,0,0,2.46,11.59a6.47,6.47,0,0,0,7.74.26l0.05,0.05a1.65,1.65,0,0,0,.5,1.24l2.38,2.38A1.68,1.68,0,0,0,15.5,13.12ZM6.4,2A4.41,4.41,0,1,1,2,6.4,4.43,4.43,0,0,1,6.4,2Z" transform="translate(-.01)"></path>
          </svg>
        </button>
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      
  







<div class="page__hero--overlay"
  style=" background-image: url('/assets/images/research.jpg');"
>
  
    <div class="wrapper">
      <h1 id="page-title" class="page__title" itemprop="headline">
        
          Review of Papers

        
      </h1>
      
        <p class="page__lead">Here is the draft of a list of the papers I have read with a short personal summary of them
</p>
      
      
      
      
    </div>
  
  
    <span class="page__hero-caption">thanks to dribble for this amazing design
</span>
  
</div>




  
    



<nav class="breadcrumbs">
  <ol itemscope itemtype="https://schema.org/BreadcrumbList">
    
    
    
      
        <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
          <a href="https://jkobject.com/" itemprop="item"><span itemprop="name">Home</span></a>
          <meta itemprop="position" content="1" />
        </li>
        <span class="sep">/</span>
      
      
        <li class="current">Review of Papers</li>
      
    
  </ol>
</nav>

  


<div id="main" role="main">
  
  <div class="sidebar sticky">
  


<div itemscope itemtype="https://schema.org/Person">

  
    <div class="author__avatar">
      
        <img src="/assets/images/id.png" alt="Jérémie Kalfon" itemprop="image">
      
    </div>
  

  <div class="author__content">
    
      <h3 class="author__name" itemprop="name">Jérémie Kalfon</h3>
    
    
      <div class="author__bio" itemprop="description">
        <p>I am an Engineer, Researcher and Entrepreneur. I wish to make the world a better place with science!</p>

      </div>
    
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
        <li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place">
          <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name">Paris - Boston</span>
        </li>
      

      
        
          
            <li><a href="https://twitter.com/jkobject" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i><span class="label">Twitter</span></a></li>
          
        
          
            <li><a href="https://Scholar.google.com/citations?user=zyXiydsAAAAJ" rel="nofollow noopener noreferrer"><i class="fas fa-fw fa-graduation-cap" aria-hidden="true"></i><span class="label">Scholar</span></a></li>
          
        
          
            <li><a href="https://LinkedIn.com/in/jkobject" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span class="label">LinkedIn</span></a></li>
          
        
      

      

      
        <li>
          <a href="mailto:jkobject@gmail.com">
            <meta itemprop="email" content="jkobject@gmail.com" />
            <i class="fas fa-fw fa-envelope-square" aria-hidden="true"></i><span class="label">Email</span>
          </a>
        </li>
      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      <!--
  <li>
    <a href="http://link-to-whatever-social-network.com/user/" itemprop="sameAs" rel="nofollow noopener noreferrer">
      <i class="fas fa-fw" aria-hidden="true"></i> Custom Social Profile Link
    </a>
  </li>
-->
    </ul>
  </div>
</div>

  
  </div>



  <article class="page" itemscope itemtype="https://schema.org/CreativeWork">
    <meta itemprop="headline" content="Review of Papers">
    <meta itemprop="description" content="Here is the draft of a list of the papers I have read with a short personal summary of them">
    
    

    <div class="page__inner-wrap">
      

      <section class="page__content" itemprop="text">
        
          <aside class="sidebar__right ">
            <nav class="toc">
              <header><h4 class="nav__title"><i class="fas fa-file-alt"></i> On this page</h4></header>
              <ul class="toc__menu">
  <li><a href="#overview">Overview</a>
    <ul>
      <li><a href="#summary">SUMMARY</a></li>
      <li><a href="#evaluation">EVALUATION</a></li>
      <li><a href="#synthesis">SYNTHESIS</a></li>
      <li><a href="#7-things-that-would-make-papers-better-after-156-papers">7 things that would make papers better (after 156 papers)</a></li>
    </ul>
  </li>
  <li><a href="#papers">Papers</a>
    <ul>
      <li><a href="#attention-in-a-bayesian-framework---sahani">Attention in a bayesian framework - sahani</a></li>
      <li><a href="#neural-coding--peter-latham">neural coding  Peter latham</a></li>
      <li><a href="#cracking-the-neural-code-peter-latham">cracking the neural code –peter latham</a></li>
      <li><a href="#alex-graves-talk-at-ijcnn">Alex graves talk at IJCNN</a></li>
      <li><a href="#neural-turing-machine-graves">neural turing machine Graves</a></li>
      <li><a href="#predicting-the-sequence-specificities-of-dna--and-rna-binding-proteins-by-deep-learning-brendan-frey">Predicting the sequence specificities of DnA- and RnA-binding proteins by deep learning Brendan Frey</a></li>
      <li><a href="#deep-learning-free-text-and-sentence-embedding--sanjeev-arora">Deep-learning-free Text and Sentence Embedding  –Sanjeev Arora</a></li>
      <li><a href="#how-do-we-capture-structure-in-relational-data-sanjeev">How do we capture structure in relational data –Sanjeev</a></li>
      <li><a href="#cancer-metastasis-detection-with-neural-conditional-random-field-yi-li">Cancer Metastasis Detection With Neural Conditional Random Field –Yi Li</a></li>
      <li><a href="#systematic-identification-of-phosphorylation--mediated-protein-interaction-switches">Systematic identification of phosphorylation- mediated protein interaction switches</a></li>
      <li><a href="#a-polymorphism-in-the-tumor-suppressor-p53-affects-aging-and-longevity-in-mouse-models">A polymorphism in the tumor suppressor p53 affects aging and longevity in mouse models</a></li>
      <li><a href="#gene2vector">gene2vector</a></li>
      <li><a href="#multi-omics-factor-analysis---a-framework-for-unsupervised-integration-of-multi-omic-data-sets--ricard-argelaguet-et-al">Multi-Omics factor analysis - a framework for unsupervised integration of multi-omic data sets – Ricard Argelaguet et al.</a></li>
      <li><a href="#basset-learning-the-regulatory-code-of-the-accessible-genome-with-deep-convolutional-neural-networks--david-kelley">Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks – David Kelley</a></li>
      <li><a href="#machine-theory-of-mind">Machine theory of Mind</a></li>
      <li><a href="#analysis-of-genomic-sequence-motifs-for-deciphering-transcription-factor-binding-and-transcriptional-regulation-in-eukaryotic-cells-valentina-boeva">Analysis of Genomic Sequence Motifs for Deciphering Transcription Factor Binding and Transcriptional Regulation in Eukaryotic Cells –Valentina Boeva</a></li>
      <li><a href="#gpu-kernels-for-block-sparse-weights-scott-gray-openai">GPU Kernels for Block-Sparse Weights –Scott Gray, openAI</a></li>
      <li><a href="#node2vec-scalable-feature-learning-for-networks--aditya-grover-stanford">node2vec: Scalable Feature Learning for Networks – Aditya Grover, stanford</a></li>
      <li><a href="#microrna-signature-analysis-in-colorectal-cancer-identification-of-expression-profiles-in-stage-ii-tumors-associated-with-aggressive-disease-kah-hoong-chang">MicroRNA signature analysis in colorectal cancer: identification of expression profiles in stage II tumors associated with aggressive disease –Kah Hoong Chang</a></li>
      <li><a href="#efficient-and-robust-automated-machine-learning-matthias-feurer-freiburg-university">Efficient and Robust Automated Machine Learning –Matthias Feurer, Freiburg University</a></li>
      <li><a href="#vicus-exploiting-local-structures-to-improve-network-based-analysis-of-biological-data--bo-wang-serafim-batzoglou">Vicus: Exploiting local structures to improve network-based analysis of biological data – Bo Wang, Serafim Batzoglou</a></li>
      <li><a href="#active-learning-of-cortical-connectivity-from-two-photon-imaging-data-mart-́ın-a-bertra-́n-duke-uni">Active learning of cortical connectivity from two-photon imaging data –Mart ́ın A. Bertra ́n, Duke Uni</a></li>
      <li><a href="#finding-syntax-in-human-encephalography-with-beam-search-john-hale-oxf-deepmind">Finding Syntax in Human Encephalography with Beam Search –John Hale, Oxf, DeepMind</a></li>
      <li><a href="#glow-flow-based-generative-network-openai">GLOW: FLow based generative network –OpenAI</a></li>
      <li><a href="#prediction-of-acute-myeloid-leukemia-risk-in-healthy-individuals-eran-segal-moritz-gerstung">Prediction of acute myeloid leukemia risk in healthy individuals –Eran Segal, Moritz Gerstung</a></li>
      <li><a href="#a-synthetic-diploid-benchmark-for-accurate-variant-calling-evaluation--mc-arthur">A synthetic-diploid benchmark for accurate variant-calling evaluation – Mc Arthur</a></li>
      <li><a href="#meta-analysis-of-the-diagnostic-and-clinical-utility-of-genome-and-exome-sequencing-and-chromosomal-microarray-in-children-with-suspected-genetic-diseases--michelle-clark">Meta-analysis of the diagnostic and clinical utility of genome and exome sequencing and chromosomal microarray in children with suspected genetic diseases – Michelle Clark</a></li>
      <li><a href="#multi-scale-deep-tensor-factorization-learns-a-latent-representation-of-the-human-epigenome--jacob-schreiber-uw">Multi-scale deep tensor factorization learns a latent representation of the human epigenome – Jacob Schreiber UW</a></li>
      <li><a href="#machine-learning-applications-in-genetics-and-genomics--william-noble-uw">Machine learning applications in genetics and genomics – William Noble, UW</a></li>
      <li><a href="#bioambients-an-abstraction-for-biological-compartments">BioAmbients: An abstraction for biological compartments</a></li>
      <li><a href="#personal-clinical-history-predicts-antibiotic-resistance-in-urinary-tract-infections--yelinsnitsernovich">Personal clinical history predicts antibiotic resistance in urinary tract infections -Yelin,Snitser,Novich</a></li>
      <li><a href="#deciphering-regulatory-dna-sequences-and-noncoding-genetic-variants-using-neural-network-models-of-massively-parallel-reporter-assays---rajiv-movva-kundaje">Deciphering regulatory DNA sequences and noncoding genetic variants using neural network models of massively parallel reporter assays - Rajiv Movva, Kundaje</a></li>
      <li><a href="#clinical-concept-embeddings-learned-from-massive-sources-of-medical-data--andrew-l-beam1-benjamin-kompa-unc-harvard">Clinical Concept Embeddings Learned from Massive Sources of Medical Data – Andrew L Beam1, Benjamin Kompa UNC, Harvard</a></li>
      <li><a href="#pooled-screens-in-human-cells-david-feldman-el-al">pooled screens in human cells, David Feldman el al</a></li>
      <li><a href="#manifold-tiling-localized-receptive-fields-are-optimal-in-similarity-preserving-neural-networks-pehlevan-tepper-fi-ccb">Manifold-tiling Localized Receptive Fields are Optimal in Similarity-preserving Neural Networks, Pehlevan, Tepper FI CCB</a></li>
      <li><a href="#can-deep-learning-improve-genomic-prediction-of-complex-human-traits---pau-bellot-et-al">Can deep learning improve genomic prediction of complex human traits? - Pau Bellot et al</a></li>
      <li><a href="#learning-graph-level-representation-for-drug-discovery--junying-li-et-al">Learning Graph-Level Representation for Drug Discovery – Junying Li et. al.</a></li>
      <li><a href="#multiplier-a-transfer-learning-framework-reveals-systemic-features-of-rare-autoimmune-disease--jaclyn-n-taroni-et-al">MultiPLIER: a transfer learning framework reveals systemic features of rare autoimmune disease – Jaclyn N. Taroni et. al.</a></li>
      <li><a href="#on-entropy-and-information-in-gene-interaction-networks--z-s-wallace1-s-b-rosenthal">On entropy and information in gene interaction networks – Z. S. Wallace1, S. B. Rosenthal</a></li>
      <li><a href="#allele-specific-binding-of-rna-binding-proteins-reveals-functional-genetic-variants-in-the-rna--ei-wen-yang-jae-hoon-bahn">Allele-specific binding of RNA-binding proteins reveals functional genetic variants in the RNA – Ei-Wen Yang, Jae Hoon Bahn</a></li>
      <li><a href="#in-silico-prediction-of-high-resolution-hi-c-interaction-matrices-shilu-zhang1-deborah-chasman">In silico prediction of high-resolution Hi-C interaction matrices –Shilu Zhang1, Deborah Chasman.</a></li>
      <li><a href="#winner-take-all-autoencoders--brendan-frey">Winner-Take-All Autoencoders – Brendan Frey</a></li>
      <li><a href="#dissecting-splicing-decisions-and-cell-to-cell-variability-with-designed-sequence-libraries--eran-segal-martin-mikl">Dissecting splicing decisions and cell-to-cell variability with designed sequence libraries – Eran Segal, Martin Mikl</a></li>
      <li><a href="#quantum-entanglement-in-neural-network-states--dong-ling-deng-xiaopeng-li">Quantum Entanglement in Neural Network States – Dong-Ling Deng, Xiaopeng Li</a></li>
      <li><a href="#cancer-archetypes-co-opt-and-adapt-the-transcriptional-programs-of-existing-cellular-states--maayan-baron1-isabella-s-kim">Cancer archetypes co-opt and adapt the transcriptional programs of existing cellular states – Maayan Baron1, Isabella S. Kim</a></li>
      <li><a href="#deep-learning-sequence-based-ab-initio-prediction-of-variant-effects-on-expression-and-disease-risk--troyanskaya-fi">Deep learning sequence-based ab initio prediction of variant effects on expression and disease risk – Troyanskaya, FI</a></li>
      <li><a href="#one-shot-imitation-learning--ilya-sutskever-pieter-abbeel">One-Shot Imitation Learning – Ilya Sutskever, Pieter Abbeel</a></li>
      <li><a href="#mcm2-promotes-symmetric-inheritance-of-modified-histones-during-dna-replication--nataliya-petryk-maria-dalby">MCM2 promotes symmetric inheritance of modified histones during DNA replication – Nataliya Petryk, Maria Dalby</a></li>
      <li><a href="#deep-surveying-of-alternative-splicing-complexity-in-the-human-transcriptome-by-high-throughput-sequencing--qun-pan-ofer-shai-frey">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing – Qun Pan, Ofer Shai, Frey</a></li>
      <li><a href="#sequential-regulatory-activity-prediction-across-chromosomes-with-convolutional-neural-networks-david-r-kelley-yakir-a-reshef">Sequential regulatory activity prediction across chromosomes with convolutional neural networks. –David R. Kelley, Yakir A. Reshef</a></li>
      <li><a href="#deciphering-the-splicing-code--yoseph-barash-frey">Deciphering the splicing code — Yoseph Barash, Frey</a></li>
      <li><a href="#impala--deepmind">IMPALA – DeepMind</a></li>
      <li><a href="#unsupervised-embedding-of-single-cell-hi-c-data---jie-liu-galip-gurkan-yardımcı-uw">Unsupervised embedding of single-cell Hi-C data - Jie Liu, Galip Gurkan Yardımcı, UW</a></li>
      <li><a href="#an-integrated-encyclopedia-of-dna-elements-in-the-human-genome--encode">An integrated encyclopedia of DNA elements in the human genome – ENCODE</a></li>
      <li><a href="#engineering-stem-cell-organoids--xiaolei-yin-benjamin-e-mead">Engineering Stem Cell Organoids – Xiaolei Yin Benjamin E. Mead</a></li>
      <li><a href="#advantages-and-limitations-of-current-network-inference-methods--riet-de-smet-and-kathleen-marchal">Advantages and limitations of current network inference methods – Riet De Smet and Kathleen Marchal</a></li>
      <li><a href="#optimization-techniques-on-riemannian-manifolds--steven-smith">Optimization Techniques on Riemannian Manifolds – Steven Smith</a></li>
      <li><a href="#the-tsetlin-machine---a-game-theoretic-bandit-driven-approach-to-optimal-pattern-recognition-with-propositional-logic--granmo">The Tsetlin Machine - A Game Theoretic Bandit Driven Approach to Optimal Pattern Recognition with Propositional Logic – Granmo</a></li>
      <li><a href="#the-genetics-of-transcription-factor-dna-binding-variation--deplancke">The Genetics of Transcription Factor DNA Binding Variation – Deplancke</a></li>
      <li><a href="#seganndb-interactive-web-based-genomic-segmentation-hocking">SegAnnDB interactive Web-based genomic segmentation –Hocking</a></li>
      <li><a href="#defining-a-cancer-dependency-map--aviad-tsherniak-francisca-vazquez-phil-g-montgomery-broadinstitute">Defining a Cancer Dependency Map – Aviad Tsherniak, Francisca Vazquez, Phil G. Montgomery BroadInstitute</a></li>
      <li><a href="#improved-estimation-of-cancer-dependencies-from-large-scale-rnai-screens-using-model-based-normalization-and-data-integration--james-mcfarland-broadinstitute">Improved estimation of cancer dependencies from large-scale RNAi screens using model-based normalization and data integration – james McFarland BroadInstitute</a></li>
      <li><a href="#genetic-and-transcriptional-evolution-alters-cancer-cell-line-drug-response--uri-ben-david1-benjamin-siranosian-gavin-ha-helen-tang-yaara-oren-kunihiko-hinohara-craig-strathdee-joshua-dempster-nicholas-lyons-robert-burns-anwesha-nag-guillaume-kugener-beth-cimini-peter-tsvetkov-yosef-maruvka1-ryan-orourke-anthony-garrity-andrew-tubelli-pratiti-bandopadhay-aviad-tsherniak">Genetic and transcriptional evolution alters cancer cell line drug response – Uri Ben-David1, Benjamin Siranosian, Gavin Ha, Helen tang, Yaara Oren, Kunihiko Hinohara, craig Strathdee, Joshua Dempster, Nicholas lyons, robert Burns, Anwesha Nag, Guillaume Kugener, Beth cimini, Peter tsvetkov, Yosef Maruvka1, ryan O’rourke, Anthony Garrity, Andrew tubelli, Pratiti Bandopadhay, Aviad tsherniak</a></li>
      <li><a href="#massively-multilingual-sentence-embeddings-for-zero-shot-cross-lingual-transfer-and-beyond--mikel-artetxe-holger-schwenk">Massively Multilingual Sentence Embeddings for Zero-Shot Cross-Lingual Transfer and Beyond – Mikel Artetxe, Holger Schwenk.</a></li>
      <li><a href="#cytogan-generative-modeling-of-cell-images--imaging-platform-broad-institute-of-mit-and-harvard">CytoGAN: Generative Modeling of Cell Images – Imaging Platform Broad Institute of MIT and Harvard</a></li>
      <li><a href="#pathway-perturbations-in-signaling-networks-linking-genotype-to-phenotype-babu-for-barth-epfl">Pathway perturbations in signaling networks: Linking genotype to phenotype –Babu for Barth EPFL</a></li>
      <li><a href="#pharmacogenomics-of-gpcr-drug-targets---masuho-leonie-j-jahn-kirill-a-martemyanov-david-e-gloriam-m-madan-babu">Pharmacogenomics of GPCR Drug Targets –  Masuho, Leonie J. Jahn, Kirill A. Martemyanov, David E. Gloriam, M. Madan Babu</a></li>
      <li><a href="#papers-gilchrist">Papers Gilchrist</a>
        <ul>
          <li><a href="#combining-models-of-protein-translation-and-population-genetics-to-predict-protein-production-rates-from-codon-usage-patterns">Combining Models of Protein Translation and Population Genetics to Predict Protein Production Rates from Codon Usage Patterns</a></li>
          <li><a href="#explaining-complex-codon-usage-patterns-with-selection-for-translational-efficiency-mutation-bias-and-genetic-drift">Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift</a></li>
          <li><a href="#estimating-gene-expression-and-codon-specific-translational-efficiencies-mutation-biases-and-selection-coefficients-from-genomic-data-alone">Estimating Gene Expression and Codon-Specific Translational Efficiencies, Mutation Biases, and Selection Coefficients from Genomic Data Alone</a></li>
          <li><a href="#quantifying-codon-usage-in-signal-peptides-gene-expression-and-amino-acid-t-usage-explain-apparent-selection-for-inefficient-codons">Quantifying codon usage in signal peptides: Gene expression and amino acid T usage explain apparent selection for inefficient codons</a></li>
        </ul>
      </li>
      <li><a href="#where-genome-biology-is-going-next">Where genome biology is going next</a></li>
      <li><a href="#a-topological-and-conformational-stability-alphabet-for-multipass-membrane-proteins--patrick-barth">A topological and conformational stability alphabet for multipass membrane proteins – Patrick Barth</a></li>
      <li><a href="#primer-on-deep-learning-in-genomics">Primer on Deep Learning in Genomics</a></li>
      <li><a href="#a-guide-to-dl-in-healthcare--andre-esteva-alexandre-robicquet">A guide to DL in healthcare – Andre Esteva, Alexandre robicquet</a></li>
      <li><a href="#mammalian-genes-are-transcribed-with-widely-different-bursting-kinetics">Mammalian Genes Are Transcribed with Widely Different Bursting Kinetics</a></li>
      <li><a href="#what-are-super-enhancers--sebastian-pott--jason-d-lieb">what are super enhancers? – Sebastian Pott &amp; Jason D Lieb</a></li>
      <li><a href="#super-enhancers-in-the-control-of-cell-identity-and-disease--denes-hnisz-brian-j-abraham-tong-ihn-lee">Super-Enhancers in the Control of Cell Identity and Disease – Denes Hnisz, Brian J. Abraham, Tong Ihn Lee</a></li>
      <li><a href="#network-in-network--min-lin-qiang-chen-shuicheng-yan">Network In Network – Min Lin, Qiang Chen, Shuicheng Yan</a></li>
      <li><a href="#selective-inhibition-of-tumor-oncogenes-by-disruption-of-super-enhancers--jakob-love-heather-a-hoke-charles-y-lin">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers – Jakob Love, Heather A. Hoke, Charles Y. Lin</a></li>
      <li><a href="#ataris-computational-quantification-of-gene-suppression-phenotypes-from-multisample-rnai-screens-diane-d-shao-aviad-tsherniak-shuba-gopal-barbara-a-weir">ATARiS: Computational quantification of gene suppression phenotypes from multisample RNAi screens– Diane D. Shao, Aviad Tsherniak, Shuba Gopal, Barbara A. Weir</a></li>
      <li><a href="#pan-cancer-genome-and-transcriptome-analyses-of-1699-paediatric-leukaemias-and-solid-tumours-xiaotu-ma1-yu-liu-yanling-liu-ludmil-b-alexandrov-michael-n-edmonson">Pan-cancer genome and transcriptome analyses of 1,699 paediatric leukaemias and solid tumours– Xiaotu Ma1, Yu Liu, Yanling Liu, Ludmil B. Alexandrov, Michael n. Edmonson</a></li>
      <li><a href="#the-landscape-of-genomic-alterations-across-childhood-cancerssusanne-n-gröbner-barbara-c-worst-joachim-weischenfeldt-ivo-buchhalter-kortine-kleinheinz-vasilisa-a-rudneva-pascal-d-johann">The landscape of genomic alterations across childhood cancers–Susanne N. Gröbner, Barbara C. Worst, Joachim Weischenfeldt Ivo Buchhalter, Kortine Kleinheinz, Vasilisa A. Rudneva Pascal D. Johann</a></li>
      <li><a href="#a-landscape-of-pharmacogenomic-interactions-in-cancer-francesco-iorio-theo-a-knijnenburg-daniel-j-vis">A Landscape of Pharmacogenomic Interactions in Cancer– Francesco Iorio, Theo A. Knijnenburg, Daniel J. Vis</a></li>
      <li><a href="#curiosity-driven-exploration-by-self-supervised-prediction-deepak-pathak-pulkit-agrawal">Curiosity-driven Exploration by Self-supervised Prediction– Deepak Pathak Pulkit Agrawal</a></li>
      <li><a href="#master-transcription-factors-and-mediator-establish-super-enhancers-at-key-cell-identity-genes-warren-a-whyte14-david-a-orlando14-denes-hnisz">Master Transcription Factors and Mediator Establish Super-Enhancers at Key Cell Identity Genes– Warren A. Whyte,1,4 David A. Orlando,1,4 Denes Hnisz</a></li>
      <li><a href="#the-hallmarks-of-cancer-douglas-hanahan-and-robert-a-weinberg">The Hallmarks of Cancer– Douglas Hanahan* and Robert A. Weinberg</a></li>
      <li><a href="#coming-of-age-ten-years-of-next--generation-sequencing-technologiesara-goodwin-john-d-mcpherson-and-w-richard-mccombie">Coming of age: ten years of next- generation sequencing technologie–Sara Goodwin, John D. McPherson and W. Richard McCombie</a></li>
      <li><a href="#high-performance-medicine-the-convergence-of-human-and-artificial-intelligence-eric-j-topol">High-performance medicine: the convergence of human and artificial intelligence– Eric J. Topol</a></li>
      <li><a href="#generating-focussed-molecule-libraries-for-drug-discovery-with-recurrent-neural-networks-marwin-hs-segler1-thierry-kogej-christian-tyrchan-and-mark-p-waller">Generating Focussed Molecule Libraries for Drug Discovery with Recurrent Neural Networks– Marwin HS Segler1, Thierry Kogej, Christian Tyrchan and Mark P Waller</a></li>
      <li><a href="#core-transcriptional-regulatory-circuitry-in-human-embryonic-stem-cells">Core Transcriptional Regulatory Circuitry in Human Embryonic Stem Cells</a></li>
      <li><a href="#selective-gene-dependencies-in-mycn-amplified-neuroblastoma-include-the-core-transcriptional-regulatory-circuitry-adam-d-durbin--mark-w-zimmerman-neekesh-v-dharia">Selective gene dependencies in MYCN-amplified neuroblastoma include the core transcriptional regulatory circuitry– Adam D. Durbin  Mark W. Zimmerman Neekesh V. Dharia</a></li>
      <li><a href="#neuroblastoma-is-composed-of-two-super-enhancer--associated-differentiation-states-tim-van-groningen-jan-koster-linda-j-valentijn-danny-a-zwijnenburg-nurdan-akogul-nancy-e-hasselt-marloes-broekmans">Neuroblastoma is composed of two super-enhancer- associated differentiation states– Tim van Groningen, Jan Koster, Linda J Valentijn, Danny A Zwijnenburg, Nurdan Akogul, Nancy E Hasselt, Marloes Broekmans</a></li>
      <li><a href="#interaction-networks-for-learning-about-objects-relations-and-physics-deepmind">Interaction Networks for Learning about Objects, Relations and Physics– DeepMind</a></li>
      <li><a href="#precision-targeting-of-bfl-1a1-and-an-atm-co--dependency-in-human-cancer-rachel-m-guerra-gregory-h-bird-edward-p-harvey">Precision Targeting of BFL-1/A1 and an ATM Co- dependency in Human Cancer– Rachel M. Guerra, Gregory H. Bird, Edward P. Harvey</a></li>
      <li><a href="#pioneer-factors-govern-super-enhancer-dynamics-in-stem-cell-plasticity-and-lineage-choice-rene-c-adam-hanseul-yang-shira-rockowitz-samantha-b-larsen-maria-nikolova-daniel-s-oristian">Pioneer factors govern super-enhancer dynamics in stem cell plasticity and lineage choice– Rene C. Adam, Hanseul Yang, Shira Rockowitz, Samantha B. Larsen, Maria Nikolova, Daniel S. Oristian</a></li>
      <li><a href="#the-evolving-war-on-cancer-daniel-a-haber-nathanael-s-gray-and-jose-baselga">The Evolving War on Cancer– Daniel A. Haber, Nathanael S. Gray and Jose Baselga</a></li>
      <li><a href="#statistical-methods-in-integrative-genomics-sylvia-richardson-george-c-tseng-and-wei-sun">Statistical Methods in Integrative Genomics– Sylvia Richardson, George C. Tseng, and Wei Sun</a></li>
      <li><a href="#the-cancer-genome-atlas-pan-cancer-analysis-project-the-cancer-genome-atlas-research-network-john-n-weinstein-eric-a-collisson-gordon-b-mills-kenna-r-mills-shaw5-brad-a-ozenberge">The Cancer Genome Atlas Pan-Cancer analysis project– The Cancer Genome Atlas Research Network, John N Weinstein, Eric A Collisson, Gordon B Mills, Kenna R Mills Shaw5, Brad A Ozenberge…</a></li>
      <li><a href="#the-cancer-cell-line-encyclopedia-enables-predictive-modelling-of-anticancer-drug-sensitivity-jordi-barretina-giordano-caponigro-nicolas-stransky-kavitha-venkatesan-adam-a-margolin-sungjoon-kim-christopher-j-wilson4-joseph-leha-gregory-v-kryukov-dmitriy-sonkin-anupama-reddy-manway-liu-lauren-murray-michael-f-berger-john-e-monahan">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity– Jordi Barretina, Giordano Caponigro, Nicolas Stransky, Kavitha Venkatesan, Adam A. Margolin, Sungjoon Kim, Christopher J. Wilson4, Joseph Leha, Gregory V. Kryukov, Dmitriy Sonkin, Anupama Reddy, Manway Liu, Lauren Murray, Michael F. Berger, John E. Monahan…</a></li>
      <li><a href="#deepnano-deep-recurrent-neural-networks-for-base-calling-in-minion-nanopore-reads-vladimir-bozabroňa-brejova-toma-svinař">DeepNano: Deep recurrent neural networks for base calling in MinION nanopore reads— Vladimir Boza,Broňa Brejova, Toma svinař*</a></li>
      <li><a href="#systematic-investigation-of-genetic-vulnerabilities-across-cancer-cell-lines-reveals-lineage-specific-dependencies-in-ovarian-cancer--hiu-wing-cheunga-glenn-s-cowleyb-barbara-a-weirb-jesse-s-boehmb-scott-rusinb-justine-a-scottb-alexandra-eastb-levi-d-alib-patrick-h-lizotte">Systematic investigation of genetic vulnerabilities across cancer cell lines reveals lineage-specific dependencies in ovarian cancer – Hiu Wing Cheunga, Glenn S. Cowleyb, Barbara A. Weirb, Jesse S. Boehmb, Scott Rusinb, Justine A. Scottb, Alexandra Eastb, Levi D. Alib, Patrick H. Lizotte…</a></li>
      <li><a href="#systematic-identification-of-genomic-markers-of-drug-sensitivity-in-cancer-cells-mathew-j-garnett-elena-j-edelman-sonja-j-heidorn-chris-d-greenman-anahita-dastur-king-wai-lau-patricia-greninger">Systematic identification of genomic markers of drug sensitivity in cancer cells– Mathew J. Garnett, Elena J. Edelman, Sonja J. Heidorn, Chris D. Greenman, Anahita Dastur, King Wai Lau, Patricia Greninger…</a></li>
      <li><a href="#advancing-the-aging-biology-toolkit-troy-k-coody-and-adam-l-hughes">Advancing the aging biology toolkit– TROY K COODY AND ADAM L HUGHES</a></li>
      <li><a href="#models-of-human-core-transcriptional-regulatory-circuitries-violaine-saint-andré-alexander-j-federation-charles-y-lin">Models of human core transcriptional regulatory circuitries– Violaine Saint-André, Alexander J. Federation, Charles Y. Lin</a></li>
      <li><a href="#pan-cancer-network-analysis-identifies-combinations-of-rare-somatic-mutations-across-pathways-and-protein-complexes-mark-d-m-leiserson-fabio-vandin-hsin-ta-wu-jason-r-dobson">Pan-cancer network analysis identifies combinations of rare somatic mutations across pathways and protein complexes– Mark D M Leiserson, Fabio Vandin, Hsin-Ta Wu, Jason R Dobson</a></li>
      <li><a href="#auto-encoding-variational-bayes--diederik-p-kingma-max-welling">Auto-Encoding Variational Bayes – Diederik P. Kingma, Max Welling</a></li>
      <li><a href="#towards-systematic-functional-characterization-of-cancer-genome-jesse-s-boehm-and-william-c-hahn-broadinstitute">Towards systematic functional characterization of cancer genome– Jesse S. Boehm, and William C. Hahn, BroadInstitute</a></li>
      <li><a href="#a-combination-cdk46-and-igf1r-inhibitor-strategy-for-ewing-sarcoma--lillian-m-guenther-neekesh-v-dharia-linda-ross-amy-conway1-amanda-l-robichaud">A Combination CDK4/6 and IGF1R Inhibitor Strategy for Ewing Sarcoma – Lillian M. Guenther, Neekesh V. Dharia, Linda Ross, Amy Conway1, Amanda L. Robichaud…</a></li>
      <li><a href="#core-transcriptional-regulatory-circuit-controlled-by-the-tal1-complex-in-human-t-cell-acute-lymphoblastic-leukemia-takaomi-sanda-lee-n-lawton-m-inmaculada-barrasa-peng-fan-holger-kohlhamme">Core Transcriptional Regulatory Circuit Controlled by the TAL1 Complex in Human T Cell Acute Lymphoblastic Leukemia– Takaomi Sanda, Lee N. Lawton, M. Inmaculada Barrasa, Peng Fan, Holger Kohlhamme…</a></li>
      <li><a href="#v-net-fully-convolutional-neural-networks-for-volumetric-medical-image-segmentation-fausto-milletari1-nassir-navab-seyed-ahmad-ahmadi">V-Net: Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation– Fausto Milletari1, Nassir Navab, Seyed-Ahmad Ahmadi</a></li>
      <li><a href="#crispr-cas9-screen-reveals-a-mycn-amplified-neuroblastoma-dependency-on-ezh2-liying-chen-gabriela-alexe-neekesh-v-dharia-linda-ross-amanda-balboni-ini">CRISPR-Cas9 screen reveals a MYCN-amplified neuroblastoma dependency on EZH2– Liying Chen, Gabriela Alexe, Neekesh V. Dharia, Linda Ross, Amanda Balboni Ini…</a></li>
      <li><a href="#the-variability-of-expression-of-many-genes-and-most-functional-pathways-is-observed-to-increase-with-age-in-brain-transcriptome-data-veronika-r-kedlian-handan-melike-donertas-janet-m-thornton">The variability of expression of many genes and most functional pathways is observed to increase with age in brain transcriptome data.– Veronika R. Kedlian, Handan Melike Donertas, Janet M. Thornton</a></li>
      <li><a href="#ewsfli-confers-tumor-cell-synthetic-lethality-to-cdk12-inhibition-in-ewing-sarcoma-amanda-balboni-iniguez-bjorn-stolte-emily-jue-wang-">EWS/FLI Confers Tumor Cell Synthetic Lethality to CDK12 Inhibition in Ewing Sarcoma– Amanda Balboni Iniguez, Bjorn Stolte, Emily Jue Wang, …</a></li>
      <li><a href="#cell-detection-in-microscopy-images-with-deep-convolutional-neural-network-and-compressed-sensing-yao-xue-and-nilanjan-ray">Cell Detection in Microscopy Images with Deep Convolutional Neural Network and Compressed Sensing– Yao Xue and Nilanjan Ray</a></li>
      <li><a href="#cosmic-exploring-the-worlds-knowledge-of-somatic-mutations-in-human-cancer-simon-a-forbes-david-beare-prasad-gunasekaran-kenric-leung-nidhi-bindal-harry-boutselakis-minjie-ding">COSMIC: exploring the world’s knowledge of somatic mutations in human cancer– Simon A. Forbes, David Beare, Prasad Gunasekaran, Kenric Leung, Nidhi Bindal, Harry Boutselakis, Minjie Ding…</a></li>
      <li><a href="#impact-genomic-annotation-of-cell-state-specific-regulatory-elements-inferred-from-the-epigenome-of-bound-transcription-factors-tiffany-amariuta-yang-luo-steven-gazal-emma-e-davenport">IMPACT: Genomic Annotation of Cell-State-Specific Regulatory Elements Inferred from the Epigenome of Bound Transcription Factors– Tiffany Amariuta, Yang Luo, Steven Gazal, Emma E. Davenport…</a></li>
      <li><a href="#extensive-variation-in-chromatin-states-across-humans-maya-kasowski-sofia-kyriazopoulou-panagiotopoulou-fabian-grubert-judith-b-zaugganshul-kundaje-yuling-liu-alan-p-boyle-qiangfeng-cliff-zhang-fouad-zakharia-damek-v-spacek-jingjing-li-dan-xie-anthony-olarerin-george-lars-m-steinmetz-john-b-hogenesch-manolis-kellis-serafim-batzoglou">Extensive Variation in Chromatin States Across Humans– Maya Kasowski, Sofia Kyriazopoulou-Panagiotopoulou, Fabian Grubert, Judith B. Zaugg,Anshul Kundaje, Yuling Liu, Alan P. Boyle, Qiangfeng Cliff Zhang, Fouad Zakharia, Damek V. Spacek, Jingjing Li, Dan Xie, Anthony Olarerin-George, Lars M. Steinmetz, John B. Hogenesch, Manolis Kellis, Serafim Batzoglou</a></li>
      <li><a href="#chromhmm-automating-chromatin--state-discovery-and-characterization-jason-ernst-manolis-kellis">ChromHMM: automating chromatin- state discovery and characterization– Jason Ernst, Manolis Kellis</a></li>
      <li><a href="#histone-modifications-at-human-enhancers-reflect-global-cell-type-specific-gene-expression-nathaniel-d-heintzman-gary-c-hon-r-david-hawkins1-pouya-kheradpour-alexander-stark-manolis-kellis">Histone modifications at human enhancers reflect global cell-type-specific gene expression– Nathaniel D. Heintzman, Gary C. Hon, R. David Hawkins1, Pouya Kheradpour, Alexander Stark, Manolis Kellis…</a></li>
      <li><a href="#strelka2-fast-and-accurate-calling-of-germline-and-somatic-variants-sangtae-kim-konrad-scheffler-aaron-l-halpern-mitchell-a-bekritsky">Strelka2: fast and accurate calling of germline and somatic variants– Sangtae Kim, Konrad Scheffler, Aaron L. Halpern, Mitchell A. Bekritsky</a></li>
      <li><a href="#star-ultrafast-universal-rna-seq-alignerpacbio">STAR: ultrafast universal RNA-seq aligner–PacBio</a></li>
      <li><a href="#star-fusion-fast-and-accurate-fusion-transcript-detection-from-rna-seq-brian-j-haas1-alex-dobin-nicolas-stransky-bo-li-aviv-regev">STAR-Fusion: Fast and Accurate Fusion Transcript Detection from RNA-Seq– Brian J. Haas1, Alex Dobin, Nicolas Stransky, Bo Li, Aviv Regev…</a></li>
      <li><a href="#title-exploring-genetic-interaction-manifolds-constructed-from-rich-phenotypes--thomas-m-norman-max-a-horlbeck-joseph-m-replogle">Title: Exploring genetic interaction manifolds constructed from rich phenotypes – Thomas M. Norman, Max A. Horlbeck, Joseph M. Replogle</a></li>
      <li><a href="#rsem-accurate-transcript-quantification-from-rna-seq-data-with-or-without-a-reference-genome-bo-li-colin-dewey">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome– Bo Li, Colin Dewey</a></li>
      <li><a href="#sensitive-detection-of-somatic-point-mutations-in-impure-and-heterogeneous-cancer-samples">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples</a></li>
      <li><a href="#ngsea-network-based-gene-set-enrichment-analysis-for-interpreting-gene-expression-phenotypes-with-functional-gene-sets-heonjong-han-sangyoung-lee-and-insuk-lee">NGSEA: network-based gene set enrichment analysis for interpreting gene expression phenotypes with functional gene sets– Heonjong Han, Sangyoung Lee, and Insuk Lee</a></li>
      <li><a href="#reverse-engineering-cellular-networks-califano-lab-aracne">Reverse engineering cellular networks– Califano Lab (ARAcne)</a></li>
      <li><a href="#limma-powers-differential-expression-analyses-for-rna-sequencing-and-microarray-studies-matthew-e-ritchie-belinda-phipson-di-wu">limma powers differential expression analyses for RNA-sequencing and microarray studies– Matthew E. Ritchie, Belinda Phipson, Di Wu…</a></li>
      <li><a href="#an-oncogenic-super-enhancer-formed-through-somatic-mutation-of-a-noncoding-intergenic-element-marc-r-mansour-brian-j-abraham-lars-anders-alla-berezovskaya-alejandro-gutierrez-adam-d-durbin">An oncogenic super-enhancer formed through somatic mutation of a noncoding intergenic element– Marc R. Mansour, Brian J. Abraham, Lars Anders, Alla Berezovskaya, Alejandro Gutierrez, Adam D. Durbin</a></li>
      <li><a href="#aging-is-associated-with-a-systemic-length-driven-transcriptome-imbalance">Aging is associated with a systemic length-driven transcriptome imbalance</a></li>
      <li><a href="#predicting-splicing-from-primary-sequence-with-deep-learning-kishore-jaganathan-sofia-kyriazopoulou-panagiotopoulou-jeremy-f-mcrae-siavash-fazel-darbandi-david-knowles-yang-i-li-jack-a-kosmicki-juan-arbelaez-wenwu-cui-grace-b-schwartz-eric-d-chow-efstathios-kanterakis-hong-gao-amirali-kia-serafim-batzoglou-stephan-j-sanders-and-kyle-kai-how-farh">Predicting Splicing from Primary Sequence with Deep Learning– Kishore Jaganathan, Sofia Kyriazopoulou Panagiotopoulou, Jeremy F. McRae, Siavash Fazel Darbandi, David Knowles, Yang I. Li, Jack A. Kosmicki, Juan Arbelaez, Wenwu Cui, Grace B. Schwartz, Eric D. Chow, Efstathios Kanterakis, Hong Gao, Amirali Kia, Serafim Batzoglou, Stephan J. Sanders, and Kyle Kai-How Farh</a></li>
      <li><a href="#network-based-inference-of-protein-activity-helps-functionalize-the-genetic-landscape-of-cancer-mariano-j-alvarez-yao-shen-federico-m-giorgi-califano-lab">Network-based inference of protein activity helps functionalize the genetic landscape of cancer– Mariano J. Alvarez, Yao Shen, Federico M. Giorgi, Califano Lab…</a></li>
      <li><a href="#three-dimensional-total-internal-reflection-fluorescence-nanoscopy-with-sub-10-nm-resolution">Three-dimensional total internal reflection fluorescence nanoscopy with sub-10 nm resolution</a></li>
      <li><a href="#inferring-protein-3d-structure-from-deep-mutation-scans-nathan-j-rollins15-kelly-p-brock125-frank-j-poelwijk">Inferring protein 3D structure from deep mutation scans– Nathan J. Rollins1,5, Kelly P. Brock1,2,5, Frank J. Poelwijk</a></li>
      <li><a href="#deep-learning-at-base-resolution-reveals-motif-syntax-of-the-cis-regulatory-code-kundaje-lab">Deep learning at base-resolution reveals motif syntax of the cis-regulatory code– Kundaje lab</a></li>
      <li><a href="#hallmarks-of-cancer-the-next-generation-douglas-hanahan12-and-robert-a-weinberg3">Hallmarks of Cancer: The Next Generation– Douglas Hanahan1,2,* and Robert A. Weinberg3,*</a></li>
      <li><a href="#next-generation-characterization-of-the-cancer-cell-line-encyclopedia-cds">Next-generation characterization of the Cancer Cell Line Encyclopedia– CDS</a></li>
      <li><a href="#off-target-toxicity-is-a-common-mechanism-of-action-of-cancer-drugs-undergoing-clinical-trials-ann-lin-cds-jc">Off-target toxicity is a common mechanism of action of cancer drugs undergoing clinical trials– Ann Lin @CDS JC</a></li>
      <li><a href="#scalable-whole-exome-sequencing-of-cell-free-dna-reveals-high-concordance-with-metastatic-tumors-viktor-adalsteinsson-gavin-ha-samuel-freeman-atish-choudhury">Scalable whole-exome sequencing of cell-free DNA reveals high concordance with metastatic tumors– Viktor Adalsteinsson, Gavin Ha, Samuel Freeman, Atish Choudhury</a></li>
      <li><a href="#organization-and-regulation-of-gene-transcription-patrick-cramer">Organization and regulation of gene transcription– Patrick Cramer</a></li>
      <li><a href="#dynamic-regulation-of-transcriptional-states-by-chromatin-and-transcription-factors-ty-c-voss-and-gordon-l-hager">Dynamic regulation of transcriptional states by chromatin and transcription factors– Ty C. Voss and Gordon L. Hager</a></li>
      <li><a href="#liquid-versus-tissue-biopsy-for-detecting-acquired-resistance-and-tumor-heterogeneity-in-gastrointestinal-cancers-aparna-r-parikh-ignaty-leshchiner-liudmila-elagina">Liquid versus tissue biopsy for detecting acquired resistance and tumor heterogeneity in gastrointestinal cancers– Aparna R. Parikh, Ignaty Leshchiner, Liudmila Elagina</a></li>
      <li><a href="#activity-by-contact-model-of-enhancer-specificity-from-thousands-of-crispr-perturbations-fulco-lander">Activity-by-Contact model of enhancer specificity from thousands of CRISPR perturbations– Fulco, Lander..</a></li>
      <li><a href="#passenger-hotspot-mutations-in-cancer-julian-m-hess-andre-bernards-jaegil-kim">Passenger Hotspot Mutations in Cancer– Julian M. Hess, Andre Bernards, Jaegil Kim</a></li>
      <li><a href="#scaling-computational-genomics-to-millions-of-individuals-with-gpus-amaro-taylor-weiner-françois-aguet">Scaling computational genomics to millions of individuals with GPUs– Amaro Taylor-Weiner, François Aguet</a></li>
      <li><a href="#a-genetics-led-approach-defines-the-drug-target--landscape-of-30-immune-related-traits-hai-fang-the-ultra-dd-consortium">A genetics-led approach defines the drug target  landscape of 30 immune-related traits– Hai Fang, The ULTRA-DD Consortium</a></li>
      <li><a href="#structure-based-function-prediction-using-graph-convolutional-networks-ian-m-fisk-rick-bonneau">Structure-Based Function Prediction using Graph Convolutional Networks– Ian M Fisk, Rick Bonneau</a></li>
      <li><a href="#visualization-and-analysis-of-single-cell-rna-seq-data-by-kernel--based-similarity-learning-junjie-zhu-serafim-batzoglou">Visualization and analysis of single-cell RNA-seq data by kernel- based similarity learning– Junjie Zhu, Serafim Batzoglou</a></li>
      <li><a href="#umap-uniform-manifold-approximation-and-projection-for-dimension-reduction">UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction–</a></li>
      <li><a href="#reproducible-pharmacogenomic-profiling-of-cancer-cell-line-panels-peter-m-haverty-eva-lin-jenille-tan2-yihong-yu">Reproducible pharmacogenomic profiling of cancer cell line panels– Peter m. Haverty, Eva Lin, Jenille Tan2, yihong yu…</a></li>
      <li><a href="#ezh2-oncogenic-mutations-drive-epigenetic-transcriptional-and-structural-changes-within-chromatin-domains-maria-c-donaldson-collier15-stephanie-sungalee-epfl">EZH2 oncogenic mutations drive epigenetic, transcriptional, and structural changes within chromatin domains– Maria C. Donaldson-Collier1,5, Stephanie Sungalee, EPFL</a></li>
      <li><a href="#master-transcription-factors-and-mediator-establish-super-enhancers-at-key-cell-identity-genes-young-lab">Master Transcription Factors and Mediator Establish Super-Enhancers at Key Cell Identity Genes– Young Lab</a></li>
      <li><a href="#diggit-a-bioconductor-package-to-infer-genetic-variants-driving-cellular-phenotype--mariano-j-alvarez-james-c-chen-and-andrea-califano">DIGGIT: a Bioconductor package to infer genetic variants driving cellular phenotype – Mariano J. Alvarez, James C. Chen and Andrea Califano</a></li>
      <li><a href="#moderated-estimation-of-fold-change-and-dispersion-for-rna-seq-data-with-deseq2--michael-i-love-wolfgang-huber-and-simon-anders">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2 – Michael I Love, Wolfgang Huber and Simon Anders</a></li>
      <li><a href="#tads-as-the-caller-calls-them-elzo-de-wit">TADs as the caller calls them– Elzo de Wit</a></li>
      <li><a href="#understanding-and-improving-interpolation-in-autoencoders-via-an-adversarial-regularizer-david-berthelot-ian-goodfellow">UNDERSTANDING AND IMPROVING INTERPOLATION IN AUTOENCODERS VIA AN ADVERSARIAL REGULARIZER– David Berthelot, Ian Goodfellow…</a></li>
      <li><a href="#transcriptional-amplification-in-tumor-cells-with-elevated-c-myc-charles-lin-young-lab">Transcriptional Amplification in Tumor Cells with Elevated c-Myc– Charles Lin, Young Lab</a></li>
      <li><a href="#slam-seq-defines-direct-gene-regulatory-functions-of-the-brd4-myc-axis-matthias-muhar-anja-ebert">SLAM-seq defines direct gene-regulatory functions of the BRD4-MYC axis– Matthias Muhar, Anja Ebert</a></li>
      <li><a href="#transcription-factors-activate-genes-through-the-phase-separation-capacity-of-their-activation-domains-ann-boija-isaac-a-klein-benjamin-r-sabari-youngs-lab">Transcription Factors Activate Genes through the Phase-Separation Capacity of Their Activation Domains– Ann Boija, Isaac A. Klein, Benjamin R. Sabari, Young’s lab</a></li>
      <li><a href="#a-blueprint-for-advancing-genetics-based-cancer-therapy--bill-sellers">A Blueprint for Advancing Genetics-Based Cancer Therapy – Bill Sellers</a></li>
      <li><a href="#genome-wide-somatic-variant-calling-via-colored-debrujin-graphs--giuseppe-narzisi-1-andré-corvelo1-kanika-arora">Genome-wide somatic variant calling via colored debrujin graphs- Giuseppe Narzisi 1, André Corvelo1, Kanika Arora</a></li>
      <li><a href="#atomnet-a-deep-convolutional-neural-network-for-bioactivity-prediction-in-structure-based-drug-discovery-atomwise-team">AtomNet: A Deep Convolutional Neural Network for Bioactivity Prediction in Structure-based Drug Discovery– atomwise team</a></li>
      <li><a href="#evaluating-variant-calling-tools-for-non-matched-next-generation-sequencing-data">Evaluating Variant Calling Tools for Non-Matched Next-Generation Sequencing Data–</a></li>
      <li><a href="#identifying-gene-expression-programs-of-cell-type-identity-and-cellular-activity-with-single-cell-rna-seq-dylan-kotliar-adrian-veres-m-aurel-nagy">Identifying Gene Expression Programs of Cell-type Identity and Cellular Activity with Single-Cell RNA-Seq– Dylan Kotliar, Adrian Veres, M. Aurel Nagy</a></li>
      <li><a href="#deepcpg-accurate-prediction-of-single-cell-dna-methylation-states-using-deep-learning-christof-angermueller1-heather-j-lee-wolf-reik">DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning– Christof Angermueller1, Heather J. Lee, Wolf Reik</a></li>
      <li><a href="#lighcpg-a-multi-view-cpg-sites-detection-on-single-cell-whole-genome-sequence-data">LighCpG: a multi-view CpG sites detection on single-cell whole genome sequence data</a></li>
      <li><a href="#massively-multiplex-chemical-transcriptomics-at-single-cell-resolution-sanjay-r-srivatsan-josé-l-mcfaline-figueroa">Massively multiplex chemical transcriptomics at single-cell resolution– Sanjay R. Srivatsan, José L. McFaline-Figueroa</a></li>
      <li><a href="#improved-protein-structure-prediction-using-potentials-from-deep-learning-deepmind">Improved protein structure prediction using potentials from deep learning– DeepMind</a></li>
      <li><a href="#pan-cancer-analysis-of-whole-genome-broad-toronto">PAN CANCER ANALYSIS OF WHOLE GENOME: Broad, Toronto,</a>
        <ul>
          <li><a href="#a-deep-learning-system-can-accurately-classify-primary-and-metastatic-cancers-based">A deep learning system can accurately classify primary and metastatic cancers based</a></li>
        </ul>
      </li>
      <li><a href="#growth-differentiation-factor-11-is-a-circulating-factor-that-reverses-age-related-cardiac-hypertrophy-francesco-s-loffredo-matthew-l-steinhauser-steven-m-jay">Growth Differentiation Factor 11 Is a Circulating Factor that Reverses Age-Related Cardiac Hypertrophy– Francesco S. Loffredo, Matthew L. Steinhauser, Steven M. Jay</a></li>
      <li><a href="#the-hallmarks-of-aging-carlos-lopez-otı-maria-a-blasco-linda-partridge">The Hallmarks of Aging– Carlos Lopez-Otı Maria A. Blasco, Linda Partridge</a></li>
      <li><a href="#predicting-deeper-into-the-future-of-semantic-segmentation">Predicting Deeper into the Future of Semantic Segmentation–</a></li>
      <li><a href="#federated-learning-strategies-for-improving-communication-efficiency-jakub-konecˇny-h-brendan-mcmahan-google">FEDERATED LEARNING: STRATEGIES FOR IMPROVING COMMUNICATION EFFICIENCY– Jakub Konecˇny, H. Brendan McMahan, Google</a></li>
    </ul>
  </li>
</ul>

            </nav>
          </aside>
        
        <h1 id="overview">Overview</h1>

<h2 id="summary">SUMMARY</h2>

<p>What is the research problem the paper attempts to address? Consider
	What is the motivation of the research work?
	What problem does it attempt to resolve?
	Is it addressing weaknesses in existing approaches?
	Does it challenge an existing research paradigm?</p>

<p>What are the claimed contributions of the paper? What is new/original in this paper?
	A new question or research area?
	A new understanding of a research problem?
	A new algorithm/experimental method/proof technique/methodology for solving problems?
	A new breed of software tools or systems?
	A new formalism or notation?
	New evidence to substantiate or disprove a previously published claim?</p>

<p>How do the authors substantiate their claims? What makes the claims scientific?
	What is the methodology adopted to substantiate the claims?
	What is the argument of the paper?
	What are the major theorems?
	What experiments are conducted?  Look at:
	data analyses/simulations/benchmarks/user studies/cases studies/examples</p>

<p>What are the conclusions? 
	What is to be learnt from the paper?
	Will standard practice in the field be changed as result of the findings?
	Are the results generizable?
	Can the results be applied to other areas of the field?
	What are the open problems?</p>

<h2 id="evaluation">EVALUATION</h2>

<p>Is the research problem significant?
	Does the work only address a minor problem?§
	Does it deepen understanding?
	Does it explore new design ideas?</p>

<p>Are the contributions significant?
	Is the paper worth reading?
	Are the authors simply repeating the state of the art?
	Are there real surprises?
	Are the authors aware of the relation of their work to the existing literature?
	Is the paper addressing a well-known open problem?</p>

<p>Are the claims valid (and modest enough)?
	have the authors (intentionally or unintentionally) cut corners?
	Has the right theorem been proven?</p>

<h2 id="synthesis">SYNTHESIS</h2>

<p>What is the crux of the research problem?
What are some alternative approaches to address the problem?
What is a better way to substantiate the claim of the authors?
What is a good argument against the case made by the authors?
How can the research results be improved?
Can the research results be applied to another context?
What are the open problems raised by this work?
Can we do better than the authors?</p>

<h2 id="7-things-that-would-make-papers-better-after-156-papers">7 things that would make papers better (after 156 papers)</h2>

<ol>
  <li>papers are using the same expressions for things that could be said in simpler manner</li>
  <li>papers are displaying citation + p values + data. within text: should be a hoverable link. 
summary should exist as in CELL with a table of content and nice image at the first Page.</li>
  <li>tables should be interactive and secondary material should be within the paper. or readily available from it.</li>
  <li>the paper, its data, its code, its bio methods and everything it cites should be accessible.</li>
  <li>there should be a table for complicated words/acronyms with explanations.</li>
  <li>for other types of info (places/genes/labs/companies there should be links)</li>
  <li>there should always be a corresponding press release/medium article corresponding to it. with simpler explanations.</li>
</ol>

<hr />

<h1 id="papers">Papers</h1>

<h2 id="attention-in-a-bayesian-framework---sahani">Attention in a bayesian framework - sahani</h2>

<p>-&gt; atention as a byesian inference. to improve impoverish stimulus
appearance of a limitation == bottleneck in such framework</p>

<p>shouldn’t you know about P to compute this minimization ? page 7</p>

<p>pb :</p>

<h2 id="neural-coding--peter-latham">neural coding  Peter latham</h2>

<p>neurons send and receive outputs which noise can be correlated meaning that there is in fact information in it. This can have a huge effect when looking at big number of neurons. We have to knwo the correlation to understand the coding as the neurons do</p>

<p>the effect of correlation can be expressed as the variation of I_shuffled ( for encoding) &amp; of I_diag for decoding information adn saying if correlation are important in this process. those values can be from 0 to 40% of the total information of the neurons</p>

<p>ICA ?
independent component analysis. if we admit that a signal is composed of multiple independent sources, each of them emitting an information (thus the corresponding random variable expresses Dependant outcomes) It is possible to each sources ( or each independent components) from the signal according to $y = Wx + o$ with W a matrix (like a Neural net) x the input and o the noise. it is part of unsupervised learning of features</p>

<p>pb : the noise is not gaussian as estimated here.</p>

<h2 id="cracking-the-neural-code-peter-latham">cracking the neural code –peter latham</h2>

<p>The Equivalence of Information-Theoretic and Likelihood-Based Methods for Neural Dimensionality Reduction Manesh sahani</p>

<p>This paper is reviewing many modern information theoretic and stochastic methods to decipher the information encoded in spikes according to stimulus. It expresses concerns and solutions for the MID methods and present some new architectures as well.</p>

<p>pb :</p>

<p>self calibrating NN for dimensionality reduction Chklovskii</p>

<p>A NN that is calibrating itself to its input and reduce the dimensionality the best way possible according to minimization rules. present online and offline with rules and reguralizers which creates update rules that can be attributed to hebbian &amp; anti hebbian behavior in neurons.</p>

<p>pb : not really biologically plausible</p>

<h2 id="alex-graves-talk-at-ijcnn">Alex graves talk at IJCNN</h2>

<p>can use ActiveComputationTimeNN to know where are the important part of the data 
DifferentiableNeuralComputer? NTM</p>

<p>DecoupledNeuralInterface to stop using backprop ( as efficient)</p>

<h2 id="neural-turing-machine-graves">neural turing machine Graves</h2>

<p>where does the vector emission of the heads comes from ? 
the NN
for examples the shifts are produced by a softmax layer applied to the output of the controller 
softmax : mise a l’expo de chaques val du vect avec normalization sur le vector ( sum =1)</p>

<p>cross entropy : objective function that uses entropy function of two stat distribution and the log loss function</p>

<p>objective function :  fonction d’apprentissage that needs to be minimized ( it gets the output and the hoped output)</p>

<p>Evolving Neural Networks through Augmenting Topologies - Kenneth O stanley</p>

<p>really nice paper, explained well, but few complexity though.</p>

<p>What if this evolving architecture leads you to train your growing NN over simplification of your data and it gets more and more complex (for images for example) and you could have different types of neurons with different acti function …</p>

<h2 id="predicting-the-sequence-specificities-of-dna--and-rna-binding-proteins-by-deep-learning-brendan-frey">Predicting the sequence specificities of DnA- and RnA-binding proteins by deep learning Brendan Frey</h2>

<p>k-mer ?
all the possible substrings of length k that are contained in a string</p>

<p>mini-batch  ?   stochastic gradient descent ?
In gradient descent algorithms, you can calculate the sum of gradients with respect to several examples and then update the parameters using this cumulative gradient. 
If you ‘see’ all training examples before one ‘update’, then it’s called full batch learning. 
If you use only one example, it’s called stochastic gradient descent (online learning). 
If you use a small batch of examples, it’s called mini-batch learning.</p>

<p>nesterov momentum ?
a way o compute how we move in our gradient descent, that is, using a momentum and in addition we give this an acceleration in the way we computer it which is called accelerated momentum (it seems to converge quicker)</p>

<p>hyperparameter search ?
a way to find the best set of parameters for a ML algo either by doing them all or by inferring a function from them if such exist.</p>

<p>alternative splicing ?
when a gene encodes many proteins trough specific selection during the RNA “epissage” == “splicing”</p>

<p>metazoa ?
eucaryotes with different tissue types</p>

<p>sequence specificity protein ?
proteins that binds inside the holes in coiled DNA or RNA in specific places regulating their transcription in many ways</p>

<h2 id="deep-learning-free-text-and-sentence-embedding--sanjeev-arora">Deep-learning-free Text and Sentence Embedding  –Sanjeev Arora</h2>
<p><a href="http://www.offconvex.org/2018/06/17/textembeddings/">link</a></p>

<p>like word2vec but for phrases</p>

<ul>
  <li>you weight each word by its inverse smoothed frequency</li>
  <li>you sum the weighted vectors</li>
  <li>you remove the component that is present in any possible phrases (corresponds to points etc..)</li>
</ul>

<p>to produce phrases you have a discourse vector representing the subject
and you have a proba function of a vector appearing at pos t given discourse vector. ct moves as a random walk</p>

<h2 id="how-do-we-capture-structure-in-relational-data-sanjeev">How do we capture structure in relational data –Sanjeev</h2>

<p>we want to have a representation of the vertices (points) that is similar
when vertices pertain to a similar neighborhood do this by learning a representation that allows us to guess what will be the next given the preceding in a random walk</p>

<p>with node2vec there is 2 ways to look at neighborhoods (homophily or structural) and you can interpolate between the two with 2 hyper params</p>

<h2 id="cancer-metastasis-detection-with-neural-conditional-random-field-yi-li">Cancer Metastasis Detection With Neural Conditional Random Field –Yi Li</h2>

<p>we want to use an information of the neighborhood of images the we find by computing an additional term on the inference of each elements according to their similarity/ distance to others and their labels. this creates a differentiable loss function (Conditional Random Field) that allows for fast computations etc. 80% success</p>

<h2 id="systematic-identification-of-phosphorylation--mediated-protein-interaction-switches">Systematic identification of phosphorylation- mediated protein interaction switches</h2>

<p>phosphorilation plays a regulatory role in prot-prot interaction by modifying the efficiency of functional domains.</p>

<p>from binding, enzymatic functions …</p>

<p>(only using simple probabilistic methods given the presence of such amino acids and other chemical parameters..) -not well explained thought</p>

<p>there is much to do here but needs even more data and need clear characterization of the multiple effects of the folding in the chemical processes of phospho switches</p>

<h2 id="a-polymorphism-in-the-tumor-suppressor-p53-affects-aging-and-longevity-in-mouse-models">A polymorphism in the tumor suppressor p53 affects aging and longevity in mouse models</h2>

<p>p53 if polymorph increase cancer probability but also increases life expectancy by preventing the knowing off of genes useful in cell replication and decreases stem cell exhaustion.</p>

<h2 id="gene2vector">gene2vector</h2>

<p>really just applying the word2vect algorithm on protein sequences and inferring some interesting results like the destructuring or presence of proteins</p>

<h2 id="multi-omics-factor-analysis---a-framework-for-unsupervised-integration-of-multi-omic-data-sets--ricard-argelaguet-et-al">Multi-Omics factor analysis - a framework for unsupervised integration of multi-omic data sets – Ricard Argelaguet et al.</h2>

<p>you integrate each data by doing a PCA on them basically.
but it uses a Latent factor model which is formulated in a bayesian way by Bayesian way. it i able to recover sparse features which explain well the underlying variations in the data and are able to thus predict outcomes in clinical data and importance features in pluripotent cell differentiation
NICE.</p>

<p>It is super efficient computationally (compared to similar methods)</p>

<h2 id="basset-learning-the-regulatory-code-of-the-accessible-genome-with-deep-convolutional-neural-networks--david-kelley">Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks – David Kelley</h2>

<p>you can learn the functional activity of ncDNA with a CNN and DNA seq (==FAIRE seq) data (ground truth here)</p>

<blockquote>
  <p>understanding the DNA–protein interactions as a function of the un- derlying sequence.</p>
</blockquote>

<p>Given training data, models parameterized by machine learning can effectively predict protein binding, DNA accessibility, histone modifications, and DNA methylation from the sequence</p>

<p>provide much information on relationship to diseases and allow fast training by reusing the weights to train on other datasets the first CNN weights are predictive of known binding configurations as well as unknown ones. (should be investigated)</p>

<h2 id="machine-theory-of-mind">Machine theory of Mind</h2>

<p>How to interpret and predict another agent’s thinking according to its behavior.</p>
<blockquote>
  <p>This “learning to learn” about new agents is what we mean by meta-learning</p>
</blockquote>

<p>principal characteristics is that the network can learn that the agent hold false belief about their environment</p>

<h2 id="analysis-of-genomic-sequence-motifs-for-deciphering-transcription-factor-binding-and-transcriptional-regulation-in-eukaryotic-cells-valentina-boeva">Analysis of Genomic Sequence Motifs for Deciphering Transcription Factor Binding and Transcriptional Regulation in Eukaryotic Cells –Valentina Boeva</h2>

<blockquote>
  <p>A eukaryotic genome contains a variety of structured patterns. A far from exhaustive list of genomic patterns includes (i) tandem repeats and transposable elements, (ii) stretches of GC- or AT-rich sequences (e.g., CpG islands in mammalian genomes), (iii) binding sites of DNA associated proteins (e.g., transcription factor binding sites), (iv) splice sites, and (v) DNA and RNA binding sites of non- coding RNA molecules. Different patterns may overlap each other. Therefore, although this review is focused on motifs for transcription factor binding sites (TFBSs), we provide a short overview of other types of genomic patterns.</p>
</blockquote>

<p>This is an overview of the different Trans Fact Bind Sites TFBS and the Motifs Repeats, AT-GC rich sequences, splice sites, miRNA binding sites, techniques and DB to use, find, retrieve known or unknownTFBS.</p>

<p><em>contains DB links to unknown DB of PWM motifs for TFBS</em></p>

<p>discovery of Shadow binding sites that are low affinity binding sites that alone are not capable of retaining the TF long enough to ensure activation/repression, but instead are used to maintain a high concentration of TF in the vicinity of the primary binding sites</p>

<h2 id="gpu-kernels-for-block-sparse-weights-scott-gray-openai">GPU Kernels for Block-Sparse Weights –Scott Gray, openAI</h2>

<p>increasing drastically computation speed on NN with blocks of zero valued weights, allowing to deepen or widen dense NN for same computational efficiency and better accuracy.</p>

<p>They use LSTMs with internal networks and connects them according to the small world paradigm (very good results)</p>

<p>–&gt; would be nice to learn the sparsity structure
according to a predefined sparsity structure, a small world and block sparse  property</p>

<p>Available on tensorflow ops.
(best use multiples of 32x32 and small float values 16 bits etc..
best minibatch is 64 (for cache dilution)</p>

<h2 id="node2vec-scalable-feature-learning-for-networks--aditya-grover-stanford">node2vec: Scalable Feature Learning for Networks – Aditya Grover, stanford</h2>

<p>How to embed a node from a network using a flexible notion of neighborhood (from random walks)</p>

<p>able to classify predict … from this representation, which involves just creating an encoding matrix that is learnt by asyncSGD by trying to predict the neighborhood given the representation</p>

<p><em>questions</em></p>

<p>What if we set out to encode not solely the local neighborhood but also some information of the distant neighborhood considering the prior that the network has the small world property?</p>

<h2 id="microrna-signature-analysis-in-colorectal-cancer-identification-of-expression-profiles-in-stage-ii-tumors-associated-with-aggressive-disease-kah-hoong-chang">MicroRNA signature analysis in colorectal cancer: identification of expression profiles in stage II tumors associated with aggressive disease –Kah Hoong Chang</h2>

<p><em>touse</em></p>

<p>Very High quality paper with a demonstration that 3 circulating miRNA
Can provide information on the colorectal cancer and its reaction to different treatment strategies, for personalized curration of the cancer, this association amongst more than 400 different miRNA (using micro array + microfluidics PCR)  was performed using an ANN to learn how to classify patients</p>

<h2 id="efficient-and-robust-automated-machine-learning-matthias-feurer-freiburg-university">Efficient and Robust Automated Machine Learning –Matthias Feurer, Freiburg University</h2>

<p>they build on the work of auto weka to solve the CASH optimization (hyperparams and model) problem, to create state of the art algo with sota performance using a tree based Bayesian method (random forest based) called SMAC. They improve this model by having a first instantiation step using meta learning: they have a model that have learnt what are the best instantiation for a given dataset using meta feature of the dataset such as entropy, number of points, classes etc… .
and secondly they use an ensembl of their best algo and not just one</p>

<h2 id="vicus-exploiting-local-structures-to-improve-network-based-analysis-of-biological-data--bo-wang-serafim-batzoglou">Vicus: Exploiting local structures to improve network-based analysis of biological data – Bo Wang, Serafim Batzoglou</h2>

<p><em>touse</em></p>

<p>They propose a new method to the manifold learning problem (dim reduction, clustering, classification etc..) Which uses local dimension learning to do its magic instead of the usual laplacian eigenvectors which learn comparing global euclidean distances.</p>

<p>It is basically the same thing as the laplacian but only it is constrained to do its computation on a local subgraph of close by datapoints</p>

<p><em>contains DB links to unknown DB of single cell RNA (12-15, 24)</em></p>

<h2 id="active-learning-of-cortical-connectivity-from-two-photon-imaging-data-mart-́ın-a-bertra-́n-duke-uni">Active learning of cortical connectivity from two-photon imaging data –Mart ́ın A. Bertra ́n, Duke Uni</h2>

<p>using bi photon imaging to look at a set of neuron activities and inferring some networks of connections given some stimulus using active learning. 
Basically, what edge should be added to the network that would maximize the likelihood of the model. (edges are weighted)
They model it with a GLM trying to optimize the BIC value of the model. to find whether promising edges to be added can be, it tries to drive the spiking rate of the model up.</p>

<h2 id="finding-syntax-in-human-encephalography-with-beam-search-john-hale-oxf-deepmind">Finding Syntax in Human Encephalography with Beam Search –John Hale, Oxf, DeepMind</h2>

<blockquote>
  <p>a pattern of results recommends the RNNG+beam search combination as a mechanistic model of the syntactic processing that occurs during normal human language comprehension</p>
</blockquote>

<p>RNNG are generative models (here for text) that rely on RNN. they can create a sentence or form a grammar tree of words given a sentence, a tree that you can parse within using beam search this parsing gives you information such as word surprisal and you can use these different statistics given the tree to generate different phrases that will create a predictable effect on the human brain meaning the way their syntax is generated is predictable of the human language comprehension.</p>

<p>The guys basically made a point in cognitive neuroscience using ANN and advanced computer science. <strong>MADNESS</strong></p>

<h2 id="glow-flow-based-generative-network-openai">GLOW: FLow based generative network –OpenAI</h2>

<p>It is a particular kind of generative NN that is fully reversible thus encode^-1 == decode it is like an auto-encoder but without the re parametrization trick and with a latent space that is the same size as the input space. in the latent space you just have a multi dim Gaussian you can then interpolate to generate new inputs. 
(costly to train though)</p>

<h2 id="prediction-of-acute-myeloid-leukemia-risk-in-healthy-individuals-eran-segal-moritz-gerstung">Prediction of acute myeloid leukemia risk in healthy individuals –Eran Segal, Moritz Gerstung</h2>

<p>age-related clonal haematopoiesis is a phenomenon is a phenomenon that exists during aging but also is an indicator of AML 7yr before getting it (leukemia form). Here they try to differentiate between the two using deep sequencing methods they find a group of SNPs and markers in parts of the sequences that explains together much of the data but not enough separately. they then did the same on regular blood samples of patients and have seen that information on the size of hematopoetic cells (blood) have a correlation with age related diseases but also pre AML conditions. thus enabling a first easy screening of risky patients.</p>

<h2 id="a-synthetic-diploid-benchmark-for-accurate-variant-calling-evaluation--mc-arthur">A synthetic-diploid benchmark for accurate variant-calling evaluation – Mc Arthur</h2>

<p>trying to create a new benchmark for variant calling tools (matching reads to a reference genome and inferring the SNPs) this is faster than de novo WGS which is super costly and long. (this is part of NGS) they use a particular type of cell lines (human) called CHM which are diploids but fully homozygous and allows to use PACbio sequences without mixing the two genomes</p>

<h2 id="meta-analysis-of-the-diagnostic-and-clinical-utility-of-genome-and-exome-sequencing-and-chromosomal-microarray-in-children-with-suspected-genetic-diseases--michelle-clark">Meta-analysis of the diagnostic and clinical utility of genome and exome sequencing and chromosomal microarray in children with suspected genetic diseases – Michelle Clark</h2>

<p>A literature and statistical review of 10 years of genomics medicine for infant genetic diseases diagnostics. better use WES/WGS than previous method called CMA (molecular analysis) better sequence infant and 2 parents as well than jsut infant. a lot of heterogeneity and thus. take stats in this study with a grain of salt.</p>

<h2 id="multi-scale-deep-tensor-factorization-learns-a-latent-representation-of-the-human-epigenome--jacob-schreiber-uw">Multi-scale deep tensor factorization learns a latent representation of the human epigenome – Jacob Schreiber UW</h2>

<p><em>touse</em></p>

<p>let’s create an embedding of all epigenomics data. “Avocado” 
<em>quality article</em></p>
<blockquote>
  <p>o high-throughput assay is perfectly reproducible, and run-to-run differences in the same experiment may reflect either biological variation in the cells being assayed or experimental variance arising from sample preparation or downstream steps in the protocol. Finally, many epigenomic assays are highly redundant with one another, and many cell types are closely related to each other, leading to highly redundant measurements</p>
</blockquote>

<p>–&gt; so Embeddings
factorize an incomplete 3D matrix of epigenetic features into 3 2D matx of different feature types (pos, assays, cells), concatenating features for each pos on the matrix and passing through 2 * 2048 dense layers NN.</p>

<p>maybe try to train on predicting different features to have more generalization
train on ENCODE instead of RoadMap.. etc.
is it fully differentiable ?
How is the vector factorization working?
how is it constrained?
…</p>
<h2 id="machine-learning-applications-in-genetics-and-genomics--william-noble-uw">Machine learning applications in genetics and genomics – William Noble, UW</h2>

<p>a review of every possible machine learning applications in genomics</p>

<h2 id="bioambients-an-abstraction-for-biological-compartments">BioAmbients: An abstraction for biological compartments</h2>

<p>Ambient Calculus, a further abstraction of PI calculus, can be the mathematical base to design a framework for abstracting biological compartments</p>

<p>Basically how to design a framework for a code allowing us to simulate cells and their inner/outer processes. Bio ambient seems to be the most general mathematical framework. Available in BioSpi</p>

<p>further improvement such as duplicate, divide, erase, destroy, 3D… needs to be achieved</p>

<h2 id="personal-clinical-history-predicts-antibiotic-resistance-in-urinary-tract-infections--yelinsnitsernovich">Personal clinical history predicts antibiotic resistance in urinary tract infections -Yelin,Snitser,Novich</h2>

<p>Using Israel’s DB and stat explore they have shown that there is relation to many parameter from urinary tracks infection and other factors and related them to biological known factors. using those predictive factors, one would be able to predict the risk of such infection and to prevent it. Among the interesting factors, relation with location (old age institute etc) pharmaceutical history, cross relations eg between age/sex.</p>

<h2 id="deciphering-regulatory-dna-sequences-and-noncoding-genetic-variants-using-neural-network-models-of-massively-parallel-reporter-assays---rajiv-movva-kundaje">Deciphering regulatory DNA sequences and noncoding genetic variants using neural network models of massively parallel reporter assays - Rajiv Movva, Kundaje</h2>

<p><em>touse</em></p>

<p>We can see how using a NN model trained to predict regulatory effects of SNP on non coding sequences from Multi array analysis datasets works well. It was used together with SNPs found in GWAS for some genetically correlated cardiac diseases. some interesting results but MPRA dataset is of poor quality in general (low correlation)
Big step forward thought.</p>

<h2 id="clinical-concept-embeddings-learned-from-massive-sources-of-medical-data--andrew-l-beam1-benjamin-kompa-unc-harvard">Clinical Concept Embeddings Learned from Massive Sources of Medical Data – Andrew L Beam1, Benjamin Kompa UNC, Harvard</h2>

<p>They manage to show 100,000 clinical concepts embeddings from 3 different data-sources, they compare them to other state of the art methods on different sets of benchmark they created using clinical data sources. They achieve the biggest dataset of embedding with sota results on most of the softs·</p>

<h2 id="pooled-screens-in-human-cells-david-feldman-el-al">pooled screens in human cells, David Feldman el al</h2>

<p>there is the introduction of doing targeted fluorescent in situ sequencing ( sequencing of specific genes from within the cell by using the fluorescence sequencing method of sequencing by synthesis.) in addition, inferring the specificities of the cell (morphology, dynamics and interaction) and to link genetic variations to</p>

<h2 id="manifold-tiling-localized-receptive-fields-are-optimal-in-similarity-preserving-neural-networks-pehlevan-tepper-fi-ccb">Manifold-tiling Localized Receptive Fields are Optimal in Similarity-preserving Neural Networks, Pehlevan, Tepper FI CCB</h2>

<p>we can see a bio plausible NN optimization function relying on Hebb. anti Hebb. neurons which can uncover the manifold in a set of data (Here done with symmetric datasets) even for highly dim. datasets. Also showing a link to kernel learning (Here it is obviously an unsupervised learning technique,)</p>

<h2 id="can-deep-learning-improve-genomic-prediction-of-complex-human-traits---pau-bellot-et-al">Can deep learning improve genomic prediction of complex human traits? - Pau Bellot et al</h2>

<p>tried to regress human traits values (waist, hips, height, bone
106 heel mineral density (BHMD), body mass index (BMI), systolic blood pressure (SBP)) using MLP, CNN &amp; LR over SNPs from WGAS. 
<em>very stupid idea if you ask me. it is not how CNN are supposed to work. It is really treating DL as magic here</em></p>

<h2 id="learning-graph-level-representation-for-drug-discovery--junying-li-et-al">Learning Graph-Level Representation for Drug Discovery – Junying Li et. al.</h2>

<p>They use GCN to extract embeddings from a network representing atoms linked together (i.e. a molecule). They use a dummy super node to represent the entire molecule in one feature and they are able to classify the molecule between ones that works and ones that don’t</p>

<h2 id="multiplier-a-transfer-learning-framework-reveals-systemic-features-of-rare-autoimmune-disease--jaclyn-n-taroni-et-al">MultiPLIER: a transfer learning framework reveals systemic features of rare autoimmune disease – Jaclyn N. Taroni et. al.</h2>

<p><em>touse</em></p>

<p>multiplier is a method of using Plier (an NMF method to extract features related to gene interactions) on a large compendium of data from many gene interaction modification in multiple diseased over multiple tissues from MicroArray data. learning on a huge multivariate dataset allows the unsupervised embeddings to be richer and more precise in what they convey. They demonstrated that they were able to find feature (and thus genetic interactions for many different complex diseases. the model tries also to make similar vectors from similar tissues and diseases, aligned.
<em>quality paper</em></p>

<h2 id="on-entropy-and-information-in-gene-interaction-networks--z-s-wallace1-s-b-rosenthal">On entropy and information in gene interaction networks – Z. S. Wallace1, S. B. Rosenthal</h2>

<p>An information concept to extract from interaction networks, some subsets that are correlated or related together. It is complementary to other enrichment methods and can find relationships amongst diseases, metabolic pathways and genes with relation to phenotypic characteristics.
and recover many interesting relationships that have been found by researchers along the way.</p>

<p><em>DisGeNet</em>
<em>InBioMap</em>
<em>UNIPROT</em>
<em>MSigDB</em>
<em>GeneOntology</em></p>

<p><em>plus présenter les derniers problèmes et questionement durant les résultats (ou on en est maintenant)</em></p>

<p><em>raconter aussi les résultat légèrement en mode histoire de ce qu’on a fait.</em></p>

<h2 id="allele-specific-binding-of-rna-binding-proteins-reveals-functional-genetic-variants-in-the-rna--ei-wen-yang-jae-hoon-bahn">Allele-specific binding of RNA-binding proteins reveals functional genetic variants in the RNA – Ei-Wen Yang, Jae Hoon Bahn</h2>

<p><em>eCLIP-Seq</em></p>

<p>GV of diseases DB:
<em>GWAS, COSMIC, ClinVar, CIVIC and iGAP</em></p>

<p><em>touse</em></p>

<p>A method to find Allele specific binding events using eCLIP-seq data (trancriptome-wide protein RNA interaction data) which gives enough data to do it in an allele specific manner, the method filters out biases in the data and appears to give SOTA results. It predicted functional SNV and how they act as regulators from direct observation instead of correlation inference
“we expect that allele-specific analyses of eCLIP will be an essential approach to deciphering the function of non-coding variants in the RNA.”</p>

<h2 id="in-silico-prediction-of-high-resolution-hi-c-interaction-matrices-shilu-zhang1-deborah-chasman">In silico prediction of high-resolution Hi-C interaction matrices –Shilu Zhang1, Deborah Chasman.</h2>

<p><em>touse</em></p>

<p>to predict the contact count of two genomic regions from their one-dimensional regulatory signals which output from HiC-Reg can be used to identify significant interactions using peak-calling algorithms that have as good experimental support as those from true counts.
a regression-based framework to predict interactions between pairs
of regions across multiple cell lines by integrating published Hi-C datasets with one-dimensional regulatory genomic datasets.</p>

<h2 id="winner-take-all-autoencoders--brendan-frey">Winner-Take-All Autoencoders – Brendan Frey</h2>

<p>winner-take-all spatial and lifetime sparsity methods to train auto-encoders that learn to do fully-connected and convolutional sparse coding which jointly trains the encoder and decoder paths by direct back-propagation, and does not require an iterative EM-like optimization technique during training. sota results especially when doing semi supervised training</p>

<h2 id="dissecting-splicing-decisions-and-cell-to-cell-variability-with-designed-sequence-libraries--eran-segal-martin-mikl">Dissecting splicing decisions and cell-to-cell variability with designed sequence libraries – Eran Segal, Martin Mikl</h2>

<p>rationally designed libraries, consisting of altogether 32,789 variants, to address fundamental questions in splicing regulation considering the process in its entirety, from the processing of the RNA to the level of the final functional gene product. They could reproducibly detect even small changes in splicing ratios and quantitatively predict splicing of novel variants with high accuracy (R2 between 0.76 and 0.85)
cells evolved to have seemingly “suboptimal” splice sites, which maximizes the potential for dynamic regulation, but can also serve to ensure optimality at the level of protein isoforms. They have established an assay that is able to assess the cell-to-cell variability of splicing decisions in large scale by measuring the protein output of alternative isoforms and shown that the level of stochasticity can be encoded in the DNA.</p>

<p><strong>Highly rich paper</strong></p>

<h2 id="quantum-entanglement-in-neural-network-states--dong-ling-deng-xiaopeng-li">Quantum Entanglement in Neural Network States – Dong-Ling Deng, Xiaopeng Li</h2>

<p>the entanglement proper- ties of neural-network quantum states in the RBM architecture. and proof that all short-range RBM states satisfy an area law of entanglement for arbitrary dimensions and bipartition geometry
For some maximal volume- law entanglement RBM representation is remarkably efficient, requiring only a small number of parameters scaling linearly with the system size which show the power of NN to describe massive entanglement
Further explore the link between ANN and entangled quantum states…</p>

<h2 id="cancer-archetypes-co-opt-and-adapt-the-transcriptional-programs-of-existing-cellular-states--maayan-baron1-isabella-s-kim">Cancer archetypes co-opt and adapt the transcriptional programs of existing cellular states – Maayan Baron1, Isabella S. Kim</h2>

<p><strong>quality paper</strong></p>

<p>study of the gene expression of individual cancer cells in zebrafish and human melanomas. three recurring gene expression patterns across melanoma cancer cells which we refer to as ‘archetypes’ because they appear to be co-opted cellular states with distinct spatial location
cancer cells between human and zebrafish melanomas exhibit the same transcriptional program.
not yet purview into the origin of archetypes. found cooperation across cancer cell lines as they form a tumor in vivo</p>

<h2 id="deep-learning-sequence-based-ab-initio-prediction-of-variant-effects-on-expression-and-disease-risk--troyanskaya-fi">Deep learning sequence-based ab initio prediction of variant effects on expression and disease risk – Troyanskaya, FI</h2>
<p><strong>addition of the second paper “Predicting effects of noncoding variants with deep learning–based sequence model”,</strong> especially the method part of it.</p>

<p><em>touse</em></p>

<p>they have used a deep CNN with a lot of different Weighted patterns to predict regulatory profile (chromatin, methylation etc…) from a raw DNA sequence. using ENCODE datasets given specific cell types. this gave them the ability to predict a cell specific profile as well as to predict variation in regulatory profiles caused by variants, using a linear model (made from non linear basis function of the different profiles data, they are able to regress how much each the regulatory features of the non coding sequence before the Transcription Start Site (and a bit of the sequence after) influence the expression profile of different gene.) they were able to retrieve expression for famous protein and how they change with different cells and to retrieve SNPs causing variation in expression that were also shown from GWAS information (even better than in GWAS) and to find interesting disease correlations as well.</p>

<h2 id="one-shot-imitation-learning--ilya-sutskever-pieter-abbeel">One-Shot Imitation Learning – Ilya Sutskever, Pieter Abbeel</h2>

<p>a paper displaying the use of DQN with two distinct networks, one with an attentional mechanism (LSTLM) that sets a set of subtasks to achieve the objective, one to control the robot given the subtasks(MLP) (or even just reconstruct the sets of tasks given the final state it needs to achieve) Here the attentional system helps to subdivide the sets of tasks given to the MLP, results are SOTA for the type of tasks (first to achieve that)</p>

<h2 id="mcm2-promotes-symmetric-inheritance-of-modified-histones-during-dna-replication--nataliya-petryk-maria-dalby">MCM2 promotes symmetric inheritance of modified histones during DNA replication – Nataliya Petryk, Maria Dalby</h2>

<p><em>not very interesting</em></p>

<p>special sequencing technique revealed that parental histone segregation is almost symmetrical with a weak inherent preference for the leading strand. making it possible for MCM2 to recycle parental histones to the lagging strand (Fig. 3F, right), while a separate pathway deposits pa- rental histones on the leading strand. In this vein, it is conceivable that histone segregation can be regulated during development to drive asymmetric cell fates.</p>

<h2 id="deep-surveying-of-alternative-splicing-complexity-in-the-human-transcriptome-by-high-throughput-sequencing--qun-pan-ofer-shai-frey">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing – Qun Pan, Ofer Shai, Frey</h2>

<p><em>old article</em> 2008</p>

<p>explaining how the splicing complexity can be understood using RNA seq and how there may be a lot of possible transcripts from one sequence with different alternative splicing according to some regulatory sequences. one interesting aspect is that it shows how much there is and the amount we did not really knew about previously.</p>

<h2 id="sequential-regulatory-activity-prediction-across-chromosomes-with-convolutional-neural-networks-david-r-kelley-yakir-a-reshef">Sequential regulatory activity prediction across chromosomes with convolutional neural networks. –David R. Kelley, Yakir A. Reshef</h2>

<p><em>touse</em></p>

<p>bassenji is another DL algorithm to quantify regulatory features (certain - DNase seq, histone, Chip-seq) from CAGE experiments. it is using a CNN with expending layers (which multiply the ROI by a factor of 2 at each layers). it is from 16 month ago and appears to be less good than more recent work but provide trainings for coverage, GC%, EM for multi-reading alignment etc..
It shows link to eQTL found similarities and GWAS data, perform prediction of why some diseases happens etc, while preventing bias from LD. finally it is giving new research ideas (more data, finding more distant reg activities, better cell specific data, more precise data) and post transcriptional regulation activities etc…</p>

<h2 id="deciphering-the-splicing-code--yoseph-barash-frey">Deciphering the splicing code — Yoseph Barash, Frey</h2>

<p><em>touse</em></p>

<p>Kinda the previous version to deepBind. Here they describe an information theoretic method based on regression for prediction of variation of splicing probabilities compared to the baseline activity. this is used by taking in account typical base patterns that would create specific sec shape or bindings for the splicing to happen in a specific manner. This allowed the discovery of new features that make alternative splicing possible and of variation in different tissue with phenotypic logics behind.</p>

<h2 id="impala--deepmind">IMPALA – DeepMind</h2>

<p>A new DeepQN framework for the in parallel training of multiple actors and diminishing the delay/dissimilarity between the actors and the learners (and the learners to themselves) this framework/architecture is made possible by a new learning method that tries to combat the off policy (dissimilarity in policy) between the different agents. it uses previous work on batch A2C, Importance sampling, and other mathematically/experimentally driven techniques to compute the best possible gradient for episodes (a,s,r).</p>

<h2 id="unsupervised-embedding-of-single-cell-hi-c-data---jie-liu-galip-gurkan-yardımcı-uw">Unsupervised embedding of single-cell Hi-C data - Jie Liu, Galip Gurkan Yardımcı, UW</h2>

<p>a paper displaying how one can use a previous technique called HiCRep which smooths the Hi-C contact matrix and then computes a weighted similarity measure separately at each genomic distance. Then averages the results for the cell itself. Finally followed by a Multidimensional Scaling (MDS) of the resulting matrix to reduce the dimensionality while preserving distance relations, allows to recover angle distance between datapoints (representing cells) which allows the classification of different cell phases. It can be done with high throughput scHiC data since using a subset of contacts still allows to recover the ground truth, especially if used in combination with a bag  of more deeply sequenced cells.</p>

<h2 id="an-integrated-encyclopedia-of-dna-elements-in-the-human-genome--encode">An integrated encyclopedia of DNA elements in the human genome – ENCODE</h2>

<p><strong>start of the Engineer’s map</strong></p>

<p>a detailed explanation of the types of data and means to get them from the encode collaboration. And an overview of this data. Interesting feedbacks from it. It displays the different things that might be achieved using it. 
Moreover it provides access and a biologically oriented summary of numerous research that have been undertaken by the consortium’s researchers. Especially their results and how they fit in a more general summary of this kind of functional genomics data.</p>

<h2 id="engineering-stem-cell-organoids--xiaolei-yin-benjamin-e-mead">Engineering Stem Cell Organoids – Xiaolei Yin Benjamin E. Mead</h2>

<p>an overview of the different use, techniques, and methods for the construction of organoids or organoids like objects from human stem cells. recapitulating OoC HoC etc. And talking about how a mix of these methods is required for the future of organoids engineering.</p>

<h2 id="advantages-and-limitations-of-current-network-inference-methods--riet-de-smet-and-kathleen-marchal">Advantages and limitations of current network inference methods – Riet De Smet and Kathleen Marchal</h2>

<p>a survey of the different network inference methods in genomics, to predict the regulatory network (in bacterial genomes) with the different algorithms that are available and their mathematical framework. The different DB that are available and the problematics and usages that everyone can have from these according to one’s needs.</p>

<h2 id="optimization-techniques-on-riemannian-manifolds--steven-smith">Optimization Techniques on Riemannian Manifolds – Steven Smith</h2>

<p>two algorithms: Newton’s method and the conjugate gradient method on Riemannian manifolds, are presented and shown to possess resp. quadratic and super-linear convergence</p>

<h2 id="the-tsetlin-machine---a-game-theoretic-bandit-driven-approach-to-optimal-pattern-recognition-with-propositional-logic--granmo">The Tsetlin Machine - A Game Theoretic Bandit Driven Approach to Optimal Pattern Recognition with Propositional Logic – Granmo</h2>

<p>the tsetlin machine is a form of machine learning techniques which uses tsetlin automaton the same way a neural network uses a neuron. However it is more computationally grounded by being in the framework of propositional logic. The paper shows how to reach an equilibrium in learning even from very noisy data by finding the right hyperparameter handling error assignment. it proves that it will be able to learn anything and then shows some sota results on some datasets. Moreover the paper derives CNN and RNN like structures from tsetlin machines.</p>

<h2 id="the-genetics-of-transcription-factor-dna-binding-variation--deplancke">The Genetics of Transcription Factor DNA Binding Variation – Deplancke</h2>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>Since most of variant associated with phenotypic changes in mammals are from non coding region, the paper establishes the different mode of genetic variation mediated changes to TF binding. sets up the ways in which they have previously been uncovered, some examples and the difficulties behind this problem. It then goes on to highlight the possible technological (sequencing &amp; computational(ML))-based techniques that might help in solving it.
</code></pre></div></div>

<h2 id="seganndb-interactive-web-based-genomic-segmentation-hocking">SegAnnDB interactive Web-based genomic segmentation –Hocking</h2>

<p>a less interesting paper that reuses a mathematical framework to help on analysis (segmentation) of cancer genomes into different categories (CN variation, breakpoints(DNA deleted, inverted, or swapped)) by using a mathematical+vision framework to understand the user input segmentation and further them to the entirety of the cell genome in a supervised fashion.</p>

<h2 id="defining-a-cancer-dependency-map--aviad-tsherniak-francisca-vazquez-phil-g-montgomery-broadinstitute">Defining a Cancer Dependency Map – Aviad Tsherniak, Francisca Vazquez, Phil G. Montgomery BroadInstitute</h2>

<p>Researchers from CDS team at the broad present their ongoing effort to analyze a wide range of cancer cell lines to infer their dependencies (survival, reproduction) on different genes. To do so they are using genome scale loss of function RNAi screens (knocking down specific genes using silencing RNA which will get infiltrated by lentiviral infection in vitro on microArray live cell chips which is highly multiplexed (looking at the expression of the co-infected GFP RNA in each cases)). Each input RNA (shRNA ~100,000) will infect some cells and silence some genes. They contain a seed which will help in the binding to the desired mRNA.<br />
However, Because of this seed, this will also create off target effects (down-regulating other genes than the desired one by partial binding of the miRNA seed) it is using a linear regression model to understand the true effect of each shRNA in silencing a particular gene. (The model was in fact very similar in intuition to a NMF but was just a linear regression: to divide the dataset (cell line perturbation per shRNA) into two distinct part: cell line perturbation per gene knock down and cell line perturbation (instead of gene knock down) per shRNA with seed k) &lt;– could be done in a more efficient fashion by using an NMF which divide it and add a third matrix term which would contain the off target effect relation across shRNAs. 
The two matrix found allow one to understand specific cases where specific cell lines require a particular gene more than another and more than the regular cell. it is called: a differential dependency and might represent important therapeutic target. 
By taking the strongest ones (6std from the mean) they found 769 such genes for the 500 cell lines. they then used another non linear regression model with gene mutations, gene copy number, gene expression from the Cancer Cell Line Encyclopedia (CCLE).
They then carefully analyzed the different types of predictive MDPs (Marker dependency pairs) and specific MDPs for 86% of the dependencies.
However, to find a full range of MDP and predictive models for each, they estimate they would need to increase the amount of cell lines to &gt;5000, to account for other types of dependencies such as ones that are not single cell specific (but are important in group behavior).
–&gt; moreover, what happen for gene combination effects and in this case, time dependent gene known down effect? is there a non linear relationship between gene knockdown and gene knockoff? what about the many different type of cancer cells and their inter relationship present in a tumor?
how many other features can be added? (long range TF?) can one reuse some pretrained DL for genomic and find what molecular features in the sequence explain each types?</p>

<h2 id="improved-estimation-of-cancer-dependencies-from-large-scale-rnai-screens-using-model-based-normalization-and-data-integration--james-mcfarland-broadinstitute">Improved estimation of cancer dependencies from large-scale RNAi screens using model-based normalization and data integration – james McFarland BroadInstitute</h2>

<p><em>basically about explanaing demeter2</em></p>

<p>explaining similar things as previously with a bit more emphasis in the comparative aspect and computational aspect of Demeter2 to other types of data. a larger dataset (790dtp). in the better explanation of the model and its results. interesting and deeper explanation of the computation in the methods:
the demeter2 model uses a Bayesian model with variational inference of some params and MAP estimates of others. it explains the full procedure to infer each params using scipy and Edward. in this model a lot has been used to find the best set of parameters / hyperparameters. comparing with ground truth or prior knowledge and values to set to test the strength of each.</p>

<h2 id="genetic-and-transcriptional-evolution-alters-cancer-cell-line-drug-response--uri-ben-david1-benjamin-siranosian-gavin-ha-helen-tang-yaara-oren-kunihiko-hinohara-craig-strathdee-joshua-dempster-nicholas-lyons-robert-burns-anwesha-nag-guillaume-kugener-beth-cimini-peter-tsvetkov-yosef-maruvka1-ryan-orourke-anthony-garrity-andrew-tubelli-pratiti-bandopadhay-aviad-tsherniak">Genetic and transcriptional evolution alters cancer cell line drug response – Uri Ben-David1, Benjamin Siranosian, Gavin Ha, Helen tang, Yaara Oren, Kunihiko Hinohara, craig Strathdee, Joshua Dempster, Nicholas lyons, robert Burns, Anwesha Nag, Guillaume Kugener, Beth cimini, Peter tsvetkov, Yosef Maruvka1, ryan O’rourke, Anthony Garrity, Andrew tubelli, Pratiti Bandopadhay, Aviad tsherniak</h2>

<p>Here we look at the problems arising from the culture of cancer cell lines in research application. because of random mutations, changes in their environment, cancer cell lines undergo evolution. this creates strong differences to the in vivo scenario. The paper tries to infer the possible differences and how they might arise (medium, population bottleneck). They have dev a tool to compute the distance to a reference. They also show how to use this such as to understand cancer cooperation (with single cell seq) and correlation genotype-phenotype.</p>

<h2 id="massively-multilingual-sentence-embeddings-for-zero-shot-cross-lingual-transfer-and-beyond--mikel-artetxe-holger-schwenk">Massively Multilingual Sentence Embeddings for Zero-Shot Cross-Lingual Transfer and Beyond – Mikel Artetxe, Holger Schwenk.</h2>

<p>the papers show the power of many different languages used to create a same embeddings for all. it displays encoding techniques and train / test strategies and a new test set. It proves to be sota on most test set used. I uses a bi-LSTM encoder of 5 layers, 83 languages and an LSTM decoder  with skip connections</p>

<h2 id="cytogan-generative-modeling-of-cell-images--imaging-platform-broad-institute-of-mit-and-harvard">CytoGAN: Generative Modeling of Cell Images – Imaging Platform Broad Institute of MIT and Harvard</h2>

<p>this work uses a regular LSGAN to learn the important feature from a dataset of cell images. they show that it is not sota but can infer better latent factors which encode phenotypic information about the cell. both in the generator and discriminator. They show how to preprocess the image to get the best results. the work would need additional information to be reproducible.</p>

<h2 id="pathway-perturbations-in-signaling-networks-linking-genotype-to-phenotype-babu-for-barth-epfl">Pathway perturbations in signaling networks: Linking genotype to phenotype –Babu for Barth EPFL</h2>

<p><em>what about gene knockdown datasets… might be very interesting. 
drug interference data sets as well.
I heard of central regulators which are specific and specifically expressed in specific cancer cell lines and that knowing that information allows one to understand the specificities of the cell line (like its reaction to drugs etc..)</em></p>

<p>In here we look at several different signaling pathways which have highly oncogenic potentials. looking at them via a What mutation, how it should work, what happens when mutations. How can it be fixed.
We are really looking at the causal explanation of a phenotype by a mutation/alt. splicing/ translocation/… .</p>

<h2 id="pharmacogenomics-of-gpcr-drug-targets---masuho-leonie-j-jahn-kirill-a-martemyanov-david-e-gloriam-m-madan-babu">Pharmacogenomics of GPCR Drug Targets –  Masuho, Leonie J. Jahn, Kirill A. Martemyanov, David E. Gloriam, M. Madan Babu</h2>

<p>Here we analyze how different drugs affecting primarily or secondarily the GPCR membrane receptors of cells can have different effects considering the naturally occurring mis-sense variation in the human population on these corresponding protein coding genes.</p>

<p><em>how comme they never thought of doing the same in pharmacollogy??</em>
<em>what about the non coding variants in the population it will reflect also some differential effects in the signaling pathway and even variation in the proportion of membrane receptors</em></p>

<p>##</p>

<p><em>I really want to see how the pipeline to integrate the entire G cycle looks like.</em></p>

<h2 id="papers-gilchrist">Papers Gilchrist</h2>
<h3 id="combining-models-of-protein-translation-and-population-genetics-to-predict-protein-production-rates-from-codon-usage-patterns">Combining Models of Protein Translation and Population Genetics to Predict Protein Production Rates from Codon Usage Patterns</h3>

<h3 id="explaining-complex-codon-usage-patterns-with-selection-for-translational-efficiency-mutation-bias-and-genetic-drift">Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift</h3>

<h3 id="estimating-gene-expression-and-codon-specific-translational-efficiencies-mutation-biases-and-selection-coefficients-from-genomic-data-alone">Estimating Gene Expression and Codon-Specific Translational Efficiencies, Mutation Biases, and Selection Coefficients from Genomic Data Alone</h3>

<h3 id="quantifying-codon-usage-in-signal-peptides-gene-expression-and-amino-acid-t-usage-explain-apparent-selection-for-inefficient-codons">Quantifying codon usage in signal peptides: Gene expression and amino acid T usage explain apparent selection for inefficient codons</h3>

<p>I had the time to read the papers you sent me and they were very good. The third one presented a great introduction to CUB and a good model which attains a very high score. I would guess that this model could not get the same score for other species. I still think that a model of CUB would always be reductionist as the CUB is a proxy for so many different processes and adaptations of the cell. The many fine grained Bayesian model that you present however seems to make sense and creating a model is to me the only way to test a theory. The ability to regress values such as the protein synthesis rate is impressive and it does not seem trivial to me that this should be the case. The model that you have developed seems very suited for answering questions about the CUB as shown for example in “quantifying codon usage in signal peptides”.</p>

<h2 id="where-genome-biology-is-going-next">Where genome biology is going next</h2>
<p>multiple researchers giving their view of where genomics is heading</p>

<h2 id="a-topological-and-conformational-stability-alphabet-for-multipass-membrane-proteins--patrick-barth">A topological and conformational stability alphabet for multipass membrane proteins – Patrick Barth</h2>

<p>Here is the study of multipass transmembrane proteins which are composed of many transmembrane helices. they have a specific confirmation which is dictated by their sequence. interaction between helices yields the specific shape of thee multipass prot.
specific residues which help connect the helices and give them their shape seems to be coevolving, stabilizing the shape and are determinant of the topology. For the first time this study analyzed trimers of helices insetead of dimers.</p>

<h2 id="primer-on-deep-learning-in-genomics">Primer on Deep Learning in Genomics</h2>

<h2 id="a-guide-to-dl-in-healthcare--andre-esteva-alexandre-robicquet">A guide to DL in healthcare – Andre Esteva, Alexandre robicquet</h2>

<p>Some simple yet necessary ideas about how one would best use DL in healthcare and what are the future and promises of it.</p>

<h2 id="mammalian-genes-are-transcribed-with-widely-different-bursting-kinetics">Mammalian Genes Are Transcribed with Widely Different Bursting Kinetics</h2>

<p>in this paper it is explained briefly (with more precision about the models in the supplementary materials) the rational behind bursting kinetics in mamlian genes. We look at a model going from in/activation; trancription rates; translation rates and degradations. using markovian prob. models. per coding genes. then correlates it with exeperimental fluo activity thanks to a poissonian noise model. the entirety of the model is then parametrized by EM algo thx to baum welsh algorithm. the computation of the states (bc it is a markov. model.) is done with the viterbi algorithm. The signal in the images were deconvolved using wavelet temporal analysis. the results described how different profiles of protein activity were highly periodic with different types of kinetics. also shown in the analysis of the states/params of the model for different mRNA. this difference could be seen also for similar genes accross cell lines.</p>

<h2 id="what-are-super-enhancers--sebastian-pott--jason-d-lieb">what are super enhancers? – Sebastian Pott &amp; Jason D Lieb</h2>
<p><em>very important</em></p>

<p>Explainingwhat are enhancers. open questions about them at the functional level.
then turns out how to compute super enhancers. their possible differences with regular ones and possible similarity which would make them just reegular enhancers specific of the particular expression of the cell line.
it turns out to display 5 questions about super enhancer and if they truly are mechanistically different.
And a framework to find it out.</p>

<p><em>how do gene dependancy relate to SEn? using the same Crispr screening technique
One Should Compute SEn without using merging of regular one according to the paper. Might help answering the question as well</em></p>

<h2 id="super-enhancers-in-the-control-of-cell-identity-and-disease--denes-hnisz-brian-j-abraham-tong-ihn-lee">Super-Enhancers in the Control of Cell Identity and Disease – Denes Hnisz, Brian J. Abraham, Tong Ihn Lee</h2>
<p><em>very important</em></p>

<p><em>might be interesting to reuse softs such as DeepPeakCaller and DeepBind</em></p>

<p><em>Maybe one might find the equivalent of superENhancer for lesser TF type when looking only at these.</em></p>

<p>_Sen are concepts associated with TF master TF, enhancer, cell fate, cell differenciation/pluripotency (see for of Pr. Gioele La manno EPFL, transcription regulation (cis &amp; trans) (also via enhancerRNAs), _</p>

<p><em>it would be valuable to identify the master transcription factors that control all cell states.</em></p>

<p><em>if alteration in superenhancer drive cell specificity. what if some cancer were some form of local valley in cell specificity out of which the cell can’t escape. this could be seen by looking at differences in superenhancer region through stages of cancer or types of cancer from a same cell line through the eyeyes of what La Manno has done.</em></p>

<p><em>when you look at Senh accross cell lines and cancers and you see ~ a thiord which are cell line specific. do the other 2/3 are shared accross cell? cell types? cell groups? random noise, i.e. not acual Senh?</em></p>

<p>First the discovery that super enhancer are linked with cancer hallmark, disease SNPs, specific per cell types, linked with specific core transcription factors, miRNA ncRNA. might explain cell diff, fate, etc.. they create feedback loop. can be transcribed into eRNA (which may contribute to gene activation) and are not only enriched in TF but also in RNA Polymerase II, Cofactors, and Chromatin Regulators. It is also a dataset of 86 cell type (human)</p>

<h2 id="network-in-network--min-lin-qiang-chen-shuicheng-yan">Network In Network – Min Lin, Qiang Chen, Shuicheng Yan</h2>

<p>putting MLP instead of regular convolution for each layer of a CNN. ReLu then finishing by a global average pooling layer feeding to a logistic regression classifier.</p>

<h2 id="selective-inhibition-of-tumor-oncogenes-by-disruption-of-super-enhancers--jakob-love-heather-a-hoke-charles-y-lin">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers – Jakob Love, Heather A. Hoke, Charles Y. Lin</h2>

<p>You have some enhancer of MYC, MCL1, IRF4, PRDM1 … which are strongly bound by BRD4 and MED1. we know that reducing the level of BRD4 by using a JQ1 tratment which will lead to a bromodomain inhibition of BRD4 and thus restrict binding of itself and its coFactors. In many types of cancer cells. this leads to a selective desctruction of them and not of regular cells. This is explained in this paper by stating that these cells are known to have very strong super enhancers to these genes and thus it is places which relies more heavily on BRD4 and which are more essential to these cell lines. Giving ideas to explain and discover more oncogenic targets.</p>

<h2 id="ataris-computational-quantification-of-gene-suppression-phenotypes-from-multisample-rnai-screens-diane-d-shao-aviad-tsherniak-shuba-gopal-barbara-a-weir">ATARiS: Computational quantification of gene suppression phenotypes from multisample RNAi screens– Diane D. Shao, Aviad Tsherniak, Shuba Gopal, Barbara A. Weir</h2>

<p>The beginning of the DepMap, before large scale and demeter. Discovering how a lot of data can help uncovering gene dependency in cancer from rnaI. Using  a Matrix Factorization. Moreover as many RNAi can affect a gene. After it, a refinment step is performed to find a subset of best performing ones according to the profiles found by the matrix factorization problem.
showing ability to:</p>
<ul>
  <li>identify fuctional relationship between genes</li>
  <li>identify nocel cancer associated genes.</li>
  <li>identify genomic predictor of gene dependency</li>
</ul>

<p><em>did not  get fig 4C …</em></p>

<h2 id="pan-cancer-genome-and-transcriptome-analyses-of-1699-paediatric-leukaemias-and-solid-tumours-xiaotu-ma1-yu-liu-yanling-liu-ludmil-b-alexandrov-michael-n-edmonson">Pan-cancer genome and transcriptome analyses of 1,699 paediatric leukaemias and solid tumours– Xiaotu Ma1, Yu Liu, Yanling Liu, Ludmil B. Alexandrov, Michael n. Edmonson</h2>

<p>A paper recapitulating the creation a first analysis of 1699 pediatric leukimia tumors at the transcription and genomic level. understanding similarities and differencies btween them/ them and adult leukimia. and some interesting new genes and genetic features. Additionaly a more thoruough analysis of WT1 D447N. <em>for which I was not 100% confident on my comprehension</em></p>

<h2 id="the-landscape-of-genomic-alterations-across-childhood-cancerssusanne-n-gröbner-barbara-c-worst-joachim-weischenfeldt-ivo-buchhalter-kortine-kleinheinz-vasilisa-a-rudneva-pascal-d-johann">The landscape of genomic alterations across childhood cancers–Susanne N. Gröbner, Barbara C. Worst, Joachim Weischenfeldt Ivo Buchhalter, Kortine Kleinheinz, Vasilisa A. Rudneva Pascal D. Johann</h2>

<p>Another presentation of a dataset containing a thousand tumours (most of them ALL and central nervous system related) with identification of regular driver genes, structural and CN variants and germline/somatic variants. with a study of the particular processes in childhood cancers as well as potential drug targets related to the discoveries listed here.</p>

<h2 id="a-landscape-of-pharmacogenomic-interactions-in-cancer-francesco-iorio-theo-a-knijnenburg-daniel-j-vis">A Landscape of Pharmacogenomic Interactions in Cancer– Francesco Iorio, Theo A. Knijnenburg, Daniel J. Vis</h2>

<p>A look at the pharmacogenomic landscape (response to 265 drugs) of 1,001 human cancer cell lines from 11,289 tumors points to new treatment applications for hundreds of known anti-cancer drugs. to determine:</p>
<ul>
  <li>the extent to which cancer cell lines recapitulate oncogenic alterations in primary tumors.</li>
  <li>which oncogenic alterations associate with drug sensitivity,</li>
  <li>whether logic combinations of multiple alterations better explain drug sensitivity.</li>
  <li>the relative contribution of different molecular data types, either individually or in combination, in predicting drug response</li>
</ul>

<p><em>here it seems that the gene expression data only helps to infer the cell type &gt; thus the dependancy according to that. thus a two step model (cell type &gt; specific sensitivity) might be more appropriate</em>
<em>Here I think you need to model some of the specific proteins to understand better what explains the sensitivity and what events (SNP, CNV, …) might help you understand that. (see work from Patrick Barth)</em></p>

<hr />
<p>CFE  cancer functional events
RACS recurrently aberrant copy number segments</p>

<hr />

<h2 id="curiosity-driven-exploration-by-self-supervised-prediction-deepak-pathak-pulkit-agrawal">Curiosity-driven Exploration by Self-supervised Prediction– Deepak Pathak Pulkit Agrawal</h2>

<p><em>what about training on the ability to predict the distribution dynamics instead of just prediction error on the next state (else might be triggered by simple noise or stochastic distributions)</em></p>

<p>agent is trained on minimizing the prediction error of state t+1 from state t and action t (<em>may be better with reusing all previous states in some form</em>)</p>

<p><em>“the agent will be completly unperturbed by external factors” yet it seems very delicate for the agent to understand what will have future effect on him down the road. I guess the human brain does not do it this way.</em></p>

<p><em>They should have tried to do parameters sharing.</em></p>

<h2 id="master-transcription-factors-and-mediator-establish-super-enhancers-at-key-cell-identity-genes-warren-a-whyte14-david-a-orlando14-denes-hnisz">Master Transcription Factors and Mediator Establish Super-Enhancers at Key Cell Identity Genes– Warren A. Whyte,1,4 David A. Orlando,1,4 Denes Hnisz</h2>

<p>Very similar to the previous papers about super enhancers. Most of the new finding is in the title.</p>

<blockquote>
  <p>Super-enhancer formation appears to occur as a conse- quence of binding of large amounts of master transcription factors to clusters of DNA sequences that are relatively abundant across these large domains. Theses domains featiure CDS which are the core constituent of Cell Identity.</p>
</blockquote>

<p><em>it seems that super enhancers activate cell identity by the type of genes they bind to. However it seems that it is always the same master. What plays on the differences? (initial parameters/ TF concentration?/other non master TF) can we reprogram cells using these TF value?</em></p>

<p><em>given that cancer seems to be something that evolves really quickly over time from a regular cell (which in case of children might have a propension to go so). what about using a flow model on the cell’s transcriptomics/genomic to understand the flow of cancerisation as a differenciation. You could see it going to a probabilistic valley with canyons and taking a path which will increase the likelyhood of turning into a cancer. Then, why not look at the cell before it become a full on cancer and see the possible differences in pathway and create things that attack cells in this particular state.</em></p>

<hr />
<p>EPUs enhancer-promoter unit</p>

<hr />
<p>MACS CHIp analysis
BOWTIE alignment
CAMs cell–cell adhesion molecules
—-</p>

<h2 id="the-hallmarks-of-cancer-douglas-hanahan-and-robert-a-weinberg">The Hallmarks of Cancer– Douglas Hanahan* and Robert A. Weinberg</h2>

<p>the foundationnal paper presenting the 6 hallmarks of cancer:</p>
<ul>
  <li>limitless replication</li>
  <li>evading apoptosis</li>
  <li>insensitivity to antigrowth signals</li>
  <li>self sufficiency in growth signals</li>
  <li>tissue invasion (metastasis)</li>
  <li>sustained angiogenesis
 via a suite of possible mechanism.</li>
</ul>

<p>showing that cancer should not happen with this likely hood but that instead cells when loosing specific sets of reparing/stability genes or phagocytosising cells (which may incorporate some instable DNA)) may undergo, with much more likelyhood: cancer.</p>

<blockquote>
  <p>we look ahead 10 or 20 years to the time when the diagnosis of all the somatically acquired lesions present in a tumor cell genome will become a routine procedure. By then, genome-wide gene expression profiles of tumor cells will also be routine. With all this information in hand, it will become possible to test definitively our proposition that the development of all types of human tumor cells is governed by a common set of rules such as those implied by the six acquired capabilities enumerated here.</p>
</blockquote>

<p><em>really a paper about understanding cancer 101</em></p>

<hr />
<p>GS growth signal</p>

<hr />

<h2 id="coming-of-age-ten-years-of-next--generation-sequencing-technologiesara-goodwin-john-d-mcpherson-and-w-richard-mccombie">Coming of age: ten years of next- generation sequencing technologie–Sara Goodwin, John D. McPherson and W. Richard McCombie</h2>

<p>a presentation of the sequencing technologies with their pros and cons and an explanation of how they work. Then a similar deeper presentation for all NGS technologies. a repartition according to differences and similarities.  the new long read sequencers from PACbio and Oxford nanopore seems extremelly powerfull and impressive.</p>

<p><em>a lot of definitions and explanations in this paper (must read)</em></p>

<hr />
<p>SBL sequencing by ligation
SBS sequencing by synthesis 
ssDNA single strand != dsDNA
dNTPs deoxynucleotide primer
CRT cyclic reversible termination
SNA single-nucleotide addition
ZMW zero-mode waveguides</p>

<hr />

<h2 id="high-performance-medicine-the-convergence-of-human-and-artificial-intelligence-eric-j-topol">High-performance medicine: the convergence of human and artificial intelligence– Eric J. Topol</h2>

<p>presenting the use of DL for radiology, pathology, dermatology, ophthalmology, cardiology, gastroenterology, cardiology</p>

<p>a very good categorization of current companies and techniques and a lot of actual comparative data against clinicains and with clinicians</p>

<blockquote>
  <p>A deep-learning algorithm for lung cancer digital pathol- ogy slides not only was able to accurately classify tumors, but also was trained to detect the pattern of several specific genomic driver mutations that would not otherwise be discernible by pathologist</p>
</blockquote>

<h2 id="generating-focussed-molecule-libraries-for-drug-discovery-with-recurrent-neural-networks-marwin-hs-segler1-thierry-kogej-christian-tyrchan-and-mark-p-waller">Generating Focussed Molecule Libraries for Drug Discovery with Recurrent Neural Networks– Marwin HS Segler1, Thierry Kogej, Christian Tyrchan and Mark P Waller</h2>

<p>using an RNN generator very similar to the ones used modelling language, to model molecules (no conformation, only placement). althought this work is nice and highlights what can be done with substantial results to show. it may lack information on conformation and it does not seem to use every information available to biochemists. It uses transfer learning and a measure of fitness which allows it to be efficient. it gives some informations of the goals in this field.</p>

<p>It may be problematic however that it does not provide radically new molecules but quite similar ones. ways to present and tests such data does not seem to be present yet. The author do present good future work ideas and skepticism.</p>

<hr />
<p>ECFP extended connecticvity fingerprint
TPM target prediction model
ESC embryonic stem cells</p>

<hr />

<h2 id="core-transcriptional-regulatory-circuitry-in-human-embryonic-stem-cells">Core Transcriptional Regulatory Circuitry in Human Embryonic Stem Cells</h2>

<p>specific papers pre-Superenahncher discovery where it displays how core TF create a feedforward circuitry in ESCs and how it is the core circuitry defining their identity.</p>

<h2 id="selective-gene-dependencies-in-mycn-amplified-neuroblastoma-include-the-core-transcriptional-regulatory-circuitry-adam-d-durbin--mark-w-zimmerman-neekesh-v-dharia">Selective gene dependencies in MYCN-amplified neuroblastoma include the core transcriptional regulatory circuitry– Adam D. Durbin  Mark W. Zimmerman Neekesh V. Dharia</h2>

<p>this paper represent a deeper work on using CRC as tools to guess gene dependencies in a specific cancer type. It proved very efficient and puts itself in the framework of discoveries linked with super enhancers.</p>

<hr />
<p>CRC core regulatory circuitry
ATAC assay of transposase accessible chromatin</p>

<hr />

<h2 id="neuroblastoma-is-composed-of-two-super-enhancer--associated-differentiation-states-tim-van-groningen-jan-koster-linda-j-valentijn-danny-a-zwijnenburg-nurdan-akogul-nancy-e-hasselt-marloes-broekmans">Neuroblastoma is composed of two super-enhancer- associated differentiation states– Tim van Groningen, Jan Koster, Linda J Valentijn, Danny A Zwijnenburg, Nurdan Akogul, Nancy E Hasselt, Marloes Broekmans</h2>

<p>In this paper, people from the broad show that analysing tumors. two types of archetypes coexist from distinct original cell lines. They can be differentiated with Super enh profiling and by understanding their circuitry, changed from one state to another. The cells have been shown to present similar effect in vivo with one archetype being more chemoresistant. Leading path to new screenings and therapies.</p>

<p><em>here it seems they identified only two archetype instead of 3…</em></p>

<p><em>it seems to make some sense and to be quite crazy that influencing just some TF would allow reprogramming of many cell types.</em></p>

<hr />
<p>IHC  Immunohistochemistry</p>

<hr />

<h2 id="interaction-networks-for-learning-about-objects-relations-and-physics-deepmind">Interaction Networks for Learning about Objects, Relations and Physics– DeepMind</h2>

<p>Nice paper about predicting the future of physical interaction by combining relational reasoning, simulation and NN. (no deep learning here!) It provides inexact but extremelly believable results in many different simulation tasks and could learn non newtonian dynamics. 
<em>It is a first step/idea to many other interesting directions</em></p>

<h2 id="precision-targeting-of-bfl-1a1-and-an-atm-co--dependency-in-human-cancer-rachel-m-guerra-gregory-h-bird-edward-p-harvey">Precision Targeting of BFL-1/A1 and an ATM Co- dependency in Human Cancer– Rachel M. Guerra, Gregory H. Bird, Edward P. Harvey</h2>

<blockquote>
  <p>Guerra et al. constructed an exquisitely selective BFL-1 inhibitor capable of covalent BFL-1 targeting and cellular penetrance without membrane disruption. Mining a genetic dependency database revealed a spectrum of BFL-1 dependency in cancer and an ATM co- dependency in AML, prompting the combination of BFL-1 and ATM inhibitors to achieve synergistic cytotoxicity.</p>
</blockquote>

<p><em>I don’t understood all the nuances of the paper, especially the design and thought process displayed in the results.</em></p>

<p><em>The papeer could have needed a bit more explanation on the underlying process it is trying to solve (schematics to avoid repeating long enumeration of action of protein)</em></p>

<p>Here the author reuse and modify a NOXA protein to create one that will very specifically bind and inhibate BFL1 (with D-NA (at N term)) and thus reactivate the apoptotic route of the cell. They use staple peptide to consolidate the protein SAHB15 and subsequently modified aa for less hydrophobic ones to create a protein that could enter the cells without destroying them. they then undertakent to test variation, precision etc.. of this protein to confirm its pharmacological relevance in BFL1 and ATM co dependant ccl</p>

<hr />
<p>ATM ataxia-telangiectasia-mutated</p>

<p>CCL cancer cell line</p>

<hr />

<h2 id="pioneer-factors-govern-super-enhancer-dynamics-in-stem-cell-plasticity-and-lineage-choice-rene-c-adam-hanseul-yang-shira-rockowitz-samantha-b-larsen-maria-nikolova-daniel-s-oristian">Pioneer factors govern super-enhancer dynamics in stem cell plasticity and lineage choice– Rene C. Adam, Hanseul Yang, Shira Rockowitz, Samantha B. Larsen, Maria Nikolova, Daniel S. Oristian</h2>

<p><em>what about computing superenhancers from raw sequence data by using a DL tool.
CNN on 250k bp (should reuse b&amp;w like parameters) or maybe multiply values into one. 2^10 by combining 5 values)</em></p>

<p>Here by using Hair Folicule stem cell as a proxy. researchers showed the impact of the pionner factor SOX9 on stem cell plasticity by activating other super enhancer link to the HFSC plasticity by initiating H3K27ac. Going with more precision in the superenhancer study for a particular/different problem.</p>

<hr />
<p>pioneer factors: dominant in their ability to engage silent, unmarked chromatin and initiate the recruitment of other factors, thereby imparting new function to regulatory DNA sequences.</p>

<p><a href="https://en.wikipedia.org/wiki/Histone_acetylation_and_deacetylation">acetylation</a></p>

<p>TACs Transit-amplifying cells</p>

<p>GREAT soft(go function enrichment)</p>

<p>HOMER soft(motif enrichment)</p>

<hr />

<h2 id="the-evolving-war-on-cancer-daniel-a-haber-nathanael-s-gray-and-jose-baselga">The Evolving War on Cancer– Daniel A. Haber, Nathanael S. Gray and Jose Baselga</h2>

<p>a review of what has been done. impacts and challenges, even reasons for the past 40yr war on cancer. what is the state of things in 2011 and what is the roadmap for the next 10-20 yrs. what will be the challenges awaiting us.</p>

<h2 id="statistical-methods-in-integrative-genomics-sylvia-richardson-george-c-tseng-and-wei-sun">Statistical Methods in Integrative Genomics– Sylvia Richardson, George C. Tseng, and Wei Sun</h2>

<blockquote>
  <ol>
    <li>All models have their explicit and implicit assumptions. Model fitting to the underlying data structure largely determines the success of omics data analysis.</li>
    <li>In data integration, certain biological mechanisms and prior knowledge are often known across different omics data. Proper modeling of such prior knowledge is crucial to enhance statistical power and identify biologically interpretable results.</li>
    <li>IncorporatingpriorbiologicalinformationusingBayesianhierarchicalmodelingisavery powerful method of data integration.</li>
    <li>Integration of multiple types of omics data involves ultrahigh-dimensional problems. Feature selection and its related model selection problem are major issues when devel- oping a novel method.</li>
    <li>Network-based methods are effective approaches to integrating multiple types of data and biological knowledge.</li>
  </ol>
</blockquote>

<p><em>we have to know tumor purity</em>
<em>hoizontal&lt;-&gt;vertical</em> from stat to bio</p>

<ul>
  <li>
    <p>Clustering
presentation of METABRIC. foundational for tumor heterogeneity detection. MDI &amp; BCC for bayesian clustering integrating many datasources.
some fou</p>
  </li>
  <li>
    <p>Regression
regressing on large p small n datasets or multi omics/response models. integrating also prior (integration) information from given knowledge
with bayesian modeling of eQTL variation impact on gene expression</p>
  </li>
  <li>
    <p>Grahical models (same as regression, integrating structural relationship prior information)
with the representation of a miRNA effect on gene expression
with the representation of eQTL impact on gene expression</p>
  </li>
</ul>

<p>PARADIGM: to model the DAG of the interation between CN, Gene expression, protein abundance, activity of complex, prot, etc…) and finds the value of unobserved variable given observed ones.</p>

<p>OmicKriging: predicting phenotype from a set of phenotype’s similary matrix</p>

<p>CONEXIC (copy number and expression in cancer) to detect modules in which copy number affects the expression of a driver gene, which in turn regulates the expression of a group of genes.</p>

<p>HotNet: identifies significantly altered subnetworks in an interaction net- work by a network diffusion approach, which can be understood as a random walk on a gene-gene interaction graph. In other words, a somatic mutation in gene A may also affect gene A’s neighbors in the interaction graph. After network diffusion, HotNet evaluates the frequency of a subnetwork being altered across patients and find those subnetworks that were recurrently altered. Network diffusion provides a network-smoothed version of the consequences of somatic mutations. clustering the network-smoothed mutation profiles by nonnegative matrix factorization</p>

<hr />
<p>Zellner’s g-prior: give $y_{i}=x_{i}^{\top }\beta +\varepsilon _{i}.$
 Zellner’s g-prior for ${\displaystyle \beta } \beta$  is a multivariate normal distribution with covariance matrix proportional to the inverse Fisher information matrix for ${\displaystyle \beta } \beta$ .
 like so : ${\displaystyle \beta |\psi \sim {\text{MVN}}[\beta _{0},g\psi ^{-1}(X^{\top }X)^{-1}].}$</p>

<hr />

<h2 id="the-cancer-genome-atlas-pan-cancer-analysis-project-the-cancer-genome-atlas-research-network-john-n-weinstein-eric-a-collisson-gordon-b-mills-kenna-r-mills-shaw5-brad-a-ozenberge">The Cancer Genome Atlas Pan-Cancer analysis project– The Cancer Genome Atlas Research Network, John N Weinstein, Eric A Collisson, Gordon B Mills, Kenna R Mills Shaw5, Brad A Ozenberge…</h2>

<p>really a presentation of TCGA and its goals and objectives as of 2013. Most specifically a project to computationally analyse a set of ~3000 tumors from 6 data types (RPPA meth CN mut miRNA eRNA) from 12 tumor organ types.</p>

<blockquote>
  <p>• What is the spectrum of nucleotide- and dinucleotide-level changes associated with different carcinogenic etiologies (for example, tobacco, pathogens or inflammation) operating in different parts of the body?
• Will integration of additional data sources, including additional tumor types from TCGA and other projects, increase the power of analysis in useful ways?
• How can characterization based on molecular changes complement pathological analysis for classification of cancers into tumor lineages with potentially different clinical management?
• Can molecular profiles effectively categorize cancers for therapeutic decision-making?
• Are there predictive expression-based signatures for genomic events that transcend tissues, reflecting pathways disrupted by the alterations?
• Will comprehensive protein analysis through emerging mass spectrometry approaches in the Clinical Proteomic Tumor Analysis Consortium (CPTAC) and other efforts extend the power of the genomic, transcriptomic and proteomic analyses in TCGA?
• Will whole genome analysis demonstrate the influence of mobile elements, mutations in non-coding regions and connections to constitutional risk-associated loci?
• How are changes in protein families distributed across different tumor types?
• Are aberrations in specific protein domains or pathways distributed differentially across tumor lineages?
• Beyond the known examples, including in cervical, head and neck, esophageal and hepatocellular cancers, can we identify other cancer types that show virally mediated initiation?
• Are bacteria associated with different cancer lineages (as Fusobacteria are in colorectal cancer43)?
• Can the answers to any of these questions help in the design of novel therapies and clinical trials, with the ultimate goal of improving patient outcomes?</p>
</blockquote>

<h2 id="the-cancer-cell-line-encyclopedia-enables-predictive-modelling-of-anticancer-drug-sensitivity-jordi-barretina-giordano-caponigro-nicolas-stransky-kavitha-venkatesan-adam-a-margolin-sungjoon-kim-christopher-j-wilson4-joseph-leha-gregory-v-kryukov-dmitriy-sonkin-anupama-reddy-manway-liu-lauren-murray-michael-f-berger-john-e-monahan">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity– Jordi Barretina, Giordano Caponigro, Nicolas Stransky, Kavitha Venkatesan, Adam A. Margolin, Sungjoon Kim, Christopher J. Wilson4, Joseph Leha, Gregory V. Kryukov, Dmitriy Sonkin, Anupama Reddy, Manway Liu, Lauren Murray, Michael F. Berger, John E. Monahan…</h2>

<p>Here there is a presentation of CCLE and examples of what can be done by using CCLE’s data (with additional experiments) to understand relationship between genes plasma cell lineage correlated with sensitivity to IGF1 receptor, AHR expression was associated with MEK inhibitor efficacy in NRAS-mutant lines and SLFN11 expression predicted sensitivity to topoisomerase inhibitors. using elastic net modelling studies identified key pre- dictors or mechanistic effectors of drug response.</p>

<h2 id="deepnano-deep-recurrent-neural-networks-for-base-calling-in-minion-nanopore-reads-vladimir-bozabroňa-brejova-toma-svinař">DeepNano: Deep recurrent neural networks for base calling in MinION nanopore reads— Vladimir Boza,Broňa Brejova, Toma svinař*</h2>

<p>a simple vanilla bidirecRNN to associate bp to reading events from the oxford nanopore minION machine. it achieves similar results to oxford’s LSTM algorithm while being opensource and faster. 
it uses longrange relationships information to estimate the actual values.</p>

<p><em>some things are not explaned nor well document (maybe in the code).</em>
<em>further tests of the accuracy accross more data types etc would be essential to assess the actual quality of this algorithm</em></p>

<h2 id="systematic-investigation-of-genetic-vulnerabilities-across-cancer-cell-lines-reveals-lineage-specific-dependencies-in-ovarian-cancer--hiu-wing-cheunga-glenn-s-cowleyb-barbara-a-weirb-jesse-s-boehmb-scott-rusinb-justine-a-scottb-alexandra-eastb-levi-d-alib-patrick-h-lizotte">Systematic investigation of genetic vulnerabilities across cancer cell lines reveals lineage-specific dependencies in ovarian cancer – Hiu Wing Cheunga, Glenn S. Cowleyb, Barbara A. Weirb, Jesse S. Boehmb, Scott Rusinb, Justine A. Scottb, Alexandra Eastb, Levi D. Alib, Patrick H. Lizotte…</h2>

<p>after using the project achilles data and making specific analyses. the researchers found that PAX8 was a specific dependency in ovarian cancers.</p>

<hr />
<p>woe weight of evidence</p>

<hr />

<h2 id="systematic-identification-of-genomic-markers-of-drug-sensitivity-in-cancer-cells-mathew-j-garnett-elena-j-edelman-sonja-j-heidorn-chris-d-greenman-anahita-dastur-king-wai-lau-patricia-greninger">Systematic identification of genomic markers of drug sensitivity in cancer cells– Mathew J. Garnett, Elena J. Edelman, Sonja J. Heidorn, Chris D. Greenman, Anahita Dastur, King Wai Lau, Patricia Greninger…</h2>

<p>a presentation of what will become project PRISM. using results from 130 compounds to 600 CCL. finding known and a lot of unexpected correlations. using regular MANOVA analysis, elastic net prediction to find most predictable transcriptionalvfeatures for drug sensitivty.</p>

<p><em>interesting use of some algos</em></p>

<hr />
<p>MSI: microsatelitte instability</p>

<p>capillary sequencing: similar to electrophoresis with a laser that makes each nucleotide shine a specific color which helps in sequencing the dna</p>

<p>PICNIC: predict copy number from microarray data</p>

<p>RMA: Robust Multi-Array Average (for gene expression data)</p>

<p>heteroscedasticity variability in variance</p>

<hr />

<h2 id="advancing-the-aging-biology-toolkit-troy-k-coody-and-adam-l-hughes">Advancing the aging biology toolkit– TROY K COODY AND ADAM L HUGHES</h2>

<p>a presentation from Calico of a new method to segregate old cells (here for yeast) in order to do better analysis between the two and understand what might explain aging for further research.</p>

<h2 id="models-of-human-core-transcriptional-regulatory-circuitries-violaine-saint-andré-alexander-j-federation-charles-y-lin">Models of human core transcriptional regulatory circuitries– Violaine Saint-André, Alexander J. Federation, Charles Y. Lin</h2>

<p>Here is described a framework to guess the CRC from simple CHIP data of various human tissues. to understand cell types and more about the CRC from this data. It shows this framework mainly in analysing ESC but also provide the same analysis on 80 different cell type.</p>

<p><em>can we guess CRC and SE from simple CHIP data of each cells? 
can we infer the cell type from it? can we infer the cell type from only a small subset of RNAs which encodes CRC?</em></p>

<p><em>use it for the next further trial of my data</em>
https://hoffmanlab.org/proj/virchip/</p>

<hr />
<p>CRC: core transcriptional regulatory circuitry</p>

<p>hESC: human ESC
CNA: copy number alteration</p>

<p>RTK: receptor tyrosine kinases
PI3K: phosphoinositide 3- kinase</p>

<hr />

<h2 id="pan-cancer-network-analysis-identifies-combinations-of-rare-somatic-mutations-across-pathways-and-protein-complexes-mark-d-m-leiserson-fabio-vandin-hsin-ta-wu-jason-r-dobson">Pan-cancer network analysis identifies combinations of rare somatic mutations across pathways and protein complexes– Mark D M Leiserson, Fabio Vandin, Hsin-Ta Wu, Jason R Dobson</h2>

<p>a network analysis using a heat diffusion model accross network of interconnected genes identifies mutation (heat) that display important network for the cancers accross thousands of cancer and tens of cancer types. this analysis focus on 3 interesting pathways:</p>
<ul>
  <li>SWI/SNF chromatin-remodeling complex</li>
  <li>BAP1 complex</li>
  <li>cohesin and condensin complexes</li>
  <li>MHC class I proteins</li>
  <li>the telomerase complex</li>
</ul>

<p>they also found 92 mutated gene with usually low score in non networked single gene analysis</p>

<p>It might be used on any gene data with a value measure for each gene!</p>

<p><em>Q: How do they guess link to cancer from only presence in the netwtork?</em></p>

<p><em>N: why don’t we stop looking at what is mutated and start looking at what is not mutated? like looking at what specific circuitry cancers rely upon to stay alive, that other regular cells might not need?</em></p>

<hr />
<p>GSEA gene set enrichment analysis</p>

<hr />

<h2 id="auto-encoding-variational-bayes--diederik-p-kingma-max-welling">Auto-Encoding Variational Bayes – Diederik P. Kingma, Max Welling</h2>

<p>presentation of an estimator of variational lower bounds for approximate inference with continuous latent variables. and the presentation of an Auto Encoder (like) variational bayes which learns an even beter approximate inference using SGVB. For the case of i.i.d. datasets and continuous latent variables per datapoint.</p>

<h2 id="towards-systematic-functional-characterization-of-cancer-genome-jesse-s-boehm-and-william-c-hahn-broadinstitute">Towards systematic functional characterization of cancer genome– Jesse S. Boehm, and William C. Hahn, BroadInstitute</h2>

<p>A demonstration of the many techniques to perform functional characterization of genes:</p>
<ul>
  <li>RNAI</li>
  <li>ORF</li>
  <li>transposons</li>
  <li>small molecule</li>
  <li>(now much more, like crispr screens etc…)</li>
</ul>

<p><em>Q: I don’t really get the concept of depleted shRNA, don’t we only look at the amount of cells? do we look at how many shRNA are left to have a notion of usage?</em></p>

<hr />
<p>Transformation: The process by which a normal cell acquires cellular phenotypes of a cancer cell</p>

<p>ORF: Open Reading Frame</p>

<p>xenograft: transplanting tissue into another organism
PDX: patient derived xenograph</p>

<p>primary cells: cells coming directly from living tissues</p>

<hr />

<h2 id="a-combination-cdk46-and-igf1r-inhibitor-strategy-for-ewing-sarcoma--lillian-m-guenther-neekesh-v-dharia-linda-ross-amy-conway1-amanda-l-robichaud">A Combination CDK4/6 and IGF1R Inhibitor Strategy for Ewing Sarcoma – Lillian M. Guenther, Neekesh V. Dharia, Linda Ross, Amy Conway1, Amanda L. Robichaud…</h2>

<p>using ORF resistance screens and knowledge from the achilles project with in vitro and xenograft models.</p>

<p><em>it presents a wide variety of tests used to demonstrate such things using mainly experimental wet-lab(biocell) procedures</em></p>

<hr />
<p>SA-b-gal staining: detecting secent cells (dying)
nucleation: self assembly</p>

<hr />

<h2 id="core-transcriptional-regulatory-circuit-controlled-by-the-tal1-complex-in-human-t-cell-acute-lymphoblastic-leukemia-takaomi-sanda-lee-n-lawton-m-inmaculada-barrasa-peng-fan-holger-kohlhamme">Core Transcriptional Regulatory Circuit Controlled by the TAL1 Complex in Human T Cell Acute Lymphoblastic Leukemia– Takaomi Sanda, Lee N. Lawton, M. Inmaculada Barrasa, Peng Fan, Holger Kohlhamme…</h2>

<p>identifying it with other TF binders and position allowed to find a feedback circuit comprised of GATA3, and RUNX1 activating MYB. which shows the supreme importance of TAL1 in this case. In fine understanding some of the mechanisms behind T ALL.</p>

<hr />
<p>primagraft: xenograft without any pre in vitro culture</p>

<p>CCRF-CEM: 	A T lymphoblastoid line obtained from the peripheral blood of a 4 year old Caucasian female with acute lymphoblastoid leukaemia. These may be grown to a high density in a spinner type suspension culture. Cells are reported to be free of virus particles as determined by electron microscopy.</p>

<p>ectopic: abnormal</p>

<hr />

<h2 id="v-net-fully-convolutional-neural-networks-for-volumetric-medical-image-segmentation-fausto-milletari1-nassir-navab-seyed-ahmad-ahmadi">V-Net: Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation– Fausto Milletari1, Nassir Navab, Seyed-Ahmad Ahmadi</h2>

<p>Presenting a new model that relies solely on 3D convolution/deconvolution to present a heat map of probability to be in foreground. It achieves very good results but might lack training data compared to 2D systems which can reuse learnings. It uses a lot of feature wise transformation and no maxpooling(only conv).</p>

<p>*N: it seems that it takes too few pixels to learn interesting features (might take NxMxM pixels with N&lt;M)</p>

<p><em>N: important use of <a href="https://distill.pub/2018/feature-wise-transformations/">featurewise transfo</a></em></p>

<hr />
<p>foreground: the ROI (else background)</p>

<hr />

<h2 id="crispr-cas9-screen-reveals-a-mycn-amplified-neuroblastoma-dependency-on-ezh2-liying-chen-gabriela-alexe-neekesh-v-dharia-linda-ross-amanda-balboni-ini">CRISPR-Cas9 screen reveals a MYCN-amplified neuroblastoma dependency on EZH2– Liying Chen, Gabriela Alexe, Neekesh V. Dharia, Linda Ross, Amanda Balboni Ini…</h2>

<p>Here we can see an analysis that start to be recurrent to other related papers where results from achilles and now from crispr screens allows the test of a pathway / transcruption factor. here EZH2 in MYCN amplifyed neuroblsotame. The analsysis tries to identify druggable mechanisms, correlations to other protein expressions and TFs.</p>

<blockquote>
  <p>We assessed the clinical relevance of EZH2 inhibition, both in vitro and in vivo, using multiple perturbation approaches, such as selective small-molecule EZH2 inhib- itors, shRNA-mediated knockdown, and CRISPR-Cas9 knockout. We also studied the epigenetic networks and their response to EZH2 inhibition in MYCN-amplified neuroblastoma and explored the underlying molecular mechanisms of the sensitivity to EZH2 inhibitors using biochemical, genomic, and epigenom- ic approaches.</p>
</blockquote>

<h2 id="the-variability-of-expression-of-many-genes-and-most-functional-pathways-is-observed-to-increase-with-age-in-brain-transcriptome-data-veronika-r-kedlian-handan-melike-donertas-janet-m-thornton">The variability of expression of many genes and most functional pathways is observed to increase with age in brain transcriptome data.– Veronika R. Kedlian, Handan Melike Donertas, Janet M. Thornton</h2>

<p>Knowing that there is evidence for increaser in variability of expression accross cell lines in the human genome with age. Researchers looked at brain cells (frontal/pre frontal) and discovered a slight increase in the variability of expression accross different techniques (measures of variations, comparisons)
They <em>most importantly</em> did a go search and discovered that this increase in variability was mostly for genes related to 
ageing processes.</p>

<hr />
<p>BP GO – Biological Process Gene Ontology,
DE – differentially expressed genes,
DV – differentially variable genes,
Dvar – measure of change in the expression variability with age, GO – Gene Ontology,
GSEA – Gene Set Enrichment Analysis,
IQR – interquartile range,
KEGG – Kyoto Encyclopedia of Genes and Genomes, PMI – post-mortem interval,
QN – quantile normalization,
RIN – RNA integrity number,
Rho – Spearman correlation estimate,
SVA – Surrogate Variable Analysis</p>

<hr />

<h2 id="ewsfli-confers-tumor-cell-synthetic-lethality-to-cdk12-inhibition-in-ewing-sarcoma-amanda-balboni-iniguez-bjorn-stolte-emily-jue-wang-">EWS/FLI Confers Tumor Cell Synthetic Lethality to CDK12 Inhibition in Ewing Sarcoma– Amanda Balboni Iniguez, Bjorn Stolte, Emily Jue Wang, …</h2>

<blockquote>
  <p>Here we report that Ewing sarcoma is a disease with exquisite sensitivity to the covalent small-molecule CDK7/12/13 inhib- itors, THZ1 and THZ531. These compounds act, in part, through inhibition of CDK12 and the subsequent suppression of genes involved in DNA damage repair</p>
</blockquote>

<p>explained by the fusion protein EWS/FLI</p>

<h2 id="cell-detection-in-microscopy-images-with-deep-convolutional-neural-network-and-compressed-sensing-yao-xue-and-nilanjan-ray">Cell Detection in Microscopy Images with Deep Convolutional Neural Network and Compressed Sensing– Yao Xue and Nilanjan Ray</h2>

<p>using compressed sensing in two different manners to encode the position of features in microscopy imaging of cells thus reducing dimensions and possible prediction error from a vanilla CNN. looking at theoretical proof the CNNCS works and experimental demonstration on different datasets achieving sota results.</p>

<h2 id="cosmic-exploring-the-worlds-knowledge-of-somatic-mutations-in-human-cancer-simon-a-forbes-david-beare-prasad-gunasekaran-kenric-leung-nidhi-bindal-harry-boutselakis-minjie-ding">COSMIC: exploring the world’s knowledge of somatic mutations in human cancer– Simon A. Forbes, David Beare, Prasad Gunasekaran, Kenric Leung, Nidhi Bindal, Harry Boutselakis, Minjie Ding…</h2>

<p>A description of the COSMIC database</p>

<h2 id="impact-genomic-annotation-of-cell-state-specific-regulatory-elements-inferred-from-the-epigenome-of-bound-transcription-factors-tiffany-amariuta-yang-luo-steven-gazal-emma-e-davenport">IMPACT: Genomic Annotation of Cell-State-Specific Regulatory Elements Inferred from the Epigenome of Bound Transcription Factors– Tiffany Amariuta, Yang Luo, Steven Gazal, Emma E. Davenport…</h2>

<p>A similar work to what Brendan Frey has been doing with neural networks but with logistic regressing Except here, it is trying to infer bound AND unbound region from sequence and more explicitly, infer regulatory power of each nucleotides for a particular cell state and TF given a set of regulatory features such as open Chromatin, methylation marks etc…</p>

<h2 id="extensive-variation-in-chromatin-states-across-humans-maya-kasowski-sofia-kyriazopoulou-panagiotopoulou-fabian-grubert-judith-b-zaugganshul-kundaje-yuling-liu-alan-p-boyle-qiangfeng-cliff-zhang-fouad-zakharia-damek-v-spacek-jingjing-li-dan-xie-anthony-olarerin-george-lars-m-steinmetz-john-b-hogenesch-manolis-kellis-serafim-batzoglou">Extensive Variation in Chromatin States Across Humans– Maya Kasowski, Sofia Kyriazopoulou-Panagiotopoulou, Fabian Grubert, Judith B. Zaugg,Anshul Kundaje, Yuling Liu, Alan P. Boyle, Qiangfeng Cliff Zhang, Fouad Zakharia, Damek V. Spacek, Jingjing Li, Dan Xie, Anthony Olarerin-George, Lars M. Steinmetz, John B. Hogenesch, Manolis Kellis, Serafim Batzoglou</h2>

<p>Paper presenting the early finding in chromatin variation accross human genre without much variation in gene expression (can be related to cell type/state?). Some marks are highly inherited. They also showed that places of this variability are also high GWAS region. Which is a base results of enhancers and chromatin modification</p>

<h2 id="chromhmm-automating-chromatin--state-discovery-and-characterization-jason-ernst-manolis-kellis">ChromHMM: automating chromatin- state discovery and characterization– Jason Ernst, Manolis Kellis</h2>

<p>A presentation of ChromHMM which is a markovian models to infer chromatin modification and create chrom states from reads from Chip seq of this chromatin. (denoising kinda?)</p>

<h2 id="histone-modifications-at-human-enhancers-reflect-global-cell-type-specific-gene-expression-nathaniel-d-heintzman-gary-c-hon-r-david-hawkins1-pouya-kheradpour-alexander-stark-manolis-kellis">Histone modifications at human enhancers reflect global cell-type-specific gene expression– Nathaniel D. Heintzman, Gary C. Hon, R. David Hawkins1, Pouya Kheradpour, Alexander Stark, Manolis Kellis…</h2>

<p>another landmark paper from 2009 explaining the viariability in chromatin modification, its link with TFBS regulatory power and (for some of them H3K27 for ex) its implication in specific cellular states.</p>

<h2 id="strelka2-fast-and-accurate-calling-of-germline-and-somatic-variants-sangtae-kim-konrad-scheffler-aaron-l-halpern-mitchell-a-bekritsky">Strelka2: fast and accurate calling of germline and somatic variants– Sangtae Kim, Konrad Scheffler, Aaron L. Halpern, Mitchell A. Bekritsky</h2>

<p>a variant caller for germline and somatic sequencing. Improves liquid model analysis mainly since take in account P() of being contaminated. It models sequencing noise via e (1) the genotype probability computed by the core variant probability model, (2) root-mean-square mapping quality,(3) strand bias, (4) the fraction of reads consistent with locus haplo- type model, and (5) the complexity of the reference context.</p>

<h2 id="star-ultrafast-universal-rna-seq-alignerpacbio">STAR: ultrafast universal RNA-seq aligner–PacBio</h2>

<p>star is the first and most used algorithm specifically designed for aligning rna seq data (especially paired end reads) from 3rd gen sequencer (longer reads). it is 30 times faster than previous algo and uses a two step algo:</p>
<ul>
  <li>seed searching (Maximal mappable array) implemented through uncompressed suffix arrays which account for mismatches, low quality read ends, multiple solutions and gaps (introduced by splicing)</li>
  <li>Clustering, stitching and scoring based on proximity of each read, accounting for overlapping reads and distant reads based on a user provided threshold</li>
</ul>

<h2 id="star-fusion-fast-and-accurate-fusion-transcript-detection-from-rna-seq-brian-j-haas1-alex-dobin-nicolas-stransky-bo-li-aviv-regev">STAR-Fusion: Fast and Accurate Fusion Transcript Detection from RNA-Seq– Brian J. Haas1, Alex Dobin, Nicolas Stransky, Bo Li, Aviv Regev…</h2>

<p>star fusion uses the output from star on porblematic mappings to reprocess the reads in order to find potential fusions, remmapping based on coordinate overlap (using interval trees). 
Finally it filters the output by removing fusions with minimal supports (depending on the support type), merging closeby fusions, removing lowly supported isoforms. removing too similar gene fusion isoforms and when there is too many fusion for the same gene.</p>

<h2 id="title-exploring-genetic-interaction-manifolds-constructed-from-rich-phenotypes--thomas-m-norman-max-a-horlbeck-joseph-m-replogle">Title: Exploring genetic interaction manifolds constructed from rich phenotypes – Thomas M. Norman, Max A. Horlbeck, Joseph M. Replogle</h2>

<p>A presentation of a vision to repoduce from perturb seq screen + transcriptome analysis, the genetic interaction of each genes. From this, guess the manifold of cell state. 
They present a first simple and criticable example of it but address more a vision and an idea than anything else. Are perturb seq accurate enough for guessing interactions? Can all possible state be estimated from such method?</p>

<h2 id="rsem-accurate-transcript-quantification-from-rna-seq-data-with-or-without-a-reference-genome-bo-li-colin-dewey">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome– Bo Li, Colin Dewey</h2>

<p>this package used in the GTEx pipeline uses EM algo to quantify gene expression given alignment or reconstructing the transcriptome from scratch. EM is mainly used to take in account isoforms and give Bayesian posterior probability. It takes in account quality score of reads and gives CI. It can also be used as a generative model from the weights learnt from each gene</p>
<blockquote>
  <p>Taking advantage of this module, we have deter- mined that a large number of short SE reads is best for gene-level quantification, while PE reads may improve within-gene isoform frequencies for the mouse and human transcript sets.</p>
</blockquote>

<h2 id="sensitive-detection-of-somatic-point-mutations-in-impure-and-heterogeneous-cancer-samples"><a href="https://www.nature.com/articles/nbt.2514">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples</a></h2>

<h2 id="ngsea-network-based-gene-set-enrichment-analysis-for-interpreting-gene-expression-phenotypes-with-functional-gene-sets-heonjong-han-sangyoung-lee-and-insuk-lee">NGSEA: network-based gene set enrichment analysis for interpreting gene expression phenotypes with functional gene sets– Heonjong Han, Sangyoung Lee, and Insuk Lee</h2>

<p>This method is using the GSEA method (with AE) and rescore these by adding the average value of their connected gene in the network given by https://www.inetbio.org/humannet/ they then showed that it was improving on regular GSEA and discovered more data than what Broad’s Cmap found for drug targets for example.</p>

<h2 id="reverse-engineering-cellular-networks-califano-lab-aracne">Reverse engineering cellular networks– Califano Lab (ARAcne)</h2>

<p>Presentation of this algorithm that can compute Mutual information between pairs of genes categorized by concentration of their related output molecules. This then NxN matrix can then be used to find networks. these are then filtered on confidence interval and DPIs to create resulting network. the more condition on the cell the better the results. <em>might be interesting to use on Apollo data</em></p>

<h2 id="limma-powers-differential-expression-analyses-for-rna-sequencing-and-microarray-studies-matthew-e-ritchie-belinda-phipson-di-wu">limma powers differential expression analyses for RNA-sequencing and microarray studies– Matthew E. Ritchie, Belinda Phipson, Di Wu…</h2>

<p>a presentation of the tool suite based on analysis of count data from gene products of different assay types. Based on a GLM taking in account a wide range of dependencies between genes, experiments, assays etc.. and allowing to analysis data at all stages of processing. 
It is presented pre processing stage of RNA seq and micro array data. DE computations. Expr. Prof. Association, Hypthesis testing</p>

<h2 id="an-oncogenic-super-enhancer-formed-through-somatic-mutation-of-a-noncoding-intergenic-element-marc-r-mansour-brian-j-abraham-lars-anders-alla-berezovskaya-alejandro-gutierrez-adam-d-durbin">An oncogenic super-enhancer formed through somatic mutation of a noncoding intergenic element– Marc R. Mansour, Brian J. Abraham, Lars Anders, Alla Berezovskaya, Alejandro Gutierrez, Adam D. Durbin</h2>

<p>A following paper on the reason behind an overexpression of a TF which drives oncogenecity in AML, which can be explained by the binding of MYB opening chromatine and  due to a mutation on non coding region creating an MYB TFBS.</p>

<h2 id="aging-is-associated-with-a-systemic-length-driven-transcriptome-imbalance"><a href="https://www.biorxiv.org/content/10.1101/691154v1.full">Aging is associated with a systemic length-driven transcriptome imbalance</a></h2>
<p>high length transc. is associated with gene regulation and dna binding. It may be seen as a non critical state of the cell. This is caused by the many different insults in the cell that forces it, little by little to act in this critical state it cannot escape.</p>

<h2 id="predicting-splicing-from-primary-sequence-with-deep-learning-kishore-jaganathan-sofia-kyriazopoulou-panagiotopoulou-jeremy-f-mcrae-siavash-fazel-darbandi-david-knowles-yang-i-li-jack-a-kosmicki-juan-arbelaez-wenwu-cui-grace-b-schwartz-eric-d-chow-efstathios-kanterakis-hong-gao-amirali-kia-serafim-batzoglou-stephan-j-sanders-and-kyle-kai-how-farh">Predicting Splicing from Primary Sequence with Deep Learning– Kishore Jaganathan, Sofia Kyriazopoulou Panagiotopoulou, Jeremy F. McRae, Siavash Fazel Darbandi, David Knowles, Yang I. Li, Jack A. Kosmicki, Juan Arbelaez, Wenwu Cui, Grace B. Schwartz, Eric D. Chow, Efstathios Kanterakis, Hong Gao, Amirali Kia, Serafim Batzoglou, Stephan J. Sanders, and Kyle Kai-How Farh</h2>

<p><a href="https://docs.google.com/presentation/d/1uMa0qYcTopxR6AQVouirs_MDCnCEvw5Wls5Ap_FUtCY/edit#slide=id.g59623d53cc_0_81">presentation</a></p>

<h2 id="network-based-inference-of-protein-activity-helps-functionalize-the-genetic-landscape-of-cancer-mariano-j-alvarez-yao-shen-federico-m-giorgi-califano-lab">Network-based inference of protein activity helps functionalize the genetic landscape of cancer– Mariano J. Alvarez, Yao Shen, Federico M. Giorgi, Califano Lab…</h2>

<p><em>Aracne outperforms Chip experiments, only with RNA seq data</em>
-&gt; does not get the same information thought: Regulators and regulons is different from TFB and DNA acetylation
Knowing regulons (genesets regulated by a regulator) we can then express a set of statistical analysis whereby looking at rna seq data, estimating the effect of regulons, from the expression of genes and of regulators of these genes. trying to remove other effects. this is done by comparing RNAseq to other batches of RNA seq to get predictive power. Then can be used Enrichment analysis test on ranks of expression to determine the effect of specific genes on regulons and specific regulons on genes. This acts as a network inference of “protein” (not really) “activity” 
An interesting thing is to be able to infer mutations that are likely to cause up or down regulation effect in this network. (CCLE)</p>

<p>All in all it tries to unsupervisely attain the same result as CRC analysis (it might get btw more predictive power if using a hiererchical system based on CRCs(and here you need ChIP))</p>

<h2 id="three-dimensional-total-internal-reflection-fluorescence-nanoscopy-with-sub-10-nm-resolution"><a href="https://www.biorxiv.org/content/10.1101/693994v1.full">Three-dimensional total internal reflection fluorescence nanoscopy with sub-10 nm resolution</a></h2>

<p>getting sub-10nm resolution in the 3D. an amelioration from super res DNA paint. by having a computation of distance to both emiting surface and receiving surface bu knowing the internal physics of the two. 
enabling the direct recognition of protein assemblies at the molecular level.</p>

<h2 id="inferring-protein-3d-structure-from-deep-mutation-scans-nathan-j-rollins15-kelly-p-brock125-frank-j-poelwijk">Inferring protein 3D structure from deep mutation scans– Nathan J. Rollins1,5, Kelly P. Brock1,2,5, Frank J. Poelwijk</h2>

<p>Using mutation scans where mutating and translating an rna will lead to a protein of a likely different affinity to a kown bindinig protein. If the change of two position leads to a different change expected from adding the unique change at these two positions. Then we expect the two regions to be near each other in the protein.
Using that they were able to prove that it works for region of highest increase. But did not show recreation of known protein structure from this data.</p>

<h2 id="deep-learning-at-base-resolution-reveals-motif-syntax-of-the-cis-regulatory-code-kundaje-lab">Deep learning at base-resolution reveals motif syntax of the cis-regulatory code– Kundaje lab</h2>

<blockquote>
  <p>These results indicate that while local sequence context (1 kb) is sufficient to accurately capture the shape of ChIP-nexus profiles and positions of binding footprints, longer sequences or other measurements such as local chromatin state may be required to better predict TF occupancy (45)</p>
</blockquote>

<p>Using a CNN with deconv layer in the end to predict binding profile from sequence based on chip exo data (like chip seq but on a per bp basis). They found it learnt syntax specific binding profile. most of the time unregarding on the cell type (i.e. the current binding profile which is also influenced by the chromatin state). Inspecting the learnt parameters with many toolings and domain knowledge. They could find cooperative binding in step specific manner based on motifs spacing, motif linked to binding of other TFs, matching footprints. They found specific binding to tRNA genes of oct4. influence of transposons and filtering of them based on Information content of the learnt motif instances. periodic patterns in nanog binding showing behavior of nucleosome unwinding. All in all making of rules of the cis regulation syntax shown in theory and some experiment. such as the TF collective model, billboard model enhanceosome model.</p>

<h2 id="hallmarks-of-cancer-the-next-generation-douglas-hanahan12-and-robert-a-weinberg3">Hallmarks of Cancer: The Next Generation– Douglas Hanahan1,2,* and Robert A. Weinberg3,*</h2>

<p>We present again the 6 hallmarks, with new major pathways insights into each with a clearer picture of the linked cell circuitry. It also showed 2 additional hallmarks and 2 enabling characteristics that have emerged in the past decade: reprogrammation of the energy metabolism, evading immune destruction, genome instability tumor promoting inflammation. A new view on the importanfce of the tumor micro environment s gven with CSC, endothelial C, Pericytes, Immune infla. C, Cancer Assoc. Fibroblasts, C of the tumor stroma..
And a view of the resulting extracellular signaling network. which drives, e.g. metastatic dissemination.<br />
Summarizing the research and knowledge in cancer biology of the past 15 years it shows the promise of targeted therapies and how to think about them and the future of cancer research.</p>

<h2 id="next-generation-characterization-of-the-cancer-cell-line-encyclopedia-cds">Next-generation characterization of the Cancer Cell Line Encyclopedia– CDS</h2>

<p>Here is presented a brief summary of the depmap omics project (previously CCLE), trying to do multi omics analysis of thousands of cancer cell lines. Here Mahmoud Ghandi correlated each different dataset with an other or with exome wide RNAi screens (project achilles) to find a some biological signal.</p>

<h2 id="off-target-toxicity-is-a-common-mechanism-of-action-of-cancer-drugs-undergoing-clinical-trials-ann-lin-cds-jc">Off-target toxicity is a common mechanism of action of cancer drugs undergoing clinical trials– Ann Lin @CDS JC</h2>

<p>It seems to be a wake up call for pharmaceutical companies and an example of zombie effect in research. Some stragihtfoward test showed that the mechanism was off target for RNAi discovered dependencies and that the drugs were, as well. It showed a lot of the reasons why and hopes to place the burden of proof to drug companies. to recentrate efforts on understanding MOA and to get more big data information on rnai and use cripr tool as much as possible today.</p>

<h2 id="scalable-whole-exome-sequencing-of-cell-free-dna-reveals-high-concordance-with-metastatic-tumors-viktor-adalsteinsson-gavin-ha-samuel-freeman-atish-choudhury">Scalable whole-exome sequencing of cell-free DNA reveals high concordance with metastatic tumors– Viktor Adalsteinsson, Gavin Ha, Samuel Freeman, Atish Choudhury</h2>

<p>IN this document, it is shown that with sufficiently good models of quality of DNA from blood biopsies, it is possible to achieve very similar results to regular DNA analysis of biopsies. This is proven by doing correlation analysis and using the data for application of target discovery, neoantigen therapies. Moreover it is shown that by its broad sampling of the body’s activity, cfDNA seems to be better in discovering subclonal population and finding mechanism of resistance of these populations, thus allowing better prediction of resistance to therapies</p>

<h2 id="organization-and-regulation-of-gene-transcription-patrick-cramer">Organization and regulation of gene transcription– Patrick Cramer</h2>

<p>A review of the mechanism of gene transcription with chromatine and transcription factor at the biomolecular level. Making sense of transcription initiation, chromatine openning, transcription pausing, splicing, end of transcription and the auto organization of TF into condensates at specific tads to recruit the T Initiation C and others, to recruit the T elongation Factors.</p>

<h2 id="dynamic-regulation-of-transcriptional-states-by-chromatin-and-transcription-factors-ty-c-voss-and-gordon-l-hager">Dynamic regulation of transcriptional states by chromatin and transcription factors– Ty C. Voss and Gordon L. Hager</h2>

<p>This paper presents the idea that fine grained transcriptional modification occurs often and starts to be discovered with new imaging techniques and modeled. This can only occurs thanks to   chromatin remodelers allowing regions to be accessed by transcriptional factories. It also shows the machinery in the context of transcriptional burst and hit and run model. with cobinding explanation and the multi range of inhibition.</p>

<h2 id="liquid-versus-tissue-biopsy-for-detecting-acquired-resistance-and-tumor-heterogeneity-in-gastrointestinal-cancers-aparna-r-parikh-ignaty-leshchiner-liudmila-elagina">Liquid versus tissue biopsy for detecting acquired resistance and tumor heterogeneity in gastrointestinal cancers– Aparna R. Parikh, Ignaty Leshchiner, Liudmila Elagina</h2>

<p>showing off the ability of blood biopsy to detect a lot of clonal and sub clonal population and find some of the mutations and copy number relevant for cancer resistance mechanisms. allowing to find a better set of targeted therapies. (in depth)</p>

<h2 id="activity-by-contact-model-of-enhancer-specificity-from-thousands-of-crispr-perturbations-fulco-lander">Activity-by-Contact model of enhancer specificity from thousands of CRISPR perturbations– Fulco, Lander..</h2>

<p>A presentation of a new model of enhancer-gene interaction based on simple dynamics extracted from thousands of crisprI experiments on enhancers.</p>

<h2 id="passenger-hotspot-mutations-in-cancer-julian-m-hess-andre-bernards-jaegil-kim">Passenger Hotspot Mutations in Cancer– Julian M. Hess, Andre Bernards, Jaegil Kim</h2>

<p>a presentation of a model of bias in mutations on a bp level a contrario to most models on gene level. This is helped by the amount of wes profiling of cancer tissue. They show that some base level bias is at play based on simple metrics and provide a new statistical model to estimate background noise based on base level information of closest bases and test it on bases of known effect (deleterious in cancer genomes) log normal poisson</p>

<h2 id="scaling-computational-genomics-to-millions-of-individuals-with-gpus-amaro-taylor-weiner-françois-aguet">Scaling computational genomics to millions of individuals with GPUs– Amaro Taylor-Weiner, François Aguet</h2>

<p>They used pytorch and tensorflow to do reimplement existing algorithms.</p>

<h2 id="a-genetics-led-approach-defines-the-drug-target--landscape-of-30-immune-related-traits-hai-fang-the-ultra-dd-consortium">A genetics-led approach defines the drug target  landscape of 30 immune-related traits– Hai Fang, The ULTRA-DD Consortium</h2>

<p>This paper present a complex pipeline to do target prioritization based on pop genetics gwas studies genes of interest, annotation of them based on groupings in HiC data (co expression) mutation data etc… Then creating a graph based on PPI data. (more than just pathways but groups of interacting ones)
Based on this they validate that they find all of the most famous targets for common immune diseases such as arthrytis.
Calls to the higher need for annotation in depmap’s prioritization.</p>

<h2 id="structure-based-function-prediction-using-graph-convolutional-networks-ian-m-fisk-rick-bonneau">Structure-Based Function Prediction using Graph Convolutional Networks– Ian M Fisk, Rick Bonneau</h2>

<p>Presentation of two models. One using a language model LSTM to predict contact maps from protein 1st structure. One using a graphConvNet on the graph network based on a thresholding of a contact map, to predict GO terms i.e. functions of proteins and a tool to do feature extraction via activation map to decipher the residues that are predictive of function and which are often true known binding sites.</p>

<h2 id="visualization-and-analysis-of-single-cell-rna-seq-data-by-kernel--based-similarity-learning-junjie-zhu-serafim-batzoglou">Visualization and analysis of single-cell RNA-seq data by kernel- based similarity learning– Junjie Zhu, Serafim Batzoglou</h2>

<p>presentation of a algorithm to learn a similarity matrix for sc RNAseq based on a finding a solution to an equation with 3 learned parameters, including the similarity matrix and a cluster structure by refining one of each paramterers sequentially until convergence. It also contains a diffusion step. It is a mathematical model based on some prior on the data (scRNA like structure and presence of clear clusters). It can be used for as inputs to dim reduction tasks and clustering tasks.</p>

<h2 id="umap-uniform-manifold-approximation-and-projection-for-dimension-reduction">UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction–</h2>

<p>Umap is a dimensionality reduction algorithm. It takes points defined by vectors of high dimensions: 2D vector= (x,y), 3D vector= (x,y,z), ND vector= (x,y,z,a,b,c,…n). and puts them back in lower dimensions. This allows for visualizing point clouds in 2D and also finding a subspace/ base / manifold in which the cloud data lies. This can be better visualizing in the 3D to 2D case 
This reduced vector is also called an embedding of the first one.</p>

<p><img src="../../dimred.jpg" alt="scikit learn" />
view of learning the shape of a “sheet of paper” only by knowing a set of points that lie on it.</p>

<p>Unlike many other famous techniques (<a href="https://medium.com/@raghavan99o/principal-component-analysis-pca-explained-and-implemented-eeab7cb73b72">PCA</a>, <a href="d">MDS</a>…) it learns this structure by using local information about the points (neighbors), like <a href="https://medium.com/@raghavan99o/principal-component-analysis-pca-explained-and-implemented-eeab7cb73b72">t-SNE</a>.</p>

<p>Neighbor information is better encoded into a graph with edges representing distances to nearest points.</p>

<p>This distance is normalized by all distances this point has to its neighbors.</p>

<p>then for all the neighbors we create a set (a list of unique things): for all the edges. this set will be fuzzy as it also contains a value representing how much each neighbors is in the set of neighbors based on its distance to the center point.</p>

<p>We will then merge all the sets created for each point.
i.e. we will then merge the edges and keep only the ones with the lowest distances.</p>

<p>We then take each edges and considers them together as a graph.</p>

<p><em>Note that we are thinking here about graph of nodes and edges but the topological logic could be applied to the faces made by each set of 3 edges and the volumes made by each set of 4 faces, etc …</em></p>

<p>We then take the core components of this graph using <a href="https://scikit-learn.org/stable/auto_examples/cluster/plot_cluster_comparison.html#sphx-glr-auto-examples-cluster-plot-cluster-comparison-py">spectral embedding</a>, where we basically take the <a href="https://www.youtube.com/watch?v=PFDu9oVAE-g">eigenvector</a> of a <a href="https://en.wikipedia.org/wiki/Adjacency_matrix">matrix of adjacency</a> (a square of numbers where rows represent the nodes of the graph and the columns represent the same thing and the values of the matrix are the distance between, each point)</p>

<p>It is a <a href="https://www.youtube.com/watch?v=PFDu9oVAE-g">decomposition</a> of the information into two things, one that define a base for the object to exist in and the other defining the layout of the object. you need both to reconstruct the object.</p>

<p>Here we will only take the first 2 most important value of the base and considers it as what defines our object in 2 dimension (it can be any number lower than the initial number of dimension). The two values here are the dimensions and their dimensions are each of the different points</p>

<p>Finally we will try to improve this first version by a <a href="https://en.wikipedia.org/wiki/Mathematical_optimization">minimization</a> process. There, we want to apply a physical-like force between the nodes so that close connected edges get closer together (up to a minimum distance)</p>

<p><img src="../../images/umap.png" alt="" /> 
we can see the relation to the number of samples used and the max distance to create graph edges.</p>

<p>All in all, there is a theoretical explanation of the algorithm that can be seen in terms of learning <a href="https://en.wikipedia.org/wiki/Fuzzy_logic">fuzzy</a> <a href="https://en.wikipedia.org/wiki/Simplicial_set">simplicial sets</a> defining the local structure of the high dimensional data and of the lower dimensional one and making the two sets converge.</p>

<p>The explanation defined in part 2 of the paper can be understood by anyone after reading <a href="https://books.google.com/books?id=xbL11KfgiEAC&amp;printsec=frontcover&amp;hl=fr&amp;source=gbs_ge_summary_r&amp;cad=0#v=onepage&amp;q&amp;f=false">this book</a> which will allow the reader to gain the vocabulary to understand the wikipedia pages defining the vocabulary used in the paper. :p</p>

<p>More information about Riemannian topology can then be learned <a href="http://www.math.ens.fr/~feydy/Teaching/geometrie_riemannienne_espaces_de_formes.pdf">here</a>. It is in French however ;)</p>

<h2 id="reproducible-pharmacogenomic-profiling-of-cancer-cell-line-panels-peter-m-haverty-eva-lin-jenille-tan2-yihong-yu">Reproducible pharmacogenomic profiling of cancer cell line panels– Peter m. Haverty, Eva Lin, Jenille Tan2, yihong yu…</h2>

<p>A presentation of a meta analysis and replication test of large scale pharmacogenomic profiling of CCLs both at sanger and broad by independant panel as a response to another research. It showed that flaws in the critics of the reproducibiliity existed as it was not judging on the full dataset and it was not looking at the same thing as what the large scale studies where interested in.It also shows that it is able to correlate both results by doind the analysis differently and to reproduce them (with more correlation to broad’s)</p>

<h2 id="ezh2-oncogenic-mutations-drive-epigenetic-transcriptional-and-structural-changes-within-chromatin-domains-maria-c-donaldson-collier15-stephanie-sungalee-epfl">EZH2 oncogenic mutations drive epigenetic, transcriptional, and structural changes within chromatin domains– Maria C. Donaldson-Collier1,5, Stephanie Sungalee, EPFL</h2>

<p>This group is analysing an oncogenic mutation in a gene which is implied in the presence of histone modification at key location. They show that this mutation cause the presence of this modification everywhere and mostly in some sets of genes that are within Topologically associating domains. They found that the TADs might change a bit based on this modification and thus, change the transcriptional status of genes which are themselves tied to oncogenic states.</p>

<h2 id="master-transcription-factors-and-mediator-establish-super-enhancers-at-key-cell-identity-genes-young-lab">Master Transcription Factors and Mediator Establish Super-Enhancers at Key Cell Identity Genes– Young Lab</h2>

<p>A paper I had already read before discovering and explaining what are super enhancers and how they differ from regular ones. It is in the context of muring ESCs. We can see their analysis by size. location in TAds, circuitry binding to TF and to cell state. cobinding and Mediator content. etc..</p>

<h2 id="diggit-a-bioconductor-package-to-infer-genetic-variants-driving-cellular-phenotype--mariano-j-alvarez-james-c-chen-and-andrea-califano">DIGGIT: a Bioconductor package to infer genetic variants driving cellular phenotype – Mariano J. Alvarez, James C. Chen and Andrea Califano</h2>

<p>A very small paper. more of an abstract that does not explain the algorithm but shows a way to improve gwas studies by improving QTL estimation based on knowledge of master regulators and impacts of CNV on expression. The paper was not precise enough for me to make any judgment on the logic. It is not a good paper.</p>

<h2 id="moderated-estimation-of-fold-change-and-dispersion-for-rna-seq-data-with-deseq2--michael-i-love-wolfgang-huber-and-simon-anders">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2 – Michael I Love, Wolfgang Huber and Simon Anders</h2>

<p>Here is a presentation of DESeq2 which improves on the previous version as it is trying to shrink the variance in the data based on the amount of support you get from the data itself. This will reduce the estimate of the fold change based on fitting the data to a NB distribution in a bayesian framework where the estimates of a MaxLikelyhood round are fitted again to a distribution with a prior as a zero centered Normal with a MAP round. The confidence values and the  ah ok ones pass a Wald Test which provide a z stat.</p>

<h2 id="tads-as-the-caller-calls-them-elzo-de-wit">TADs as the caller calls them– Elzo de Wit</h2>

<p>An approach to TADs consisting as a litterature review and opinion on the origin of TADs (as extrusion of cohesin loops) and their actual mechanisms. It serves as to diminish certain claims about the role of the TADs and precise their existence as a readout of a specific biological process. Instead of as a specific process themselves.</p>

<h2 id="understanding-and-improving-interpolation-in-autoencoders-via-an-adversarial-regularizer-david-berthelot-ian-goodfellow">UNDERSTANDING AND IMPROVING INTERPOLATION IN AUTOENCODERS VIA AN ADVERSARIAL REGULARIZER– David Berthelot, Ian Goodfellow…</h2>

<p>This paper presented a novel strategy to better constraint the results of an autoencoder by enforcing it to produce a smooth transition between its learnt representations. This is done by using an adversarial procedure whereby a critic is trying to predict the interpolation amount and the encoder has to make the critic think that it is 0. This can help better learning the manifold of the data as shown in toy examples and MNIST results. However seems to still have some caveats in its current form.</p>

<h2 id="transcriptional-amplification-in-tumor-cells-with-elevated-c-myc-charles-lin-young-lab">Transcriptional Amplification in Tumor Cells with Elevated c-Myc– Charles Lin, Young Lab</h2>

<p>A paper that shows that a contrario to other TFs, cMYC promotes the expression of all expressed genes but not unexpressed ones and thus serves as a general amplifier of expression in a cell. Explaining its upregulation in many cancer genomes. &lt;– Has been shown as wrong in the paper SLAM-seq defines direct gene-regulatory functions of the BRD4-MYC axis</p>

<h2 id="slam-seq-defines-direct-gene-regulatory-functions-of-the-brd4-myc-axis-matthias-muhar-anja-ebert">SLAM-seq defines direct gene-regulatory functions of the BRD4-MYC axis– Matthias Muhar, Anja Ebert</h2>

<p>A paper using slamseq to show that MYC only regulates a set of house keeping genes which can however appear as a global down-regulator/upregulator on less precise time scales.</p>

<h2 id="transcription-factors-activate-genes-through-the-phase-separation-capacity-of-their-activation-domains-ann-boija-isaac-a-klein-benjamin-r-sabari-youngs-lab">Transcription Factors Activate Genes through the Phase-Separation Capacity of Their Activation Domains– Ann Boija, Isaac A. Klein, Benjamin R. Sabari, Young’s lab</h2>

<p>Here is presented a work showing how the TF/Co-activator transcriptional condensate is organized and composed. This allows for a precise experimental definition of this condensate and its interaction in ESCs and shows how this could explain the MLL TFtoANY fusion proteins by the reliation of this condensate on proteins disordered regions which promotes this low level interaction between one another and the formation of condensate between TFsxCoact</p>

<h2 id="a-blueprint-for-advancing-genetics-based-cancer-therapy--bill-sellers">A Blueprint for Advancing Genetics-Based Cancer Therapy – Bill Sellers</h2>

<p>What is the problem of the drugability gap (what we now as oncogenic and what we can drug). How to do a “Human Cancer Genome”? Work on the understanding and targeting of resistance (by attacking the pathways in multiple locations). How to better use disease models (by high number of them and better model like GEM). And how that might translate to other diseases.</p>

<h2 id="genome-wide-somatic-variant-calling-via-colored-debrujin-graphs--giuseppe-narzisi-1-andré-corvelo1-kanika-arora">Genome-wide somatic variant calling via colored debrujin graphs- Giuseppe Narzisi 1, André Corvelo1, Kanika Arora</h2>

<p>This is a new tool, similar to strelka2 which does not yet work on tumor only. It also shows a great analysis of the performance of each other main algorithms mutect2/1 strelka2/1 LoFreq.</p>

<h2 id="atomnet-a-deep-convolutional-neural-network-for-bioactivity-prediction-in-structure-based-drug-discovery-atomwise-team">AtomNet: A Deep Convolutional Neural Network for Bioactivity Prediction in Structure-based Drug Discovery– atomwise team</h2>

<p>Achieves sota results on predicting bioactivity of small molecules in binding sites of proteins, based on a set of bioactivity databases. Issues of measuring quality since AUC is not perfect given the data type. The issue is of using this 3D data and converting it into something that can be used by the algo. Also having very similar inputs with very dissimilar activity (need to correct for this bias and other when processing your data) They are using a 3D small conv net.</p>

<h2 id="evaluating-variant-calling-tools-for-non-matched-next-generation-sequencing-data">Evaluating Variant Calling Tools for Non-Matched Next-Generation Sequencing Data–</h2>
<p>–&gt; for reproducibility little care in speed and overall consitence in accuracy through datasets noise level and read depth, we will still stick to the HaplotypeCaller from the GATK server in tumor only mode.</p>

<h2 id="identifying-gene-expression-programs-of-cell-type-identity-and-cellular-activity-with-single-cell-rna-seq-dylan-kotliar-adrian-veres-m-aurel-nagy">Identifying Gene Expression Programs of Cell-type Identity and Cellular Activity with Single-Cell RNA-Seq– Dylan Kotliar, Adrian Veres, M. Aurel Nagy</h2>

<p>They showed that NMF is the better tool overall to detect Genetic expression programs from test scRNAseq data when run many times and combinating the results. They call this combination cNMF.</p>
<blockquote>
  <p>Are they just trying to show me what NMF and bagging is?</p>
</blockquote>

<h2 id="deepcpg-accurate-prediction-of-single-cell-dna-methylation-states-using-deep-learning-christof-angermueller1-heather-j-lee-wolf-reik">DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning– Christof Angermueller1, Heather J. Lee, Wolf Reik</h2>

<table>
  <tbody>
    <tr>
      <td>A tools  to impute low-coverage methylation profiles for sets of cells and to discover DNA sequence motifs that are associated with methylation states and cell-to-cell variability from a bi input (CPG</td>
      <td>Mut -&gt; GRU</td>
      <td>CONV -&gt; FC-&gt; CPG). I would really have wished to see the actual specificity and sensitivity given that the algorithm</td>
    </tr>
  </tbody>
</table>

<h2 id="lighcpg-a-multi-view-cpg-sites-detection-on-single-cell-whole-genome-sequence-data">LighCpG: a multi-view CpG sites detection on single-cell whole genome sequence data</h2>

<p>It is like deepCpG but using the LightGBM algorithm to make its prediction. Does seem to be somewhat equivalent but faster.</p>

<h2 id="massively-multiplex-chemical-transcriptomics-at-single-cell-resolution-sanjay-r-srivatsan-josé-l-mcfaline-figueroa">Massively multiplex chemical transcriptomics at single-cell resolution– Sanjay R. Srivatsan, José L. McFaline-Figueroa</h2>

<h2 id="improved-protein-structure-prediction-using-potentials-from-deep-learning-deepmind">Improved protein structure prediction using potentials from deep learning– DeepMind</h2>

<p>finally deepmind presents its alpha fold algorithm and release it. The algorithm is using first MSA based sequence matching (from database) and then uses the first contact estimates, sequence specific information (and BLAST data) and output of a POTTS model as input to a very deep conv net with ELU layers and skip connections to refine the contact map. the predictions are then used by a non-convex differentiable model that will predict actual pairwise distances from the given data (which is: additional input from van der walls forces model, torsion distribution and distance distribution) using gradient descent (muliple trials with different noise input at the first step)</p>

<h2 id="pan-cancer-analysis-of-whole-genome-broad-toronto">PAN CANCER ANALYSIS OF WHOLE GENOME: Broad, Toronto,</h2>

<h3 id="a-deep-learning-system-can-accurately-classify-primary-and-metastatic-cancers-based">A deep learning system can accurately classify primary and metastatic cancers based</h3>

<p>Here researcher are presenting a simple NN trained on mutation features over the largest WGS tumor panel and show that methylation patterns guessed from passenger mutation distribution is very efficient. that sequencer/mut caller difference decrease their accuracy from 90% to 80%. That they can as well predict on metastasis than primary.</p>

<p>###</p>

<h2 id="growth-differentiation-factor-11-is-a-circulating-factor-that-reverses-age-related-cardiac-hypertrophy-francesco-s-loffredo-matthew-l-steinhauser-steven-m-jay">Growth Differentiation Factor 11 Is a Circulating Factor that Reverses Age-Related Cardiac Hypertrophy– Francesco S. Loffredo, Matthew L. Steinhauser, Steven M. Jay</h2>

<p>we can see that GDF-11 seems to be a factor to precent cardiac hypertrophy in aging.
This was an experimental study using proteomic analysis of blood from aged and young mice. doing heterochronic parabiosis and (trying to) controlling for blood pressure and behavioral changes post surgery.</p>

<h2 id="the-hallmarks-of-aging-carlos-lopez-otı-maria-a-blasco-linda-partridge">The Hallmarks of Aging– Carlos Lopez-Otı Maria A. Blasco, Linda Partridge</h2>

<p>We have a good overview showing the main research and knowledge into aging, very similar to cancer hallmark although less insight is available compared to cancer
A must read.</p>

<h2 id="predicting-deeper-into-the-future-of-semantic-segmentation">Predicting Deeper into the Future of Semantic Segmentation–</h2>

<p>They are reusing a well known semantic segmentation architexture called Dilation10 and used it in an autoregressive fashion (predicting one frame and the next …) and compared it to known methods to show that this approach was the most efficient.</p>

<h2 id="federated-learning-strategies-for-improving-communication-efficiency-jakub-konecˇny-h-brendan-mcmahan-google">FEDERATED LEARNING: STRATEGIES FOR IMPROVING COMMUNICATION EFFICIENCY– Jakub Konecˇny, H. Brendan McMahan, Google</h2>


        
      </section>

      <footer class="page__meta">
        
        


        

      </footer>

      

      
    </div>

    
  </article>

  
  
</div>

    </div>

    
      <div class="search-content">
        <div class="search-content__inner-wrap"><form class="search-content__form" onkeydown="return event.key != 'Enter';">
    <label class="sr-only" for="search">
      Enter your search term...
    </label>
    <input type="search" id="search" class="search-input" tabindex="-1" placeholder="Enter your search term..." />
  </form>
  <div id="results" class="results"></div></div>

      </div>
    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    
      
        
          <li><a href="mailto:jkobject@gmail.com" rel="nofollow noopener noreferrer"><i class="fas fa-fw fa-envelope-square" aria-hidden="true"></i> Email</a></li>
        
      
        
          <li><a href="https://twitter.com/jkobject" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
        
      
        
          <li><a href="https://github.com/jkobject" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-github" aria-hidden="true"></i> GitHub</a></li>
        
      
        
          <li><a href="https://ResearchGate.net/profile/jeremie_kalfon" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-researchgate" aria-hidden="true"></i> ResearchGate</a></li>
        
      
        
          <li><a href="https://Scholar.google.com/citations?user=zyXiydsAAAAJ" rel="nofollow noopener noreferrer"><i class="fas fa-fw fa-graduation-cap" aria-hidden="true"></i> Scholar</a></li>
        
      
        
          <li><a href="https://LinkedIn.com/in/jkobject" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i> LinkedIn</a></li>
        
      
    

    <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2020 Jérémie Kalfon. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js"></script>




<script src="/assets/js/lunr/lunr.min.js"></script>
<script src="/assets/js/lunr/lunr-store.js"></script>
<script src="/assets/js/lunr/lunr-en.js"></script>







              <script>
                window.onload = function () {
                    var script = document.createElement('script');
                    var firstScript = document.getElementsByTagName('script')[0];
                    script.type = 'text/javascript';
                    script.async = true;
                    script.src = '/sw-register.js?v=' + Date.now();
                    firstScript.parentNode.insertBefore(script, firstScript);
                };
            </script>
            </body>

</html>
