<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.22.0 by Michael Rose
  Copyright 2013-2020 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
<html lang="en" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>DepMap Omics: How to | Jérémie</title>
<meta name="description" content="How DepMap Omics data is made. How to process it Yourself">


  <meta name="author" content="Jérémie Kalfon">
  
  <meta property="article:author" content="Jérémie Kalfon">
  


<meta property="og:type" content="article">
<meta property="og:locale" content="en_US">
<meta property="og:site_name" content="Jérémie">
<meta property="og:title" content="DepMap Omics: How to">
<meta property="og:url" content="http://localhost:4000/projects/depmap-omics-howto/">


  <meta property="og:description" content="How DepMap Omics data is made. How to process it Yourself">



  <meta property="og:image" content="http://localhost:4000/assets/images/depmap-logo.png">



  <meta name="twitter:site" content="@jkobject">
  <meta name="twitter:title" content="DepMap Omics: How to">
  <meta name="twitter:description" content="How DepMap Omics data is made. How to process it Yourself">
  <meta name="twitter:url" content="http://localhost:4000/projects/depmap-omics-howto/">

  
    <meta name="twitter:card" content="summary">
    
      <meta name="twitter:image" content="http://localhost:4000/assets/images/depmap-logo.png">
    
  

  



  <meta property="article:published_time" content="2020-08-10T00:00:00-04:00">





  
    <meta property="article:publisher" content="https://www.facebook.com/jkobject">
  

  


<link rel="canonical" href="http://localhost:4000/projects/depmap-omics-howto/">




<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": "Jeremie Kalfon",
      "url": "http://localhost:4000/",
      "sameAs": ["https://twitter.com/jkobject","https://LinkedIn.com/in/jkobject","https://ResearchGate.com/jeremie-kalfon"]
    
  }
</script>






<!-- end _includes/seo.html -->



  <link href="/feed.xml" type="application/atom+xml" rel="alternate" title="Jérémie Feed">


<!-- https://t.co/dKP3o1e -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5/css/all.min.css">

<!--[if IE]>
  <style>
    /* old IE unsupported flexbox fixes */
    .greedy-nav .site-title {
      padding-right: 3em;
    }
    .greedy-nav button {
      position: absolute;
      top: 0;
      right: 0;
      height: 100%;
    }
  </style>
<![endif]-->


    <!-- start custom head snippets -->

<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    extensions: [
      "MathMenu.js",
      "MathZoom.js",
      "AssistiveMML.js",
      "a11y/accessibility-menu.js"
    ],
    jax: ["input/TeX", "output/CommonHTML"],
    TeX: {
      extensions: [
        "AMSmath.js",
        "AMSsymbols.js",
        "noErrors.js",
        "noUndefined.js",
      ]
    }
  });
</script>
<script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML">
</script>
<!-- insert favicons. use https://realfavicongenerator.net/ -->

<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/site.webmanifest">
<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#2b5797">
<meta name="theme-color" content="#ffffff">

<!-- end custom head snippets -->
  </head>

  <body class="layout--single">
    <nav class="skip-links">
  <h2 class="screen-reader-text">Skip links</h2>
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
          <a class="site-logo" href="/"><img src="/assets/images/id-88.png" alt=""></a>
        
        <a class="site-title" href="/">
          Jérémie K.
          
        </a>
        <ul class="visible-links"><li class="masthead__menu-item">
              <a href="/about/" title="I am a human. I wish to make the world better with science. <3">About</a>
            </li><li class="masthead__menu-item">
              <a href="/projects/" title="">Projects</a>
            </li><li class="masthead__menu-item">
              <a href="/opinions/">Opinions</a>
            </li><li class="masthead__menu-item">
              <a href="/ideas/">Ideas</a>
            </li><li class="masthead__menu-item">
              <a href="/blog/" title="A short extract about my life experiences.">Blog</a>
            </li><li class="masthead__menu-item">
              <a href="/papers/">Papers</a>
            </li><li class="masthead__menu-item">
              <a href="/courses/" title="Here is the list of the Courses I have received and the one I have given.">Courses</a>
            </li><li class="masthead__menu-item">
              <a href="/culture/" title="This is my personal list of books, Comics and films I love.">Culture</a>
            </li><li class="masthead__menu-item">
              <a href="/categories/" title="Everything sorted by categories.">Subjects</a>
            </li></ul>
        
        <button class="search__toggle" type="button">
          <span class="visually-hidden">Toggle search</span>
          <svg class="icon" width="16" height="16" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15.99 16">
            <path d="M15.5,13.12L13.19,10.8a1.69,1.69,0,0,0-1.28-.55l-0.06-.06A6.5,6.5,0,0,0,5.77,0,6.5,6.5,0,0,0,2.46,11.59a6.47,6.47,0,0,0,7.74.26l0.05,0.05a1.65,1.65,0,0,0,.5,1.24l2.38,2.38A1.68,1.68,0,0,0,15.5,13.12ZM6.4,2A4.41,4.41,0,1,1,2,6.4,4.43,4.43,0,0,1,6.4,2Z" transform="translate(-.01)"></path>
          </svg>
        </button>
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      


  
    



<nav class="breadcrumbs">
  <ol itemscope itemtype="https://schema.org/BreadcrumbList">
    
    
    
      
        <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
          <a href="http://localhost:4000/" itemprop="item"><span itemprop="name">Home</span></a>
          <meta itemprop="position" content="1" />
        </li>
        <span class="sep">/</span>
      
      
        
        <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
          <a href="/projects" itemprop="item"><span itemprop="name">Projects</span></a>
          <meta itemprop="position" content="2" />
        </li>
        <span class="sep">/</span>
      
    
      
      
        <li class="current">DepMap Omics: How to</li>
      
    
  </ol>
</nav>

  


<div id="main" role="main">
  


  <article class="page" itemscope itemtype="https://schema.org/CreativeWork">
    <meta itemprop="headline" content="DepMap Omics: How to">
    <meta itemprop="description" content="How DepMap Omics data is made. How to process it Yourself">
    <meta itemprop="datePublished" content="2020-08-10T00:00:00-04:00">
    

    <div class="page__inner-wrap">
      
        <header>
          <h1 id="page-title" class="page__title" itemprop="headline">DepMap Omics: How to
</h1>
          

  <p class="page__meta">
    

    

    
      
      

      <span class="page__meta-readtime">
        <i class="far fa-clock" aria-hidden="true"></i>
        
          6 minute read
        
      </span>
    
  </p>


        </header>
      

      <section class="page__content" itemprop="text">
        
          <aside class="sidebar__right ">
            <nav class="toc">
              <header><h4 class="nav__title"><i class="fas fa-file-alt"></i> On this page</h4></header>
              <ul class="toc__menu"><li><a href="#introduction">Introduction</a><ul><li><a href="#rnaseq-pipeline">RNAseq pipeline</a></li><li><a href="#wes-pipeline">WES pipeline</a></li></ul></li><li><a href="#why">Why</a></li><li><a href="#installation">Installation</a><ul><li><a href="#clone-the-required-repositories">clone the required repositories</a></li><li><a href="#you-will-need-the-approriate-r-packages-and-python-packages">You will need the approriate R packages and python packages</a></li><li><a href="#creating-your-terra-workspaces">Creating your Terra Workspaces:</a></li></ul></li><li><a href="#running-the-notebooks">Running the notebooks</a><ul><li><a href="#1-uploading-and-pre-processing">1. Uploading and pre-processing</a></li><li><a href="#2-running-terra-pipelines">2. Running Terra Pipelines</a></li><li><a href="#3-downloading-and-post-processing-often-called-22-4-on-local-in-the-notebooks">3. Downloading and post-processing (often called 2.2-4 on local in the notebooks)</a><ul><li><a href="#post-processing-tasks">Post-processing tasks</a></li></ul></li><li><a href="#4-qc-grouping-and-uploading-to-the-portal">4. QC, grouping and uploading to the portal</a></li></ul></li></ul>

            </nav>
          </aside>
        
        <p><em>The goal of this blog post is to explain how to run the DepMap omics pipeline as our brave Associate Computational Biologist do.</em></p>

<p><img src="/assets/images/depmap-logo.png" alt="" /></p>

<h2 id="introduction">Introduction</h2>

<p>Large panels of comprehensively characterized human cancer models have provided a rigorous framework with which to study genetic variants, candidate targets, and small-molecule and biological therapeutics and to identify new marker-driven cancer dependencies.</p>

<p>As part of its discovery effort of Cancer Dependencies an Targets, <a href="https://depmap.org/portal/">DepMap</a> analyzes raw omics data from cell lines on a quarterly basis. This part of the project if often referred to as CCLE. The bulk of ccle data consists of many omics sequences as shows in the <a href="https://pubmed.ncbi.nlm.nih.gov/31068700/">CCLE2 paper</a>. However only RNAseq and Whole Exome seq gets generated and analyzed quarterly. This project is lead by a team of researchers from the <a href="http://cancerdatascience.org/">CDS group</a> at the Broad Institute and draws from other projects such as <a href="https://www.gtexportal.org/home/">GTeX</a>, <a href="https://software.broadinstitute.org/cancer/cga/">TCGA</a> and <a href="https://portals.broadinstitute.org/cellfactory#home">CCLF</a>.</p>

<h3 id="rnaseq-pipeline">RNAseq pipeline</h3>
<p><img src="/assets/images/RNAseq_pipeline.png" alt="" /></p>

<h3 id="wes-pipeline">WES pipeline</h3>
<p><img src="/assets/images/WES_pipeline.png" alt="" /></p>

<p>The following tools are used in our pipelines:</p>
<ul>
  <li><strong>star</strong>:
    <ul>
      <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/23104886">https://www.ncbi.nlm.nih.gov/pubmed/23104886</a></li>
      <li><a href="https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf">https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf</a></li>
    </ul>
  </li>
  <li><strong>rsem</strong>:
    <ul>
      <li><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323</a></li>
    </ul>
  </li>
  <li><strong>star fusion</strong>:
    <ul>
      <li><a href="https://github.com/STAR-Fusion/STAR-Fusion/wiki">https://github.com/STAR-Fusion/STAR-Fusion/wiki</a></li>
      <li><a href="http://biorxiv.org/content/early/2017/03/24/120295">http://biorxiv.org/content/early/2017/03/24/120295</a></li>
    </ul>
  </li>
  <li><strong>mutect</strong>:
    <ul>
      <li><a href="https://software.broadinstitute.org/cancer/cga/mutect">https://software.broadinstitute.org/cancer/cga/mutect</a></li>
      <li><a href="https://youtu.be/rN-cLrb5aGs">https://youtu.be/rN-cLrb5aGs</a></li>
      <li><a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/4.beta.4/org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.php">https://software.broadinstitute.org/gatk/documentation/tooldocs/4.beta.4/org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.php</a></li>
      <li><a href="https://www.nature.com/articles/nbt.2514">https://www.nature.com/articles/nbt.2514</a></li>
    </ul>
  </li>
  <li><strong>gatk cnv</strong>:
    <ul>
      <li><a href="https://software.broadinstitute.org/gatk/documentation/article?id=11682">https://software.broadinstitute.org/gatk/documentation/article?id=11682</a></li>
    </ul>
  </li>
  <li><strong>strelka</strong>:
    <ul>
      <li><a href="https://www.nature.com/articles/s41592-018-0051-x">https://www.nature.com/articles/s41592-018-0051-x</a></li>
      <li><a href="https://github.com/Illumina/strelka">https://github.com/Illumina/strelka</a></li>
    </ul>
  </li>
</ul>

<p>Furthermore the pipelines make use of several software development tools. In particular some familiarity with the following tools are recommneded:</p>

<ul>
  <li><a href="https://www.learnpython.org/">python</a></li>
  <li><a href="https://www.codecademy.com/learn/learn-r">R</a></li>
  <li><a href="https://jupyter.org/index.html">jupyter</a></li>
  <li><a href="https://software.broadinstitute.org/wdl/documentation/">WDL</a></li>
  <li><a href="https://cloud.google.com/sdk/docs/quickstart-macos">gcp</a></li>
  <li><a href="https://docs.docker.com/get-started/">docker</a></li>
  <li><a href="https://software.broadinstitute.org/firecloud/documentation/">Terra</a></li>
  <li><a href="https://github.com/broadinstitute/dalmatian">dalmatian</a></li>
  <li><a href="https://docs.google.com/document/d/1zTtaN-Px64f8JvgydZNdBbzBpFWyZzEpshSNxQh43Oc/edit#heading=h.dz5wh0l4bu9g">Terra and gcp</a></li>
</ul>

<h2 id="why">Why</h2>

<p>Knowing how to run these pipelines will help you if you want to:</p>
<ol>
  <li>run your own large scale multi omics analysis pipeline. #Fancy</li>
  <li>if you want to reprocess some of DepMap-omics sequencing in the same way it has been done to generate what is available on depmap.</li>
</ol>

<p>The next sections are a detailed walkthrough to run DempMap omics pipelines on the <a href="https://app.terra.bio/">Terra</a> platform.</p>

<h2 id="installation">Installation</h2>

<h3 id="clone-the-required-repositories">clone the required repositories</h3>

<p>This repo uses some important data and code from the <a href="https://www.github.com/jkobject/JKBio">JKBio Library</a> custom repositories. This repository and the <em>ccle_processing</em> one should be cloned into the same path using the following command:</p>

<div class="language-bash highlighter-rouge"><div class="highlight"><pre class="highlight"><code>git clone https://github.com/broadinstitute/ccle_processing.git
git clone https://github.com/jkobject/JKBio.git
</code></pre></div></div>

<h3 id="you-will-need-the-approriate-r-packages-and-python-packages">You will need the approriate R packages and python packages</h3>

<p><em>note that up to date versions are recommended</em></p>

<ol>
  <li>You will need to install <a href="https://jupyter.org/install">Jupyter Notebook</a> and Google Cloud SDK
    <ul>
      <li>install <a href="https://cloud.google.com/sdk/docs/downloads-interactive">Google Cloud SDK</a>.</li>
      <li>authenticate your Google account by running <code class="language-plaintext highlighter-rouge">gcloud auth application-default login</code> in the terminal.</li>
    </ul>
  </li>
  <li>For R packages, a loading function contains most required ones (in <a href="https://github.com/broadinstitute/ccle_proccessing/blob/master/src/load_libraries_and_annotations.R">here</a>)</li>
  <li>install the following additional R packages using the provided commands:
    <ul>
      <li>Another R package needs to be installed like so: <code class="language-plaintext highlighter-rouge">cd src/cdsomics &amp;&amp; R CMD INSTALL . &amp;&amp; cd -</code>.</li>
      <li>And Celllinemapr: <code class="language-plaintext highlighter-rouge">cd ../JKBio/cell_line_mapping-master/celllinemapr &amp;&amp; R CMD INSTALL . &amp;&amp; cd -</code>.</li>
    </ul>
  </li>
  <li>For Python use the requirements.txt file <code class="language-plaintext highlighter-rouge">pip install -r requirements.txt</code>.</li>
</ol>

<h3 id="creating-your-terra-workspaces">Creating your Terra Workspaces:</h3>

<p>How to create workspaces and upload data to it is explained in <a href="https://software.broadinstitute.org/firecloud/documentation/">Terra’s documentation</a></p>

<p>For the specific workflow ids, parameters and workspace data you can use the file <code class="language-plaintext highlighter-rouge">data/xQx/.json</code> in our <em>ccle_processing</em> repository, which lists the parameters used for each workflow of each off the 3 workspaces in our pipelines for each releases (the .csv file lists the workflows with their correct name):
In order to have a working environment you will need to make sure you have:</p>
<ul>
  <li>set a billing account</li>
  <li>created a workspace</li>
  <li>added the correct workspace data to it (the workspace parameters/data listed in the <code class="language-plaintext highlighter-rouge">GENERAL</code> field of the json file)</li>
  <li>imported the right workflows (in the csv file)</li>
  <li>parametrized them (in the json file)</li>
  <li>imported your data (<em>Tools available in <code class="language-plaintext highlighter-rouge">TerraFunction</code> in the <code class="language-plaintext highlighter-rouge">JKBio</code> package as well as the <code class="language-plaintext highlighter-rouge">dRalmatian</code> package can be used to automate these processes.</em>)</li>
</ul>

<p>Once this is done, you can run your Jupyter Notebook server and open one of the 3 <code class="language-plaintext highlighter-rouge">\*_CCLE</code> Jupyter files in the <em>ccle_processing</em> repo, corresponding to our 3 pipelines.</p>
<ul>
  <li>RNAseq processing of gene level and transcript level read counts and exon counts.</li>
  <li>WESeq processing of Copy Number (gene level and segment level)</li>
  <li>RNAseq &amp; WESeq processing of Mutations (cancer drivers and germlines)</li>
</ul>

<p>These notebooks architecture is as follows:</p>

<ol>
  <li>Uploading and pre-processing</li>
  <li>Running the Terra Pipelines</li>
  <li>Downloading and post-processing</li>
  <li>QC, grouping and uploading to the portal</li>
</ol>

<h2 id="running-the-notebooks">Running the notebooks</h2>

<p>Each Following steps correspond to a specific part of each of the 3 notebooks.</p>

<p><em>Note that you can try it out using our publicly available bam files on <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA523380">SRA</a></em></p>

<h3 id="1-uploading-and-pre-processing">1. Uploading and pre-processing</h3>

<p><strong>You can skip this step and go to part2.</strong></p>

<p>The first step in the notebook is about getting the samples generated at the the Broad Institute which are located in different Google Cloud storage paths. This section also searches for duplicates and finds/adds the metadata we will need in order to have coherent and complete sample information.</p>

<p><strong>Remarks:</strong></p>
<ul>
  <li>in the initialization step you can remove any imports related to <code class="language-plaintext highlighter-rouge">taiga</code> and <code class="language-plaintext highlighter-rouge">gsheet</code> to avoid possible errors.</li>
  <li>feel free to reuse <code class="language-plaintext highlighter-rouge">createDatasetWithNewCellLines</code>, <code class="language-plaintext highlighter-rouge">GetNewCellLinesFromWorkspaces</code> or any other function for your own needs.</li>
</ul>

<h3 id="2-running-terra-pipelines">2. Running Terra Pipelines</h3>

<p>Before running this part, you need to make sure that your dalmatian <code class="language-plaintext highlighter-rouge">workspacemanager</code> object is initialized with the right workspace you created and that the functions take as input you workflow names. You also need to make sure that you created your sample set with all your samples and that you initialized the <code class="language-plaintext highlighter-rouge">sampleset</code> string with its name
You can then run the pipeline on your samples. It would approximately take a day to run all of the pipelines.</p>

<p><strong>Remarks:</strong></p>
<ul>
  <li>for the copy number pipeline we have parametrized both an XX version and an XY version, we recommend using the XY version as it covers the entire genome</li>
  <li>for the mutation calling pipeline we are working on Tumor-Normal pairs which explain some of the back and forth between the two workspace data table (the existing workflows work with or without matched normals).</li>
  <li>for the expression pipeline, we have an additional set of workflows to call mutations from RNAseq, this might not be relevant to your need.</li>
</ul>

<h3 id="3-downloading-and-post-processing-often-called-22-4-on-local-in-the-notebooks">3. Downloading and post-processing (often called <strong>2.2-4 on local</strong> in the notebooks)</h3>

<p>This step will do the following tasks:</p>
<ul>
  <li>delete large intermediary files on the workspace that are not needed</li>
  <li>retrieve from the workspace interesting QC results.</li>
  <li>copy realigned BAM files to a target bucket.</li>
  <li>download the results.</li>
  <li>remove all duplicate samples from our downloaded file (keeping only the latest version of each samples).</li>
  <li>saving the current pipeline configuration.</li>
</ul>

<p><em>You will at least need to download results, but other steps could be useful for some users</em></p>

<h4 id="post-processing-tasks">Post-processing tasks</h4>

<blockquote>
  <p>Unfortunately for now the post-processing tasks are not all made to be easily run outside of the CCLE pipelines. Most of them are in R and are run with the Rpy2 tool.</p>
</blockquote>

<p>So amongst these functions, some of them might be of a lesser interest to an external user. The most important ones for each pipelines are:</p>

<ul>
  <li><code class="language-plaintext highlighter-rouge">processSegments</code></li>
  <li><code class="language-plaintext highlighter-rouge">interpolateGapsInSegmented</code></li>
  <li><code class="language-plaintext highlighter-rouge">extendEndsOfSegments</code></li>
  <li><code class="language-plaintext highlighter-rouge">generateEntrezGenes</code></li>
  <li>
    <p><code class="language-plaintext highlighter-rouge">generateGeneLevelMatrixFromSegments</code></p>
  </li>
  <li><code class="language-plaintext highlighter-rouge">readMutations</code></li>
  <li><code class="language-plaintext highlighter-rouge">createSNPs</code></li>
  <li><code class="language-plaintext highlighter-rouge">filterAllelicFraction</code></li>
  <li><code class="language-plaintext highlighter-rouge">filterMinCoverage</code> <em>you would need to rewrite this function as it now runs on DepMap columns. In one line it requires that the total coverage of that site (aggregated across methods) &gt; 8 and that there are at least 4 alternate alleles for each mutations</em></li>
  <li><code class="language-plaintext highlighter-rouge">maf_add_variant_annotations</code></li>
  <li>
    <p><code class="language-plaintext highlighter-rouge">mutation_maf_to_binary_matrix</code></p>
  </li>
  <li><code class="language-plaintext highlighter-rouge">readTranscripts</code></li>
  <li><code class="language-plaintext highlighter-rouge">readCounts</code></li>
  <li><code class="language-plaintext highlighter-rouge">readTPM</code></li>
  <li><code class="language-plaintext highlighter-rouge">readFusions</code></li>
  <li><code class="language-plaintext highlighter-rouge">filterFusions</code></li>
  <li>the <code class="language-plaintext highlighter-rouge">step 2.2.5</code> where we remove samples with more then 39k transcripts with 0 readcounts.</li>
  <li><code class="language-plaintext highlighter-rouge">prepare_depmap_\*\_for_taiga</code></li>
</ul>

<p><strong>Remarks:</strong></p>
<ul>
  <li>in the RNAseq pipeline we have an additional sub-pipeline at the end of the notebook to process the fusion calls from STAR-Fusion.</li>
  <li>to get the exact same results as in CCLE, be sure to run <code class="language-plaintext highlighter-rouge">genecn = genecn.apply(lambda x: np.log2(1+x))</code> to produce the <code class="language-plaintext highlighter-rouge">genecn</code> dataframe in the CNV pipeline (present at the end of the validation steps).</li>
  <li>we do not yet have integrated our germline calling in the mutation pipeline but you can still find the haplotypeCaller|DeepVariant workflows and their parameters</li>
</ul>

<h3 id="4-qc-grouping-and-uploading-to-the-portal">4. QC, grouping and uploading to the portal</h3>

<p>These tasks should not be very interesting for any outside user as they revolve around manual checks of the data, comparison to previous releases, etc.</p>

<p>In these tasks we also prepare the data to be released to different groups, removing the samples per access category: Blacklist|Internal|DepMapConsortium|Public. We then upload the data to a server called <strong>taiga</strong> where it they be used by the DepMap portal to create the plots you know and love.</p>

<p>You can however find all of these files directly on the <a href="https://depmap.org/portal/download/">download section</a></p>

<hr />

<p>you can find the project’s repository <a href="https://github.com/broadinstitute/ccle_processing">here</a>
the blog post is available on <a href="">depmap’s blog</a> and <a href="https://www.jkobject.com/projects/depmap_omics_howto/">Jérémie’s blog</a></p>

<p>Please feel free to ask any question on our <a href="https://depmap.org/portal/faq/">FAQ</a></p>

<p>Jérémie Kalfon</p>

        
      </section>

      <footer class="page__meta">
        
        
  


  

  <p class="page__taxonomy">
    <strong><i class="fas fa-fw fa-tags" aria-hidden="true"></i> Tags: </strong>
    <span itemprop="keywords">
    
      <a href="/tags/#broad-institute" class="page__taxonomy-item" rel="tag">Broad Institute</a><span class="sep">, </span>
    
      <a href="/tags/#cancer-dependencies" class="page__taxonomy-item" rel="tag">Cancer Dependencies</a><span class="sep">, </span>
    
      <a href="/tags/#data-science" class="page__taxonomy-item" rel="tag">Data Science</a><span class="sep">, </span>
    
      <a href="/tags/#genomics" class="page__taxonomy-item" rel="tag">Genomics</a><span class="sep">, </span>
    
      <a href="/tags/#research" class="page__taxonomy-item" rel="tag">Research</a>
    
    </span>
  </p>




  


  

  <p class="page__taxonomy">
    <strong><i class="fas fa-fw fa-folder-open" aria-hidden="true"></i> Categories: </strong>
    <span itemprop="keywords">
    
      <a href="/categories/#broadinstitute" class="page__taxonomy-item" rel="tag">BroadInstitute</a>
    
    </span>
  </p>


        
  <p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2020-08-10T00:00:00-04:00">August 10, 2020</time></p>


      </footer>

      <section class="page__share">
  
    <h4 class="page__share-title">Share on</h4>
  

  <a href="https://twitter.com/intent/tweet?via=jkobject&text=DepMap+Omics%3A+How+to%20http%3A%2F%2Flocalhost%3A4000%2Fprojects%2Fdepmap-omics-howto%2F" class="btn btn--twitter" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Twitter"><i class="fab fa-fw fa-twitter" aria-hidden="true"></i><span> Twitter</span></a>

  <a href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Flocalhost%3A4000%2Fprojects%2Fdepmap-omics-howto%2F" class="btn btn--facebook" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Facebook"><i class="fab fa-fw fa-facebook" aria-hidden="true"></i><span> Facebook</span></a>

  <a href="https://www.linkedin.com/shareArticle?mini=true&url=http%3A%2F%2Flocalhost%3A4000%2Fprojects%2Fdepmap-omics-howto%2F" class="btn btn--linkedin" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on LinkedIn"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span> LinkedIn</span></a>
</section>


      
  <nav class="pagination">
    
      <a href="/projects/wrn-dependencies/" class="pagination--pager" title="WRN dependencies
">Previous</a>
    
    
      <a href="/projects/AMLproject/" class="pagination--pager" title="AMLproject
">Next</a>
    
  </nav>

    </div>

    
  </article>

  
  
    <div class="page__related">
      <h4 class="page__related-title">You may also enjoy</h4>
      <div class="grid__wrapper">
        
          



<div class="grid__item">
  <article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork">
    
      <div class="archive__item-teaser">
        <img src="/assets/images/tf_cobinding.jpg" alt="">
      </div>
    
    <h2 class="archive__item-title no_toc" itemprop="headline">
      
        <a href="/blog/cds-presentation/" rel="permalink">A look at Cobinding Analysis
</a>
      
    </h2>
    

  <p class="page__meta">
    

    

    
      
      

      <span class="page__meta-readtime">
        <i class="far fa-clock" aria-hidden="true"></i>
        
          less than 1 minute read
        
      </span>
    
  </p>


    <p class="archive__item-excerpt" itemprop="description">A look at my cobinding analysis as part of my research on the epigenetics of AML dependencies
</p>
  </article>
</div>

        
          



<div class="grid__item">
  <article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork">
    
      <div class="archive__item-teaser">
        <img src="/assets/images/yearbroaad.jpg" alt="">
      </div>
    
    <h2 class="archive__item-title no_toc" itemprop="headline">
      
        <a href="/blog/one-year-at-the-broad/" rel="permalink">A year at the Broad Institute
</a>
      
    </h2>
    

  <p class="page__meta">
    

    

    
      
      

      <span class="page__meta-readtime">
        <i class="far fa-clock" aria-hidden="true"></i>
        
          7 minute read
        
      </span>
    
  </p>


    <p class="archive__item-excerpt" itemprop="description">Short stories about my professional experiences.
</p>
  </article>
</div>

        
          



<div class="grid__item">
  <article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork">
    
      <div class="archive__item-teaser">
        <img src="/assets/images/dimred.jpg" alt="">
      </div>
    
    <h2 class="archive__item-title no_toc" itemprop="headline">
      
        <a href="/blog/umap-explanation/" rel="permalink">UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction
</a>
      
    </h2>
    

  <p class="page__meta">
    

    

    
      
      

      <span class="page__meta-readtime">
        <i class="far fa-clock" aria-hidden="true"></i>
        
          4 minute read
        
      </span>
    
  </p>


    <p class="archive__item-excerpt" itemprop="description">Short stories about my professional experiences.
</p>
  </article>
</div>

        
          



<div class="grid__item">
  <article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork">
    
      <div class="archive__item-teaser">
        <img src="/assets/images/advancedstudy.jpg" alt="">
      </div>
    
    <h2 class="archive__item-title no_toc" itemprop="headline">
      
        <a href="/blog/a-month-at-the-broad/" rel="permalink">My first month at the Broad Institute
</a>
      
    </h2>
    

  <p class="page__meta">
    

    

    
      
      

      <span class="page__meta-readtime">
        <i class="far fa-clock" aria-hidden="true"></i>
        
          3 minute read
        
      </span>
    
  </p>


    <p class="archive__item-excerpt" itemprop="description">I arrived in Boston exactly a month ago. I managed to get my Visa 5 hours before catching…
</p>
  </article>
</div>

        
      </div>
    </div>
  
  
</div>

    </div>

    
      <div class="search-content">
        <div class="search-content__inner-wrap"><form class="search-content__form" onkeydown="return event.key != 'Enter';">
    <label class="sr-only" for="search">
      Enter your search term...
    </label>
    <input type="search" id="search" class="search-input" tabindex="-1" placeholder="Enter your search term..." />
  </form>
  <div id="results" class="results"></div></div>

      </div>
    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    
      
        
          <li><a href="mailto:jkobject@gmail.com" rel="nofollow noopener noreferrer"><i class="fas fa-fw fa-envelope-square" aria-hidden="true"></i> Email</a></li>
        
      
        
          <li><a href="https://twitter.com/jkobject" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
        
      
        
          <li><a href="https://github.com/jkobject" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-github" aria-hidden="true"></i> GitHub</a></li>
        
      
        
          <li><a href="https://ResearchGate.net/profile/jeremie_kalfon" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-researchgate" aria-hidden="true"></i> ResearchGate</a></li>
        
      
        
          <li><a href="https://Scholar.google.com/citations?user=zyXiydsAAAAJ" rel="nofollow noopener noreferrer"><i class="fas fa-fw fa-graduation-cap" aria-hidden="true"></i> Scholar</a></li>
        
      
        
          <li><a href="https://LinkedIn.com/in/jkobject" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i> LinkedIn</a></li>
        
      
    

    
      <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
    
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2021 Jérémie Kalfon. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js"></script>




<script src="/assets/js/lunr/lunr.min.js"></script>
<script src="/assets/js/lunr/lunr-store.js"></script>
<script src="/assets/js/lunr/lunr-en.js"></script>




    
  <script>
    var disqus_config = function () {
      this.page.url = "http://localhost:4000/projects/depmap-omics-howto/";  /* Replace PAGE_URL with your page's canonical URL variable */
      this.page.identifier = "/projects/depmap-omics-howto"; /* Replace PAGE_IDENTIFIER with your page's unique identifier variable */
    };
    (function() { /* DON'T EDIT BELOW THIS LINE */
      var d = document, s = d.createElement('script');
      s.src = 'https://jkobject.disqus.com/embed.js';
      s.setAttribute('data-timestamp', +new Date());
      (d.head || d.body).appendChild(s);
    })();
  </script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


  





              <script>
                window.onload = function () {
                    var script = document.createElement('script');
                    var firstScript = document.getElementsByTagName('script')[0];
                    script.type = 'text/javascript';
                    script.async = true;
                    script.src = '/sw-register.js?v=' + Date.now();
                    firstScript.parentNode.insertBefore(script, firstScript);
                };
            </script>
            </body>

</html>
